File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-CG-IG-HLA-FullBundle-01.fsh

Instance: bundle-CG-IG-HLA-FullBundle-01
InstanceOf: Bundle
Description: "Example of a HLA bundle"
Usage: #example
* type = #transaction
* entry[0].fullUrl = "urn:uuid:13f34265-335c-4853-bc38-0815315edafa"
* entry[=].resource = CG-IG-HLA-FullBundle-01-1
* entry[=].request.method = #POST
* entry[=].request.url = "Patient"
* entry[+].fullUrl = "urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340"
* entry[=].resource = CG-IG-HLA-FullBundle-01-2
* entry[=].request.method = #POST
* entry[=].request.url = "Specimen"
* entry[+].fullUrl = "urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950"
* entry[=].resource = CG-IG-HLA-FullBundle-01-3
* entry[=].request.method = #POST
* entry[=].request.url = "Organization"
* entry[+].fullUrl = "urn:uuid:00ef18ad-ed04-4b2c-81ee-b69bb243f0d5"
* entry[=].resource = CG-IG-HLA-FullBundle-01-4
* entry[=].request.method = #POST
* entry[=].request.url = "Organization"
* entry[+].fullUrl = "urn:uuid:99309303-045e-4cf4-90d7-250d7a7476ea"
* entry[=].resource = CG-IG-HLA-FullBundle-01-5
* entry[=].request.method = #POST
* entry[=].request.url = "ServiceRequest"
* entry[+].fullUrl = "urn:uuid:8200dab6-18a2-4550-b913-a7db480c0804"
* entry[=].resource = CG-IG-HLA-FullBundle-01-6
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:7c393185-f15c-45bc-a714-c0fdbea32675"
* entry[=].resource = CG-IG-HLA-FullBundle-01-7
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:65c85f14-c3a0-4b72-818f-820e04fcc621"
* entry[=].resource = CG-IG-HLA-FullBundle-01-8
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:fbba9fe7-0ece-4ec1-9233-a437a8d242a0"
* entry[=].resource = CG-IG-HLA-FullBundle-01-9
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:b7765bbf-df40-486a-9f2f-404309643de6"
* entry[=].resource = CG-IG-HLA-FullBundle-01-10
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:d98d92a7-0e86-4ae5-b036-b7e1bba6ec32"
* entry[=].resource = CG-IG-HLA-FullBundle-01-11
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:49a86246-4004-42eb-9bdc-f542f93f9228"
* entry[=].resource = CG-IG-HLA-FullBundle-01-12
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:cbabf93e-1b4b-46f2-ba1e-d84862670670"
* entry[=].resource = CG-IG-HLA-FullBundle-01-13
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:c233ad3d-1572-48d6-93da-0a583535e138"
* entry[=].resource = CG-IG-HLA-FullBundle-01-14
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:05fa52d7-5c67-460a-8722-d3460b24d6fe"
* entry[=].resource = CG-IG-HLA-FullBundle-01-15
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:db69e549-6267-4777-b4b9-8813f3329309"
* entry[=].resource = CG-IG-HLA-FullBundle-01-16
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:e2092243-2970-49d2-a90f-b90d1d49715a"
* entry[=].resource = CG-IG-HLA-FullBundle-01-17
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:792be53e-d4fb-4887-a367-815ef6c706e5"
* entry[=].resource = CG-IG-HLA-FullBundle-01-18
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:60613a43-c4cb-4502-b3e2-cf9215feaa70"
* entry[=].resource = CG-IG-HLA-FullBundle-01-19
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:bb55c2bc-5ad2-4bc1-8ff3-c407d06b12d0"
* entry[=].resource = CG-IG-HLA-FullBundle-01-20
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:46938bb2-0486-4e87-bfd3-89aab2d5e22f"
* entry[=].resource = CG-IG-HLA-FullBundle-01-21
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:2ae2ff34-279e-43c2-9018-b054fd3fc1ce"
* entry[=].resource = CG-IG-HLA-FullBundle-01-22
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:19153ef1-68c6-47a2-9676-c4eefbd39af9"
* entry[=].resource = CG-IG-HLA-FullBundle-01-23
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "urn:uuid:709c5315-9403-4867-9d82-0b953836665f"
* entry[=].resource = CG-IG-HLA-FullBundle-01-24
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:8b2aa21c-1426-4717-8ab0-a84d83df7d47"
* entry[=].resource = CG-IG-HLA-FullBundle-01-25
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:0e0a780e-4486-4cd0-bfae-7243c579f208"
* entry[=].resource = CG-IG-HLA-FullBundle-01-26
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:b0a4b18e-94e7-4b1b-8031-c7ae4bdd8db9"
* entry[=].resource = CG-IG-HLA-FullBundle-01-27
* entry[=].request.method = #POST
* entry[=].request.url = "DiagnosticReport"

Instance: CG-IG-HLA-FullBundle-01-1
InstanceOf: Patient
Usage: #inline
* identifier.use = #usual
* identifier.type = $IDTYPE#DR
* identifier.system = "urn:oid:0.0.0.0.0.0.0"
* identifier.value = "12345"
* identifier.period.start = "2012-11-10"
* identifier.assigner.display = "aDonorRegistry"
* name[0].use = #official
* name[=].text = "John Storm"
* name[=].family = "Storm"
* name[=].given = "John"
* name[+].use = #nickname
* name[=].text = "Johnny Storm"
* name[=].family = "Storm"
* name[=].given = "Johnny"
* name[+].use = #nickname
* name[=].text = "The Human Torch"
* gender = #male
* birthDate = "1986-12-31"

Instance: CG-IG-HLA-FullBundle-01-2
InstanceOf: Specimen
Usage: #inline
* identifier.system = "http://myorgsurl.com"
* identifier.value = "123"
* accessionIdentifier.system = "http://mylabsurl.com"
* accessionIdentifier.value = "456"
* type = $SCT#122555007 "Venous blood specimen"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"

Instance: CG-IG-HLA-FullBundle-01-3
InstanceOf: Organization
Usage: #inline
* name = "aTypingLab Inc"
* alias = "aTL"
* telecom.system = #phone
* telecom.value = "1-800-555-1234"
* telecom.use = #work
* telecom.rank = 1
* address.use = #work
* address.type = #both
* address.text = "123 Main St, Sometown, ND 99999"
* address.line = "123 Main St"
* address.city = "Sometown"
* address.state = "ND"
* address.postalCode = "99999"
* address.country = "USA"

Instance: CG-IG-HLA-FullBundle-01-4
InstanceOf: Organization
Usage: #inline
* name = "aDonorRegistry"
* alias = "ADR"
* telecom.system = #phone
* telecom.value = "1-800-555-6789"
* telecom.use = #work
* telecom.rank = 1
* address.use = #work
* address.type = #both
* address.text = "456 Main St, Anytown ND, 00000"
* address.line = "456 Main St"
* address.city = "Anytown"
* address.state = "ND"
* address.postalCode = "00000"
* address.country = "USA"

Instance: CG-IG-HLA-FullBundle-01-5
InstanceOf: ServiceRequest
Usage: #inline
* status = #completed
* intent = #order
* code = $LNC#13303-3 "HLA-A+B+C (class I) [Type]"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* authoredOn = "2016-11-15"
* requester = Reference(urn:uuid:00ef18ad-ed04-4b2c-81ee-b69bb243f0d5) "aDonorRegistry"
  * type = "Organization"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
  * type = "Organization"
* reasonCode.text = "tissue typing for donor registry"

Instance: CG-IG-HLA-FullBundle-01-6
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00001
* referenceSeq.referenceSeqId.text = "HLA-A*01:01:01:01"
* referenceSeq.windowStart = 503
* referenceSeq.windowEnd = 773
* observedSeq = "GCTCCCACTCCATGAGGTATTTCTTCACATCCGTGTCCCGGCCCGGCCGCGGGGAGCCCCGCTTCATCGCCGTGGGCTACGTGGACGACACGCAGTTCGTGCGGTTCGACAGCGACGCCGCGAGCCAGAAGATGGAGCCGCGGGCGCCGTGGATAGAGCAGGAGGGGCCGGAGTATTGGGACCAGGAGACACGGAATATGAAGGCCCACTCACAGACTGACCGAGCGAACCTGGGGACCCTGCGCGGCTACTACAACCAGAGCGAGGACG"

Instance: CG-IG-HLA-FullBundle-01-7
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00001
* referenceSeq.referenceSeqId.text = "HLA-A*01:01:01:01"
* referenceSeq.windowStart = 1014
* referenceSeq.windowEnd = 1290
* observedSeq = "GTTCTCACACCATCCAGATAATGTATGGCTGCGACGTGGGGCCGGACGGGCGCTTCCTCCGCGGGTACCGGCAGGACGCCTACGACGGCAAGGATTACATCGCCCTGAACGAGGACCTGCGCTCTTGGACCGCGGCGGACATGGCAGCTCAGATCACCAAGCGCAAGTGGGAGGCGGTCCATGCGGCGGAGCAGCGGAGAGTCTACCTGGAGGGCCGGTGCGTGGACGGGCTCCGCAGATACCTGGAGAACGGGAAGGAGACGCTGCAGCGCACGG"

Instance: CG-IG-HLA-FullBundle-01-8
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00002
* referenceSeq.referenceSeqId.text = "HLA-A*01:02"
* referenceSeq.windowStart = 503
* referenceSeq.windowEnd = 773
* observedSeq = "GCTCCCACTCCATGAGGTATTTCTCCACATCCGTGTCCCGGCCCGGCAGTGGAGAGCCCCGCTTCATCGCAGTGGGCTACGTGGACGACACGCAGTTCGTGCGGTTCGACAGCGACGCCGCGAGCCAGAAGATGGAGCCGCGGGCGCCGTGGATAGAGCAGGAGGGGCCGGAGTATTGGGACCAGGAGACACGGAATATGAAGGCCCACTCACAGACTGACCGAGCGAACCTGGGGACCCTGCGCGGCTACTACAACCAGAGCGAGGACG"

Instance: CG-IG-HLA-FullBundle-01-9
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00002
* referenceSeq.referenceSeqId.text = "HLA-A*01:02"
* referenceSeq.windowStart = 1014
* referenceSeq.windowEnd = 1290
* observedSeq = "GTTCTCACACCATCCAGATAATGTATGGCTGCGACGTGGGGCCGGACGGGCGCTTCCTCCGCGGGTACCGGCAGGACGCCTACGACGGCAAGGATTACATCGCCCTGAACGAGGACCTGCGCTCTTGGACCGCGGCGGACATGGCAGCTCAGATCACCAAGCGCAAGTGGGAGGCGGTCCATGCGGCGGAGCAGCGGAGAGTCTACCTGGAGGGCCGGTGCGTGGACGGGCTCCGCAGATACCTGGAGAACGGGAAGGAGACGCTGCAGCGCACGG"

Instance: CG-IG-HLA-FullBundle-01-10
InstanceOf: Haplotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype name"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* effectiveDateTime = "2016-12-15"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "3.23"
* valueCodeableConcept.coding = $HLAALLELE#HLA-A*01:01:01G "HLA-A*01:01:01G"
* method = $GTR#GTR000000000.0
  * text = "NGS based Class I HLA-A, -B, -C genotyping"
* specimen = Reference(urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340) "buccal swab from John Storm"
* derivedFrom[0] = Reference(urn:uuid:8200dab6-18a2-4550-b913-a7db480c0804) "HLA-A*01:01:01:01, exon 2"
* derivedFrom[=].type = "MolecularSequence"
* derivedFrom[+] = Reference(urn:uuid:7c393185-f15c-45bc-a714-c0fdbea32675) "HLA-A*01:01:01:01, exon 3"
  * type = "MolecularSequence"
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4931 "HLA-A"

Instance: CG-IG-HLA-FullBundle-01-11
InstanceOf: Haplotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype name"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* effectiveDateTime = "2016-12-15"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "3.23"
* valueCodeableConcept.coding = $HLAALLELE#HLA-A*01:02 "HLA-A*01:02"
* method = $GTR#GTR000000000.0
  * text = "NGS based Class I HLA-A, -B, -C genotyping"
* specimen = Reference(urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340) "buccal swab from John Storm"
* derivedFrom[0] = Reference(urn:uuid:65c85f14-c3a0-4b72-818f-820e04fcc621) "HLA-A*01:02, exon 2"
* derivedFrom[=].type = "MolecularSequence"
* derivedFrom[+] = Reference(urn:uuid:fbba9fe7-0ece-4ec1-9233-a437a8d242a0) "HLA-A*01:02, exon 3"
  * type = "MolecularSequence"
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4931 "HLA-A"

Instance: CG-IG-HLA-FullBundle-01-12
InstanceOf: Genotype
Usage: #inline
* basedOn = Reference(urn:uuid:99309303-045e-4cf4-90d7-250d7a7476ea) "Class I HLA genotyping for John Storm"
  * type = "ServiceRequest"
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84413-4 "Genotype display name"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* effectiveDateTime = "2016-12-15"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "1.0"
* valueCodeableConcept.coding = $GLSTRING#hla#3.23.0#HLA-A:01:01G+HLA-A*01:02
* method = $GTR#GTR000000000.0
  * text = "NGS based Class I HLA-A, -B, -C genotyping"
* specimen = Reference(urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340) "buccal swab from John Storm"
* derivedFrom[0] = Reference(urn:uuid:b7765bbf-df40-486a-9f2f-404309643de6) "HLA-A*01:01:01G, exons 2 and 3"
* derivedFrom[=].type = "Observation"
* derivedFrom[+] = Reference(urn:uuid:d98d92a7-0e86-4ae5-b036-b7e1bba6ec32) "HLA-A*01:02, exons 2 and 3"
  * type = "Observation"
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4931 "HLA-A"

Instance: CG-IG-HLA-FullBundle-01-13
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00162
* referenceSeq.referenceSeqId.text = "HLA-B*15:01:01:01"
* referenceSeq.windowStart = 486
* referenceSeq.windowEnd = 756
* observedSeq = "GCTCCCACTCCATGAGGTATTTCTACACCGCCATGTCCCGGCCCGGCCGCGGGGAGCCCCGCTTCATCGCAGTGGGCTACGTGGACGACACCCAGTTCGTGAGGTTCGACAGCGACGCCGCGAGTCCGAGGATGGCGCCCCGGGCGCCATGGATAGAGCAGGAGGGGCCGGAGTATTGGGACCGGGAGACACAGATCTCCAAGACCAACACACAGACTTACCGAGAGAGCCTGCGGAACCTGCGCGGCTACTACAACCAGAGCGAGGCCG"

Instance: CG-IG-HLA-FullBundle-01-14
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00162
* referenceSeq.referenceSeqId.text = "HLA-B*15:01:01:01"
* referenceSeq.windowStart = 1001
* referenceSeq.windowEnd = 1277
* observedSeq = "GGTCTCACACCCTCCAGAGGATGTACGGCTGCGACGTGGGGCCGGACGGGCGCCTCCTCCGCGGGCATGACCAGTCCGCCTACGACGGCAAGGATTACATCGCCCTGAACGAGGACCTGAGCTCCTGGACCGCGGCGGACACGGCGGCTCAGATCACCCAGCGCAAGTGGGAGGCGGCCCGTGAGGCGGAGCAGTGGAGAGCCTACCTGGAGGGCCTGTGCGTGGAGTGGCTCCGCAGATACCTGGAGAACGGGAAGGAGACGCTGCAGCGCGCGG"

Instance: CG-IG-HLA-FullBundle-01-15
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00381
* referenceSeq.referenceSeqId.text = "HLA-B*57:01:01"
* referenceSeq.windowStart = 485
* referenceSeq.windowEnd = 755
* observedSeq = "GCTCCCACTCCATGAGGTATTTCTACACCGCCATGTCCCGGCCCGGCCGCGGGGAGCCCCGCTTCATCGCAGTGGGCTACGTGGACGACACCCAGTTCGTGAGGTTCGACAGCGACGCCGCGAGTCCGAGGATGGCGCCCCGGGCGCCATGGATAGAGCAGGAGGGGCCGGAGTATTGGGACGGGGAGACACGGAACATGAAGGCCTCCGCGCAGACTTACCGAGAGAACCTGCGGATCGCGCTCCGCTACTACAACCAGAGCGAGGCCG"

Instance: CG-IG-HLA-FullBundle-01-16
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00381
* referenceSeq.referenceSeqId.text = "HLA-B*57:01:01"
* referenceSeq.windowStart = 1001
* referenceSeq.windowEnd = 1277
* observedSeq = "GGTCTCACATCATCCAGGTGATGTATGGCTGCGACGTGGGGCCGGACGGGCGCCTCCTCCGCGGGCATGACCAGTCCGCCTACGACGGCAAGGATTACATCGCCCTGAACGAGGACCTGAGCTCCTGGACCGCGGCGGACACGGCGGCTCAGATCACCCAGCGCAAGTGGGAGGCGGCCCGTGTGGCGGAGCAGCTGAGAGCCTACCTGGAGGGCCTGTGCGTGGAGTGGCTCCGCAGATACCTGGAGAACGGGAAGGAGACGCTGCAGCGCGCGG"

Instance: CG-IG-HLA-FullBundle-01-17
InstanceOf: Haplotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype name"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* effectiveDateTime = "2016-12-15"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "3.23"
* valueCodeableConcept.coding = $HLAALLELE#HGG00041 "HLA-B*15:01:01G"
* method = $GTR#GTR000000000.0
  * text = "NGS based Class I HLA-A, -B, -C genotyping"
* specimen = Reference(urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340) "buccal swab from John Storm"
* derivedFrom[0] = Reference(urn:uuid:cbabf93e-1b4b-46f2-ba1e-d84862670670) "HLA-B*15:01:01:01, exon 2"
* derivedFrom[=].type = "MolecularSequence"
* derivedFrom[+] = Reference(urn:uuid:c233ad3d-1572-48d6-93da-0a583535e138) "HLA-B*15:01:01:01, exon 3"
  * type = "MolecularSequence"
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4932 "HLA-B"

Instance: CG-IG-HLA-FullBundle-01-18
InstanceOf: Haplotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype name"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* effectiveDateTime = "2016-12-15"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "3.23"
* valueCodeableConcept.coding = $HLAALLELE#HLA-B*57:01:01G "HLA-B*57:01:01G"
* method = $GTR#GTR000000000.0
  * text = "NGS based Class I HLA-A, -B, -C genotyping"
* specimen = Reference(urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340) "buccal swab from John Storm"
* derivedFrom[0] = Reference(urn:uuid:05fa52d7-5c67-460a-8722-d3460b24d6fe) "HLA-B*57:01:01, exon 2"
* derivedFrom[=].type = "MolecularSequence"
* derivedFrom[+] = Reference(urn:uuid:db69e549-6267-4777-b4b9-8813f3329309) "HLA-B*57:01:01, exon 3"
  * type = "MolecularSequence"
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4932 "HLA-B"

Instance: CG-IG-HLA-FullBundle-01-19
InstanceOf: Genotype
Usage: #inline
* basedOn = Reference(urn:uuid:99309303-045e-4cf4-90d7-250d7a7476ea) "Class I HLA genotyping for John Storm"
  * type = "ServiceRequest"
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84413-4 "Genotype display name"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* effectiveDateTime = "2016-12-15"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "1.0"
* valueCodeableConcept.coding = $GLSTRING#hla#3.23.0#HLA-B*15:01:01G+HLA-B*57:01:01G
* method = $GTR#GTR000000000.0
  * text = "NGS based Class I HLA-A, -B, -C genotyping"
* specimen = Reference(urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340) "buccal swab from John Storm"
* derivedFrom[0] = Reference(urn:uuid:e2092243-2970-49d2-a90f-b90d1d49715a) "HLA-B*15:01:01G, exons 2 and 3"
* derivedFrom[=].type = "Observation"
* derivedFrom[+] = Reference(urn:uuid:792be53e-d4fb-4887-a367-815ef6c706e5) "HLA-B*57:01:01G, exons 2 and 3"
  * type = "Observation"
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4932 "HLA-B"

Instance: CG-IG-HLA-FullBundle-01-20
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00401
* referenceSeq.referenceSeqId.text = "HLA-C*01:02:01"
* referenceSeq.windowStart = 486
* referenceSeq.windowEnd = 756
* observedSeq = "GCTCCCACTCCATGAAGTATTTCTTCACATCCGTGTCCCGGCCTGGCCGCGGAGAGCCCCGCTTCATCTCAGTGGGCTACGTGGACGACACGCAGTTCGTGCGGTTCGACAGCGACGCCGCGAGTCCGAGAGGGGAGCCGCGGGCGCCGTGGGTGGAGCAGGAGGGGCCGGAGTATTGGGACCGGGAGACACAGAAGTACAAGCGCCAGGCACAGACTGACCGAGTGAGCCTGCGGAACCTGCGCGGCTACTACAACCAGAGCGAGGCCG"

Instance: CG-IG-HLA-FullBundle-01-21
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00401
* referenceSeq.referenceSeqId.text = "HLA-C*01:02:01"
* referenceSeq.windowStart = 1002
* referenceSeq.windowEnd = 1278
* observedSeq = "GGTCTCACACCCTCCAGTGGATGTGTGGCTGCGACCTGGGGCCCGACGGGCGCCTCCTCCGCGGGTATGACCAGTACGCCTACGACGGCAAGGATTACATCGCCCTGAACGAGGACCTGCGCTCCTGGACCGCCGCGGACACCGCGGCTCAGATCACCCAGCGCAAGTGGGAGGCGGCCCGTGAGGCGGAGCAGCGGAGAGCCTACCTGGAGGGCACGTGCGTGGAGTGGCTCCGCAGATACCTGGAGAACGGGAAGGAGACGCTGCAGCGCGCGG"

Instance: CG-IG-HLA-FullBundle-01-22
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00413
* referenceSeq.referenceSeqId.text = "HLA-C*03:04:01:01"
* referenceSeq.windowStart = 486
* referenceSeq.windowEnd = 756
* observedSeq = "GCTCCCACTCCATGAGGTATTTCTACACCGCTGTGTCCCGGCCCGGCCGCGGGGAGCCCCACTTCATCGCAGTGGGCTACGTGGACGACACGCAGTTCGTGCGGTTCGACAGCGACGCCGCGAGTCCGAGAGGGGAGCCGCGGGCGCCGTGGGTGGAGCAGGAGGGGCCGGAGTATTGGGACCGGGAGACACAGAAGTACAAGCGCCAGGCACAGACTGACCGAGTGAGCCTGCGGAACCTGCGCGGCTACTACAACCAGAGCGAGGCCG"

Instance: CG-IG-HLA-FullBundle-01-23
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.referenceSeqId.coding.version = "3.23"
* referenceSeq.referenceSeqId.coding = $HLAALLELE#HLA00413
* referenceSeq.referenceSeqId.text = "HLA-C*03:04:01:01"
* referenceSeq.windowStart = 1001
* referenceSeq.windowEnd = 1277
* observedSeq = "GGTCTCACATCATCCAGAGGATGTATGGCTGCGACGTGGGGCCCGACGGGCGCCTCCTCCGCGGGTATGACCAGTACGCCTACGACGGCAAGGATTACATCGCCCTGAACGAGGATCTGCGCTCCTGGACCGCCGCGGACACGGCGGCTCAGATCACCCAGCGCAAGTGGGAGGCGGCCCGTGAGGCGGAGCAGCTGAGAGCCTACCTGGAGGGCCTGTGCGTGGAGTGGCTCCGCAGATACCTGAAGAATGGGAAGGAGACGCTGCAGCGCGCGG"

Instance: CG-IG-HLA-FullBundle-01-24
InstanceOf: Haplotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype name"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* effectiveDateTime = "2016-12-15"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "3.23"
* valueCodeableConcept.coding = $HLAALLELE#HLA-C*01:02:01G "HLA-C*01:02:01G"
* method = $GTR#GTR000000000.0
  * text = "NGS based Class I HLA-A, -B, -C genotyping"
* specimen = Reference(urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340) "buccal swab from John Storm"
* derivedFrom[0] = Reference(urn:uuid:bb55c2bc-5ad2-4bc1-8ff3-c407d06b12d0) "HLA-C*01:02:01, exon 2"
* derivedFrom[=].type = "MolecularSequence"
* derivedFrom[+] = Reference(urn:uuid:46938bb2-0486-4e87-bfd3-89aab2d5e22f) "HLA-C*01:02:01, exon 3"
  * type = "MolecularSequence"
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4933 "HLA-C"

Instance: CG-IG-HLA-FullBundle-01-25
InstanceOf: Haplotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype name"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* effectiveDateTime = "2016-12-15"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "3.23"
* valueCodeableConcept.coding = $HLAALLELE#HLA-C*01:02:01G "HLA-C*01:02:01G"
* method = $GTR#GTR000000000.0
  * text = "NGS based Class I HLA-A, -B, -C genotyping"
* specimen = Reference(urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340) "buccal swab from John Storm"
* derivedFrom[0] = Reference(urn:uuid:2ae2ff34-279e-43c2-9018-b054fd3fc1ce) "HLA-C*03:04:01:01, exon 2"
* derivedFrom[=].type = "MolecularSequence"
* derivedFrom[+] = Reference(urn:uuid:19153ef1-68c6-47a2-9676-c4eefbd39af9) "HLA-C*03:04:01:01, exon 3"
  * type = "MolecularSequence"
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4933 "HLA-C"

Instance: CG-IG-HLA-FullBundle-01-26
InstanceOf: Genotype
Usage: #inline
* basedOn = Reference(urn:uuid:99309303-045e-4cf4-90d7-250d7a7476ea) "Class I HLA genotyping for John Storm"
  * type = "ServiceRequest"
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84413-4 "Genotype display name"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* effectiveDateTime = "2016-12-15"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "1.0"
* valueCodeableConcept.coding = $GLSTRING#hla#3.23.0#HLA-C*01:02:01G+HLA-C*03:04:01G
* method = $GTR#GTR000000000.0
  * text = "NGS based Class I HLA-A, -B, -C genotyping"
* specimen = Reference(urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340) "buccal swab from John Storm"
* derivedFrom[0] = Reference(urn:uuid:8b2aa21c-1426-4717-8ab0-a84d83df7d47) "HLA-C*03:04:01G, exons 2 and 3"
* derivedFrom[=].type = "Observation"
* derivedFrom[+] = Reference(urn:uuid:709c5315-9403-4867-9d82-0b953836665f) "HLA-C*01:02:01G, exons 2 and 3"
  * type = "Observation"
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4933 "HLA-C"

Instance: CG-IG-HLA-FullBundle-01-27
InstanceOf: GenomicReport
Usage: #inline
* extension[0].url = "http://hl7.org/fhir/StructureDefinition/hla-genotyping-results-allele-database"
* extension[=].valueCodeableConcept.coding.system = "http://www.ebi.ac.uk/ipd/imgt/hla"
* extension[=].valueCodeableConcept.coding.version = "3.23"
* extension[+].extension[0].url = "text"
* extension[=].extension[=].valueString = "HLA-A*01:01:01G+HLA-A*01:02^HLA-B*15:01:01G+HLA-B*57:01:01G^HLA-C*01:02:01G+HLA-C*03:04:01G"
* extension[=].extension[+].url = "url"
* extension[=].extension[=].valueUri = "https://gl.nmdp.org/imgt-hla/3.23.0/multilocus-unphased-genotype/ex"
* extension[=].url = "http://hl7.org/fhir/StructureDefinition/hla-genotyping-results-glstring"
* basedOn = Reference(urn:uuid:99309303-045e-4cf4-90d7-250d7a7476ea) "Class I HLA genotyping for John Storm"
  * type = "ServiceRequest"
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE "Genetics"
* code.coding[0] = $LNC#51969-4 "Genetic analysis report"
* code.coding[+] = $HGNCID#HGNC:588 "Histocompatibility complex (HLA)"
* subject = Reference(urn:uuid:13f34265-335c-4853-bc38-0815315edafa) "John Storm"
  * type = "Patient"
* effectiveDateTime = "2016-12-15"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab, Inc"
* issued = "2016-12-15T14:15:30-06:00"
* performer = Reference(urn:uuid:9243cc20-27bd-4f87-ba90-0328ed474950) "aTypingLab Inc"
  * type = "Organization"
* specimen = Reference(urn:uuid:e44fbe33-6084-4ae2-a95e-8bc451c63340) "buccal swab from John Storm"
* result[0] = Reference(urn:uuid:49a86246-4004-42eb-9bdc-f542f93f9228) "HLA-A: HLA-A:01:01:01G+HLA-A*01:02"
* result[=].type = "Observation"
* result[+] = Reference(urn:uuid:60613a43-c4cb-4502-b3e2-cf9215feaa70) "HLA-B: HLA-B*15:01:01G+HLA-B*57:01:01G"
  * type = "Observation"
* result[+] = Reference(urn:uuid:0e0a780e-4486-4cd0-bfae-7243c579f208) "HLA-C: HLA-C*01:02:01G+HLA-C*03:04:01G"
  * type = "Observation"

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-cgexample-withGrouping.fsh

Instance: bundle-cgexample-withGrouping
InstanceOf: Bundle
Description: "Example bundle showing a variety of profiles, including how groupings of Observations might be delivered."
Usage: #example
* type = #collection
* entry[0].fullUrl = "http://example.org/fhir/DiagnosticReport/report-withGrouping"
* entry[=].resource = report-withGrouping
* entry[+].fullUrl = "http://example.org/fhir/Patient/ExamplePatient"
* entry[=].resource = ExamplePatient
* entry[+].fullUrl = "http://example.org/fhir/Specimen/ExampleSpecimen"
* entry[=].resource = ExampleSpecimen
* entry[+].fullUrl = "http://example.org/fhir/Organization/ExampleLab"
* entry[=].resource = ExampleLab
* entry[+].fullUrl = "http://example.org/fhir/ServiceRequest/ExampleServiceRequest"
* entry[=].resource = ExampleServiceRequest
* entry[+].fullUrl = "http://example.org/fhir/Observation/discrete-variant-1a"
* entry[=].resource = discrete-variant-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/dis-path-1a"
* entry[=].resource = dis-path-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/complex-variant-1a"
* entry[=].resource = complex-variant-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/complex-dis-path-1a"
* entry[=].resource = complex-dis-path-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/complex-component-D-1a"
* entry[=].resource = complex-component-D-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/complex-component-E-1a"
* entry[=].resource = complex-component-E-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/pharmPanel-1a"
* entry[=].resource = pharmPanel-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/haplotype-1a"
* entry[=].resource = haplotype-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/genotype-1a"
* entry[=].resource = genotype-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/metab-1a"
* entry[=].resource = metab-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/efficacy-1a"
* entry[=].resource = efficacy-1a
* entry[+].fullUrl = "http://example.org/fhir/Observation/highrisk-1a"
* entry[=].resource = highrisk-1a
* entry[+].fullUrl = "http://example.org/fhir/Task/usage-1a"
* entry[=].resource = usage-1a


Instance: report-withGrouping
InstanceOf: GenomicReport
Usage: #inline
* basedOn = Reference(ExampleServiceRequest)
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE
* code.coding = $LNC#51969-4 "Genetic analysis report"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* result[+] = Reference(discrete-variant-1a)
* result[+] = Reference(dis-path-1a)
* result[+] = Reference(complex-variant-1a)
* result[+] = Reference(complex-dis-path-1a)
* result[+] = Reference(pharmPanel-1a)
* result[+] = Reference(haplotype-1a)
* result[+] = Reference(genotype-1a)
* result[+] = Reference(metab-1a)
* result[+] = Reference(efficacy-1a)
* presentedForm.contentType = #text/plain
* presentedForm.data = "UHV0IHRoZSByZXBvcnQgdGV4dCBoZXJlIC0gbWltZS1lbmNvZGVk"
* conclusionCode[0] = $LNC#LA6576-8 "Positive"
* conclusionCode[+] = $SCT#10828004 "Positive"

Instance: dis-path-1a
InstanceOf: DiagnosticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(discrete-variant-1a)
* component[0].code = $LNC#53037-8
* component[=].valueCodeableConcept.coding = $LNC#LA6668-3 "Pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.coding = $MONDO#MONDO:0012624 "acyl-CoA dehydrogenase 9 deficiency"

Instance: complex-variant-1a
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* specimen = Reference(ExampleSpecimen)
* hasMember[0] = Reference(complex-component-D-1a)
* hasMember[+] = Reference(complex-component-E-1a)
/* extra slice
* component.code = $LNC#81263-6
* component.valueCodeableConcept = $LNC#LA26218-0 "Haplotype"
*/

Instance: complex-dis-path-1a
InstanceOf: DiagnosticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(complex-variant-1a)
* component[0].code = $LNC#53037-8
* component[=].valueCodeableConcept.coding = $LNC#LA6668-3 "Pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.coding = $SCT#293498008 "Debrisoquine adverse reaction (disorder)"

Instance: pharmPanel-1a
InstanceOf: Observation
Usage: #inline
* extension.url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/recommended-action"
* extension.valueReference = Reference(usage-1a)
* status = #final
* category[+].coding = $OBSCAT#laboratory
* category[+].coding = $DIAGNOSTICSERVICE#GE
* code = $NCIT#C43359 "Group"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* hasMember[0] = Reference(haplotype-1a)
* hasMember[+] = Reference(genotype-1a)
* hasMember[+] = Reference(metab-1a)
* hasMember[+] = Reference(efficacy-1a)

Instance: haplotype-1a
InstanceOf: Haplotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept.coding = $HLAALLELE#*2
* specimen = Reference(ExampleSpecimen)
* derivedFrom = Reference(discrete-variant-1a)

Instance: genotype-1a
InstanceOf: Genotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84413-4
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $PHARMVAR#CYP2C9*2*5" "CYP2C9 *2/*5"
* specimen = Reference(ExampleSpecimen)
* derivedFrom = Reference(haplotype-1a)
* component[0].code = $LNC#48018-6
* component[=].valueCodeableConcept.coding = $HGNCID#HGNC:2623 "CYP2C9"
* component[+].code = $LNC#48018-6
* component[=].valueCodeableConcept.coding = $HGNCID#HGNC:23663 "VKORC1"

Instance: metab-1a
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(genotype-1a)
* component[0].code = $LNC#51963-7
* component[=].valueCodeableConcept.coding = $RXN#11289 "Warfarin"
* component[+].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept.coding = $LNC#LA25390-8 "Rapid metabolizer"

Instance: efficacy-1a
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(genotype-1a)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept.coding = $LNC#LA6676-6 "Resistant"
* component[+].code = $LNC#51963-7
* component[=].valueCodeableConcept.coding = $RXN#11289 "Warfarin"

Instance: highrisk-1a
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(genotype-1a)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept.coding = $LNC#LA19542-2 "Low risk"
* component[+].code = $LNC#51963-7
* component[=].valueCodeableConcept.coding = $RXN#11289 "Warfarin"

Instance: usage-1a
InstanceOf: MedicationRecommendation
Usage: #inline
* status = #requested
* intent = #proposal
* code.coding = $LNC#LA26423-6 "Increase dose"
* code.text = "May need higher dosage than usual."
* focus = Reference(MedicationStatementWarfarin)
* for = Reference(ExamplePatient)
* requester = Reference(ExampleLab)
* reasonReference = Reference(metab-1a)

Instance: complex-component-D-1a
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept.coding = $LNC#LA9633-4 "Present"
* specimen = Reference(ExampleSpecimen)
* component[0].code = $LNC#81252-9 "Discrete genetic variant"
* component[=].valueCodeableConcept.coding = $CLINVAR#31934 "NM_000106.5(CYP2D6):c.886C>T (p.Arg296Cys)"
* component[+].code = $LNC#51958-7
* component[=].valueCodeableConcept.coding = $NCBIREFSEQ#NM_000106.5
* component[+].code = $LNC#48004-6
* component[=].valueCodeableConcept.coding = $HGVS#NM_000106.5:c.886C>T
* component[+].code = $LNC#48005-3
* component[=].valueCodeableConcept.coding = $HGVS#NP_000097.3:p.Arg296Cys
* component[+].code = $LNC#48019-4
* component[=].valueCodeableConcept.coding = $LNC#LA6690-7 "Substitution"

Instance: complex-component-E-1a
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* specimen = Reference(ExampleSpecimen)
* component[0].code = $LNC#81252-9 "Discrete genetic variant"
* component[=].valueCodeableConcept.coding = $CLINVAR#38486 "NM_000106.5(CYP2D6):c.1457G>C (p.Ser486Thr)"
* component[+].code = $LNC#51958-7
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_000106.5
* component[+].code = $LNC#48004-6
* component[=].valueCodeableConcept.coding = $HGVS#NM_000106.5:c.1457G>C

Instance: discrete-variant-1a
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept.coding = $LNC#LA9633-4 "Present"
* specimen = Reference(ExampleSpecimen)
* component[0].code = $LNC#81252-9 "Discrete genetic variant"
* component[=].valueCodeableConcept.coding = $CLINVAR#30880 "NM_014049.4(ACAD9):c.1249C>T (p.Arg417Cys)"
* component[+].code = $LNC#48018-6
* component[=].valueCodeableConcept.coding = $HGNCID#HGNC:21497 "ACAD9"
* component[+].code = $LNC#51958-7
* component[=].valueCodeableConcept.coding = $NCBIREFSEQ#NM_014049.4
* component[+].code = $LNC#48013-7
* component[=].valueCodeableConcept.coding = $NCBIREFSEQ#NG_017064.1
* component[+].code = $LNC#81290-9
* component[=].valueCodeableConcept.coding = $HGVS#NC_000003.11:g.128625063C>T
* component[+].code = $LNC#81252-9
* component[=].valueCodeableConcept.coding = $SNP#rs368949613
* component[+].code = $LNC#48004-6
* component[=].valueCodeableConcept.coding = $HGVS#NM_014049.4:c.1249C>T
* component[+].code = $LNC#48005-3
* component[=].valueCodeableConcept.coding = $HGVS#NP_054768.2:p.Arg417Cys
* component[+].code = $LNC#48019-4
* component[=].valueCodeableConcept.coding = $LNC#LA6690-7 "Substitution"
* component[+].code = $LNC#69547-8
* component[=].valueString = "C"
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 31731
* component[=].valueRange.high.value = 31731
* component[+].code = $LNC#69551-0
* component[=].valueString = "T"
* component[+].code = $LNC#48001-2
* component[=].valueCodeableConcept.text = "3q21"
* component[+].code = $LNC#48002-0
* component[=].valueCodeableConcept.coding = $LNC#LA6683-2 "Germline"
* component[+].code = $LNC#53034-5
* component[=].valueCodeableConcept.coding = $LNC#LA6706-1 "Heterozygous"
* component[+].code = $LNC#81258-6
* component[=].valueQuantity = 47 '%'
* component[+].code = $LNC#82121-5
* component[=].valueQuantity = 208 '1'
* component[+].code = $LNC#82155-3
* component[=].valueQuantity = 1 '1'
* component[+].code = $LNC#81301-4
* component[=].valueRange.low.value = 13200589
* component[=].valueRange.high.value = 15592000
* component[+].code = $LNC#81302-2
* component[=].valueRange.low.value = 14184616
* component[=].valueRange.high.value = 15581544
/* - extra slices
* component[+].code = $LNC#81299-0
* component[=].valueQuantity.value = 0.48
* component[+].code = $LNC#81300-6
* component[=].valueQuantity = 1396929 '1'
*/


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-cgexample.fsh

Instance: bundle-cgexample
InstanceOf: Bundle
Description: "Example bundle showing a variety of profiles."
Usage: #example
* type = #collection
* entry[0].fullUrl = "http://example.org/fhir/DiagnosticReport/report"
* entry[=].resource = report
* entry[+].fullUrl = "http://example.org/fhir/Patient/ExamplePatient"
* entry[=].resource = ExamplePatient
* entry[+].fullUrl = "http://example.org/fhir/Specimen/ExampleSpecimen"
* entry[=].resource = ExampleSpecimen
* entry[+].fullUrl = "http://example.org/fhir/Organization/ExampleLab"
* entry[=].resource = ExampleLab
* entry[+].fullUrl = "http://example.org/fhir/ServiceRequest/ExampleServiceRequest"
* entry[=].resource = ExampleServiceRequest
* entry[+].fullUrl = "http://example.org/fhir/Observation/discrete-variant"
* entry[=].resource = discrete-variant
* entry[+].fullUrl = "http://example.org/fhir/Observation/dis-path"
* entry[=].resource = dis-path
* entry[+].fullUrl = "http://example.org/fhir/Observation/complex-variant"
* entry[=].resource = complex-variant
* entry[+].fullUrl = "http://example.org/fhir/Observation/complex-dis-path"
* entry[=].resource = complex-dis-path
* entry[+].fullUrl = "http://example.org/fhir/Observation/complex-component-D"
* entry[=].resource = complex-component-D
* entry[+].fullUrl = "http://example.org/fhir/Observation/complex-component-E"
* entry[=].resource = complex-component-E
* entry[+].fullUrl = "http://example.org/fhir/Observation/haplotype"
* entry[=].resource = haplotype
* entry[+].fullUrl = "http://example.org/fhir/Observation/genotype"
* entry[=].resource = genotype
* entry[+].fullUrl = "http://example.org/fhir/Observation/metab"
* entry[=].resource = metab
* entry[+].fullUrl = "http://example.org/fhir/Observation/efficacy"
* entry[=].resource = efficacy
* entry[+].fullUrl = "http://example.org/fhir/Observation/highrisk"
* entry[=].resource = highrisk
* entry[+].fullUrl = "http://example.org/fhir/Task/usage"
* entry[=].resource = usage

Instance: report
InstanceOf: GenomicReport
Usage: #inline
* basedOn = Reference(ExampleServiceRequest)
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE
* code.coding = $LNC#51969-4 
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* result[+] = Reference(discrete-variant)
* result[+] = Reference(dis-path)
* result[+] = Reference(complex-variant)
* result[+] = Reference(complex-dis-path)
* result[+] = Reference(haplotype)
* result[+] = Reference(genotype)
* result[+] = Reference(metab)
* result[+] = Reference(efficacy)
* presentedForm.contentType = #text/plain
* presentedForm.data = "UHV0IHRoZSByZXBvcnQgdGV4dCBoZXJlIC0gbWltZS1lbmNvZGVk"
* conclusionCode[+] = $LNC#LA6576-8 "Positive"
* conclusionCode[+] = $SCT#10828004 "Positive"

Instance: discrete-variant
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept.coding = $LNC#LA9633-4 "Present"
* specimen = Reference(ExampleSpecimen)
* component[0].code = $LNC#81252-9 "Discrete genetic variant"
* component[=].valueCodeableConcept.coding = $CLINVAR#30880 "NM_014049.4(ACAD9):c.1249C>T (p.Arg417Cys)"
* component[+].code = $LNC#48018-6
* component[=].valueCodeableConcept.coding = $HGNCID#HGNC:21497 "ACAD9"
* component[+].code = $LNC#51958-7
* component[=].valueCodeableConcept.coding = $NCBIREFSEQ#NM_014049.4
* component[+].code = $LNC#48013-7
* component[=].valueCodeableConcept.coding = $NCBIREFSEQ#NG_017064.1
* component[+].code = $LNC#81290-9
* component[=].valueCodeableConcept.coding = $HGVS#NC_000003.11:g.128625063C>T
* component[+].code = $LNC#81252-9
* component[=].valueCodeableConcept.coding.version = "137"
* component[=].valueCodeableConcept.coding = $SNP#rs368949613
* component[+].code = $LNC#48004-6
* component[=].valueCodeableConcept.coding = $HGVS#NM_014049.4:c.1249C>T
* component[+].code = $LNC#48005-3
* component[=].valueCodeableConcept.coding = $HGVS#NP_054768.2:p.Arg417Cys
* component[+].code = $LNC#48019-4
* component[=].valueCodeableConcept.coding = $LNC#LA6690-7 "Substitution"
* component[+].code = $LNC#69547-8
* component[=].valueString = "C"
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 31731
* component[=].valueRange.high.value = 31731
* component[+].code = $LNC#69551-0
* component[=].valueString = "T"
* component[+].code = $LNC#48001-2
* component[=].valueCodeableConcept.text = "3q21"
* component[+].code = $LNC#48002-0
* component[=].valueCodeableConcept.coding = $LNC#LA6683-2 "Germline"
* component[+].code = $LNC#53034-5
* component[=].valueCodeableConcept.coding = $LNC#LA6706-1 "Heterozygous"
* component[+].code = $LNC#81258-6
* component[=].valueQuantity = 47 '%'
* component[+].code = $LNC#82121-5
* component[=].valueQuantity = 208 '1'
* component[+].code = $LNC#82155-3
* component[=].valueQuantity = 1 '1'
* component[+].code.coding = $LNC#81299-0
* component[=].valueQuantity.value = 0.48
* component[+].code = $LNC#81300-6
* component[=].valueQuantity = 1396929 '1'
* component[+].code = $LNC#81301-4
* component[=].valueRange.low.value = 13200589
* component[=].valueRange.high.value = 15592000
* component[+].code = $LNC#81302-2
* component[=].valueRange.low.value = 14184616
* component[=].valueRange.high.value = 15581544

Instance: dis-path
InstanceOf: DiagnosticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(discrete-variant)
* component[0].code = $LNC#53037-8
* component[=].valueCodeableConcept.coding = $LNC#LA6668-3 "Pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.coding = $MONDO#MONDO:0012624 "acyl-CoA dehydrogenase 9 deficiency"

Instance: complex-variant
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* specimen = Reference(ExampleSpecimen)
* hasMember[0] = Reference(complex-component-D)
* hasMember[+] = Reference(complex-component-E)
/* - example of open slicing - commenting this out as it causes numerous other cascading Slicing Information issues (everywhere this resource is referenced)
* component.code = $LNC#81263-6
* component.valueCodeableConcept = $LNC#LA26218-0 "Haplotype"
*/

Instance: complex-dis-path
InstanceOf: DiagnosticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(complex-variant)
* component[0].code = $LNC#53037-8
* component[=].valueCodeableConcept.coding = $LNC#LA6668-3 "Pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.coding = $SCT#293498008 "Debrisoquine adverse reaction (disorder)"

Instance: complex-component-D
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept.coding = $LNC#LA9633-4 "Present"
* specimen = Reference(ExampleSpecimen)
* component[0].code = $LNC#81252-9 "Discrete genetic variant"
* component[=].valueCodeableConcept.coding = $CLINVAR#31934 "NM_000106.5(CYP2D6):c.886C>T (p.Arg296Cys)"
* component[+].code = $LNC#51958-7
* component[=].valueCodeableConcept.coding = $NCBIREFSEQ#NM_000106.5
* component[+].code = $LNC#48004-6
* component[=].valueCodeableConcept.coding = $HGVS#NM_000106.5:c.886C>T
* component[+].code = $LNC#48005-3
* component[=].valueCodeableConcept.coding = $HGVS#NP_000097.3:p.Arg296Cys
* component[+].code = $LNC#48019-4
* component[=].valueCodeableConcept.coding = $LNC#LA6690-7 "Substitution"

Instance: complex-component-E
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* specimen = Reference(ExampleSpecimen)
* component[0].code = $LNC#81252-9 "Discrete genetic variant"
* component[=].valueCodeableConcept.coding.version = "???"
* component[=].valueCodeableConcept.coding = $CLINVAR#38486 "NM_000106.5(CYP2D6):c.1457G>C (p.Ser486Thr)"
* component[+].code = $LNC#51958-7
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_000106.5
* component[+].code = $LNC#48004-6
* component[=].valueCodeableConcept.coding = $HGVS#NM_000106.5:c.1457G>C

Instance: haplotype
InstanceOf: Haplotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept.coding = $HLAALLELE#*2
* specimen = Reference(ExampleSpecimen)
* derivedFrom = Reference(discrete-variant)

Instance: genotype
InstanceOf: Genotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84413-4
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $PHARMVAR#"CYP2C9 *2/*5" "CYP2C9 *2/*5"
* specimen = Reference(ExampleSpecimen)
* derivedFrom = Reference(haplotype)
* component[0].code = $LNC#48018-6
* component[=].valueCodeableConcept.coding = $HGNCID#HGNC:2623 "CYP2C9"
* component[+].code = $LNC#48018-6
* component[=].valueCodeableConcept.coding = $HGNCID#HGNC:23663 "VKORC1"

Instance: metab
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(genotype)
* component[0].code = $LNC#51963-7
* component[=].valueCodeableConcept.coding = $RXN#11289 "Warfarin"
* component[+].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept.coding = $LNC#LA25390-8 "Rapid metabolizer"

Instance: efficacy
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(genotype)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept.coding = $LNC#LA6676-6 "Resistant"
* component[+].code = $LNC#51963-7
* component[=].valueCodeableConcept.coding = $RXN#11289 "Warfarin"

Instance: highrisk
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2016-09-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(genotype)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept.coding = $LNC#LA19542-2 "Low risk"
* component[+].code = $LNC#51963-7
* component[=].valueCodeableConcept.coding = $RXN#11289 "Warfarin"

Instance: usage
InstanceOf: MedicationRecommendation
Usage: #inline
* status = #requested
* intent = #proposal
* code.coding = $LNC#LA26423-6 "Increase dose"
* code.text = "May need higher dosage than usual."
* focus = Reference(MedicationStatementWarfarin)
* for = Reference(ExamplePatient)
* requester = Reference(ExampleLab)
* reasonReference = Reference(metab)



---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-complexVariant-nonHGVS.fsh

Instance: bundle-complexVariant-nonHGVS
InstanceOf: Bundle
Description: "Example bundle with compound heterozygous variant, where the specific changes are represented without HGVS."
Usage: #example
* type = #transaction
* entry[0].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d"
* entry[=].resource = Inline-Instance-for-complexVariant-nonHGVS-1
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12e"
* entry[=].resource = Inline-Instance-for-complexVariant-nonHGVS-2
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12a"
* entry[=].resource = Inline-Instance-for-complexVariant-nonHGVS-3
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab"
* entry[=].resource = performingLab
* entry[=].request.method = #POST
* entry[=].request.url = "Organization"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac"
* entry[=].resource = patient
* entry[=].request.method = #POST
* entry[=].request.url = "Patient"


Instance: Inline-Instance-for-complexVariant-nonHGVS-1
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#51958-7 "Transcript ref sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_022787.3 "NM_022787.3"
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 769
* component[=].valueRange.high.value = 769
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Genomic alt allele [ID]"
* component[=].valueString = "A"
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "Heterozygous"


Instance: Inline-Instance-for-complexVariant-nonHGVS-2
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#51958-7 "Transcript ref sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_022787.3 "NM_022787.3"
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 53
* component[=].valueRange.high.value = 53
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Genomic alt allele [ID]"
* component[=].valueString = "G"
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "Heterozygous"

Instance: Inline-Instance-for-complexVariant-nonHGVS-3
InstanceOf: Genotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84413-4
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $CLINVAR#RCV000664188.1 "NM_022787.3(NMNAT1):c.[53A>G];[769G>A] AND Cone-rod dystrophy"
* hasMember[0] = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12e) "unpacked representation of NM_022787.3:c.53A>G"
* hasMember[+] = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d) "unpacked representation of NM_022787.3:c.769G>A"


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-compound-heterozygote.fsh

Instance: bundle-compound-heterozygote
InstanceOf: Bundle
Description: "Example bundle with compound heterozygous variant, where the specific changes are represented with HGVS."
Usage: #example
* type = #transaction
* entry[0].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d"
* entry[=].resource = Inline-Instance-for-compound-heterozygote-1
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12e"
* entry[=].resource = Inline-Instance-for-compound-heterozygote-2
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12a"
* entry[=].resource = Inline-Instance-for-compound-heterozygote-3
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab"
* entry[=].resource = performingLab
* entry[=].request.method = #POST
* entry[=].request.url = "Organization"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac"
* entry[=].resource = patient
* entry[=].request.method = #POST
* entry[=].request.url = "Patient"

Instance: Inline-Instance-for-compound-heterozygote-1
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_022787.3:c.769G>A "NM_022787.3:c.769G>A"
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "Heterozygous"


Instance: Inline-Instance-for-compound-heterozygote-2
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* code = $LNC#69548-6 "Genetic variant assessment"
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_022787.3:c.53A>G "NM_022787.3:c.53A>G"
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "Heterozygous"

Instance: Inline-Instance-for-compound-heterozygote-3
InstanceOf: Genotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* code = $LNC#84413-4
* valueCodeableConcept = $CLINVAR#RCV000664188.1 "NM_022787.3(NMNAT1):c.[53A>G];[769G>A] AND Cone-rod dystrophy"
* hasMember[0] = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12e) "NM_022787.3:c.53A>G"
* hasMember[+] = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d) "NM_022787.3:c.769G>A"


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-CYP2C19.fsh

Instance: bundle-CYP2C19
InstanceOf: Bundle
Description: "Example bundle containing CYP2C19 report and variants"
Usage: #example
* type = #transaction
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe01"
* entry[=].resource = dv-5c7e9e1ca62b5
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe02"
* entry[=].resource = dv-5c7e9e1ca6342
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe03"
* entry[=].resource = dv-5c7e9e1ca63bf
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe04"
* entry[=].resource = dv-5c7e9e1ca643a
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe05"
* entry[=].resource = dv-5c7e9e1ca64b5
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe06"
* entry[=].resource = dv-5c7e9e1ca6531
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe07"
* entry[=].resource = dv-5c7e9e1ca65ac
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe08"
* entry[=].resource = dv-5c7e9e1ca6627
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe09"
* entry[=].resource = dv-5c7e9e1ca66a1
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe10"
* entry[=].resource = dv-5c7e9e1ca671c
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe11"
* entry[=].resource = dv-5c7e9e1ca6797
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe12"
* entry[=].resource = dv-5c7e9e1ca6812
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe13"
* entry[=].resource = dv-5c7e9e1ca688c
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe14"
* entry[=].resource = dv-5c7e9e1ca6922
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe15"
* entry[=].resource = dv-5c7e9e1ca69a3
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe16"
* entry[=].resource = dv-5c7e9e1ca6a1d
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe17"
* entry[=].resource = dv-5c7e9e1ca6a97
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe18"
* entry[=].resource = dv-5c7e9e1ca6b11
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe19"
* entry[=].resource = dv-5c7e9e1ca6b86
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe20"
* entry[=].resource = dv-5c7e9e1ca6c31
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe21"
* entry[=].resource = dv-5c7e9e1ca6ccc
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe22"
* entry[=].resource = dv-5c7e9e1ca6d4b
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe23"
* entry[=].resource = dv-5c7e9e1ca6de3
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe24"
* entry[=].resource = dv-5c7e9e1ca6e86
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe25"
* entry[=].resource = dv-5c7e9e1ca6f11
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe26"
* entry[=].resource = dv-5c7e9e1ca6fa5
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe27"
* entry[=].resource = dv-5c7e9e1ca7025
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe28"
* entry[=].resource = dv-5c7e9e1ca70a2
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe29"
* entry[=].resource = dv-5c7e9e1ca7141
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe30"
* entry[=].resource = dv-5c7e9e1ca71bf
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe31"
* entry[=].resource = dv-5c7e9e1ca724a
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe32"
* entry[=].resource = dv-5c7e9e1ca72e4
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe33"
* entry[=].resource = dv-5c7e9e1ca7377
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe34"
* entry[=].resource = dv-5c7e9e1ca740a
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe35"
* entry[=].resource = dv-5c7e9e1ca7497
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe36"
* entry[=].resource = dv-5c7e9e1ca7513
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe37"
* entry[=].resource = dv-5c7e9e1ca758e
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe38"
* entry[=].resource = dv-5c7e9e1ca7609
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe39"
* entry[=].resource = dv-5c7e9e1ca7684
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe40"
* entry[=].resource = dv-5c7e9e1ca76ff
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe41"
* entry[=].resource = dv-5c7e9e1ca7779
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe42"
* entry[=].resource = dv-5c7e9e1ca77f4
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe43"
* entry[=].resource = dv-5c7e9e1ca786f
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe44"
* entry[=].resource = dv-5c7e9e1ca78eb
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe45"
* entry[=].resource = dv-5c7e9e1ca7967
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe46"
* entry[=].resource = dv-5c7e9e1ca79ef
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe47"
* entry[=].resource = dv-5c7e9e1ca7a79
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe48"
* entry[=].resource = dv-5c7e9e1ca7afb
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe49"
* entry[=].resource = dv-5c7e9e1ca7bcb
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe50"
* entry[=].resource = dv-5c7e9e1ca7c49
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe51"
* entry[=].resource = dv-5c7e9e1ca7cd3
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe52"
* entry[=].resource = dv-5c7e9e1ca7d94
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe53"
* entry[=].resource = dv-5c7e9e1ca7e45
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe54"
* entry[=].resource = dv-5c7e9e1ca7ed6
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe55"
* entry[=].resource = dv-5c7e9e1ca7f69
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe56"
* entry[=].resource = dv-5c7e9e1ca7ffd
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe57"
* entry[=].resource = dv-5c7e9e1ca8085
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe58"
* entry[=].resource = dv-5c7e9e1ca811f
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe59"
* entry[=].resource = dv-5c7e9e1ca81b9
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe60"
* entry[=].resource = dv-5c7e9e1ca8252
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:20ac0aeb-6bd4-4e92-a891-d44a807bfe01"
* entry[=].resource = dv-5c7e9e1ca82e4
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe62"
* entry[=].resource = dv-5c7e9e1ca837c
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe63"
* entry[=].resource = dv-5c7e9e1ca8410
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe64"
* entry[=].resource = dv-5c7e9e1ca8493
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe65"
* entry[=].resource = dv-5c7e9e1ca8514
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe66"
* entry[=].resource = dv-5c7e9e1ca8596
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe67"
* entry[=].resource = dv-5c7e9e1ca8618
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe68"
* entry[=].resource = dv-5c7e9e1ca869c
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe69"
* entry[=].resource = dv-5c7e9e1ca871e
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe70"
* entry[=].resource = dv-5c7e9e1ca879f
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe71"
* entry[=].resource = dv-5c7e9e1ca8820
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe72"
* entry[=].resource = dv-5c7e9e1ca88a5
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe73"
* entry[=].resource = dv-5c7e9e1ca8927
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe74"
* entry[=].resource = dv-5c7e9e1ca89a8
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe75"
* entry[=].resource = sid-5c7e9e1ca8a29
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeaa"
* entry[=].resource = report-CYP2C19
* entry[=].request.method = #POST
* entry[=].request.url = "DiagnosticReport"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab"
* entry[=].resource = performingLab
* entry[=].request.method = #POST
* entry[=].request.url = "Organization"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac"
* entry[=].resource = patient
* entry[=].request.method = #POST
* entry[=].request.url = "Patient"

Instance: patient
InstanceOf: Patient
Usage: #inline
* name[0].use = #official
* name[=].text = "John Storm"
* name[=].family = "Storm"
* name[=].given = "John"
* name[+].use = #nickname
* name[=].text = "Johnny Storm"
* name[=].family = "Storm"
* name[=].given = "Johnny"
* name[+].use = #nickname
* name[=].text = "The Human Torch"
* gender = #male
* birthDate = "1986-12-31"

Instance: performingLab
InstanceOf: Organization
Usage: #inline
* name = "aTypingLab Inc"
* alias = "aTL"
* telecom.system = #phone
* telecom.value = "1-800-555-1234"
* telecom.use = #work
* telecom.rank = 1
* address.use = #work
* address.type = #both
* address.text = "123 Main St, Sometown, ND 99999"
* address.line = "123 Main St"
* address.city = "Sometown"
* address.state = "ND"
* address.postalCode = "99999"
* address.country = "USA"

Instance: report-CYP2C19
InstanceOf: GenomicReport
Usage: #inline
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE
* code = $LNC#51969-4
  * text = "Genetic analysis report"
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe01)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe02)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe03)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe04)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe05)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe06)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe07)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe08)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe09)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe10)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe11)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe12)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe13)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe14)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe15)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe16)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe17)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe18)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe19)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe20)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe21)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe22)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe23)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe24)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe25)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe26)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe27)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe28)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe29)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe30)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe31)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe32)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe33)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe34)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe35)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe36)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe37)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe38)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe39)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe40)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe41)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe42)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe43)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe44)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe45)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe46)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe47)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe48)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe49)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe50)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe51)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe52)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe53)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe54)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe55)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe56)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe57)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe58)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe59)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe60)
* result[+] = Reference(urn:uuid:20ac0aeb-6bd4-4e92-a891-d44a807bfe01)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe62)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe63)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe64)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe65)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe66)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe67)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe68)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe69)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe70)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe71)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe72)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe73)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe74)
* result[+] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe75)



Instance: dv-5c7e9e1ca62b5
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96448129

Instance: dv-5c7e9e1ca6342
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96448380

Instance: dv-5c7e9e1ca63bf
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "C"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96448994

Instance: dv-5c7e9e1ca643a
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96454156

Instance: dv-5c7e9e1ca64b5
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96454241

Instance: dv-5c7e9e1ca6531
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96456161

Instance: dv-5c7e9e1ca65ac
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96462109

Instance: dv-5c7e9e1ca6627
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96464087

Instance: dv-5c7e9e1ca66a1
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "C"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96465938

Instance: dv-5c7e9e1ca671c
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96466078

Instance: dv-5c7e9e1ca6797
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96466342

Instance: dv-5c7e9e1ca6812
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96466824

Instance: dv-5c7e9e1ca688c
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "TATA"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96466884

Instance: dv-5c7e9e1ca6922
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "C"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96468808

Instance: dv-5c7e9e1ca69a3
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96469632

Instance: dv-5c7e9e1ca6a1d
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96469669

Instance: dv-5c7e9e1ca6a97
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "C"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96470954

Instance: dv-5c7e9e1ca6b11
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96475154

Instance: dv-5c7e9e1ca6b86
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "C"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96476401

Instance: dv-5c7e9e1ca6c31
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96477558

Instance: dv-5c7e9e1ca6ccc
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96478608

Instance: dv-5c7e9e1ca6d4b
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96480890

Instance: dv-5c7e9e1ca6de3
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "GT"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96484540

Instance: dv-5c7e9e1ca6e86
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96484599

Instance: dv-5c7e9e1ca6f11
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96484998

Instance: dv-5c7e9e1ca6fa5
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96485680

Instance: dv-5c7e9e1ca7025
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96486504

Instance: dv-5c7e9e1ca70a2
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "CAA"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96487380

Instance: dv-5c7e9e1ca7141
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96487451

Instance: dv-5c7e9e1ca71bf
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96487749

Instance: dv-5c7e9e1ca724a
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96488888

Instance: dv-5c7e9e1ca72e4
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96490629

Instance: dv-5c7e9e1ca7377
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "C"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96491109

Instance: dv-5c7e9e1ca740a
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96491772

Instance: dv-5c7e9e1ca7497
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96493358

Instance: dv-5c7e9e1ca7513
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96493709

Instance: dv-5c7e9e1ca758e
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96495232

Instance: dv-5c7e9e1ca7609
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96499931

Instance: dv-5c7e9e1ca7684
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96500106

Instance: dv-5c7e9e1ca76ff
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96503400

Instance: dv-5c7e9e1ca7779
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96504557

Instance: dv-5c7e9e1ca77f4
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "TA"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96504737

Instance: dv-5c7e9e1ca786f
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "C"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96506173

Instance: dv-5c7e9e1ca78eb
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96509779

Instance: dv-5c7e9e1ca7967
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96511926

Instance: dv-5c7e9e1ca79ef
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "GA"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96512422

Instance: dv-5c7e9e1ca7a79
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "CA"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96513643

Instance: dv-5c7e9e1ca7afb
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96515933

Instance: dv-5c7e9e1ca7bcb
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96516778

Instance: dv-5c7e9e1ca7c49
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96517909

Instance: dv-5c7e9e1ca7cd3
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96517959

Instance: dv-5c7e9e1ca7d94
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96519061

Instance: dv-5c7e9e1ca7e45
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96521422

Instance: dv-5c7e9e1ca7ed6
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96521657

Instance: dv-5c7e9e1ca7f69
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96522989

Instance: dv-5c7e9e1ca7ffd
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96525536

Instance: dv-5c7e9e1ca8085
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "TTGTA"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96526061

Instance: dv-5c7e9e1ca811f
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96526109

Instance: dv-5c7e9e1ca81b9
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "AT"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96527152

Instance: dv-5c7e9e1ca8252
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96527334

Instance: dv-5c7e9e1ca82e4
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96527722

Instance: dv-5c7e9e1ca837c
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6705-3 "homozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96529912

Instance: dv-5c7e9e1ca8410
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96530504

Instance: dv-5c7e9e1ca8493
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "C"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96532327

Instance: dv-5c7e9e1ca8514
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96532545

Instance: dv-5c7e9e1ca8596
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96532688

Instance: dv-5c7e9e1ca8618
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96533050

Instance: dv-5c7e9e1ca869c
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "G"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96534584

Instance: dv-5c7e9e1ca871e
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "C"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96535689

Instance: dv-5c7e9e1ca879f
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "A"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96535743

Instance: dv-5c7e9e1ca8820
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "ATTC"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "TATT"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96536045

Instance: dv-5c7e9e1ca88a5
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "A"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96536455

Instance: dv-5c7e9e1ca8927
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96536708

Instance: dv-5c7e9e1ca89a8
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NC_000010.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "heterozygous"
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "C"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "T"
* component[+].code = $LNC#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = $LNC#LA30102-0
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 96537137

Instance: sid-5c7e9e1ca8a29
InstanceOf: SequencePhaseRelationship
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#82120-7 "Allelic phase"
* subject = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac)
* effectiveDateTime = "2019-03-05"
* issued = "2019-03-05T16:04:44+00:00"
* performer = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab)
* valueCodeableConcept = SequencePhaseRelationshipCS#Cis "Cis"
* derivedFrom[0] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe60)
* derivedFrom[+] = Reference(urn:uuid:20ac0aeb-6bd4-4e92-a891-d44a807bfe01)


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-oncology-diagnostic.fsh

Instance: bundle-oncology-diagnostic
InstanceOf: Bundle
Description: "Example of an oncology report, TMB, variants, and implications."
Usage: #example
* type = #transaction
* entry[0].fullUrl = "urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d"
* entry[=].resource = Inline-Instance-for-oncology-diagnostic-1
* entry[=].request.method = #POST
* entry[=].request.url = "Patient"
* entry[+].fullUrl = "urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef"
* entry[=].resource = Inline-Instance-for-oncology-diagnostic-2
* entry[=].request.method = #POST
* entry[=].request.url = "Practitioner"
* entry[+].fullUrl = "urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d"
* entry[=].resource = Inline-Instance-for-oncology-diagnostic-3
* entry[=].request.method = #POST
* entry[=].request.url = "Specimen"
* entry[+].fullUrl = "urn:uuid:40e85eda-4d31-4bd2-a5a0-c39d63f9824f"
* entry[=].resource = Inline-Instance-for-oncology-diagnostic-4
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d"
* entry[=].resource = Inline-Instance-for-oncology-diagnostic-5
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:1bf9642d-fd1c-4819-8c5d-44fae4bad524"
* entry[=].resource = Inline-Instance-for-oncology-diagnostic-6
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:ad7a499b-a43b-4cc8-bf46-cac62d9ecae5"
* entry[=].resource = Inline-Instance-for-oncology-diagnostic-7
* entry[=].request.method = #POST
* entry[=].request.url = "DiagnosticReport"

Instance: Inline-Instance-for-oncology-diagnostic-1
InstanceOf: Patient
Usage: #inline
* identifier.value = "1007"
* name.given[0] = "Carrot"
* name.given[+] = "John"
* name.given[+] = "Mr"
* gender = #male
* birthDate = "1962-12-31"
* generalPractitioner = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)

Instance: Inline-Instance-for-oncology-diagnostic-2
InstanceOf: Practitioner
Usage: #inline
* identifier.value = "4654765876"
* name.given[0] = "Akerman"
* name.given[+] = "Philip"
* name.given[+] = "Prof"
* address.city = "Castro Valley"
* address.state = "CA"

Instance: Inline-Instance-for-oncology-diagnostic-3
InstanceOf: Specimen
Usage: #inline
* identifier.value = "076e1948-2217-11e9-94d8-12e241dc1f66"
* status = #available
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* collection.collector = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)

Instance: Inline-Instance-for-oncology-diagnostic-4
InstanceOf: MolecularBiomarker
Usage: #inline
* status = #final
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-biomarker"
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#94076-7 "Mutations/Megabase [# Ratio] in Tumor"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2019-03-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueQuantity = 25.0 '1/1000000' "Mutations/Megabase"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[biomarker-category].code = TbdCodesCS#biomarker-category
* component[biomarker-category].valueCodeableConcept.coding = MolecularBiomarkerOntologyCS#molgen "molecular sequence adjacent category"

Instance: Inline-Instance-for-oncology-diagnostic-5
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2019-03-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[0].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#81290-9 "Genomic DNA change (gHGVS)"
* component[=].valueCodeableConcept = $HGVS#NC_000007.13:g.140453136A>T "NC_000007.13:g.140453136A>T"
* component[+].code = $LNC#81252-9 "Discrete genetic variant"
* component[=].valueCodeableConcept = $CLINVAR#13961 "Clinvar ID 13961"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_004324.2:p.Val600Glu "NP_004324.2:p.Val600Glu"
* component[+].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept = $HGNCID#HGNC:1097 "BRAF"

Instance: Inline-Instance-for-oncology-diagnostic-6
InstanceOf: DiagnosticImplication
Usage: #inline
* extension[workflow-relatedArtifact].valueRelatedArtifact.type = #justification
* extension[workflow-relatedArtifact].valueRelatedArtifact.display = "https://pmkb.weill.cornell.edu/therapies/103"
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2019-03-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* method = $LNC#LA26811-2 "Computational analysis"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* derivedFrom = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA6668-3 "Pathogenic"
* component[+].code = $LNC#93044-6 "Level of Evidence"
* component[=].valueCodeableConcept = $LNC#LA30200-2 "Very strong evidence pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.text = "melanoma"

Instance: Inline-Instance-for-oncology-diagnostic-7
InstanceOf: GenomicReport
Usage: #inline
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE
* code = $LNC#51969-4
  * text = "Genetic analysis report"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2019-03-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* result[0] = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d) "BRAF V600E mutation observed"
* result[+] = Reference(urn:uuid:1bf9642d-fd1c-4819-8c5d-44fae4bad524) "melanoma implicated"


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-oncology-report-example.fsh

Instance: bundle-oncology-report-example
InstanceOf: Bundle
Description: "Example oncology bundle including report, variants, and other resources."
Usage: #example
* type = #transaction
* entry[0].fullUrl = "urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17"
* entry[=].resource = Inline-Instance-for-oncology-report-example-1
* entry[=].request.method = #POST
* entry[=].request.url = "Organization"
* entry[=].request.ifNoneExist = "identifier=http://molit.eu/fhir/genomics/NamingSystem/organization|CEGAT"
* entry[+].fullUrl = "urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648"
* entry[=].resource = Inline-Instance-for-oncology-report-example-2
* entry[=].request.method = #POST
* entry[=].request.url = "Patient"
* entry[=].request.ifNoneExist = "identifier=http://molit.eu/fhir/genomics/NamingSystem/cegat/patID|11111"
* entry[+].fullUrl = "urn:uuid:a2041c83-b73d-4fc8-9466-4ba4a92da516"
* entry[=].resource = Inline-Instance-for-oncology-report-example-3
* entry[=].request.method = #POST
* entry[=].request.url = "Specimen"
* entry[=].request.ifNoneExist = "identifier=http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID|UNKNOWN"
* entry[+].fullUrl = "urn:uuid:dac358c3-403a-4dbb-b478-4259aed882ae"
* entry[=].resource = Inline-Instance-for-oncology-report-example-4
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:1d773d66-cec7-44a2-b92a-46d00adeae00"
* entry[=].resource = Inline-Instance-for-oncology-report-example-5
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:842d9ab9-d940-4f0c-adf9-e5c528f5c0e5"
* entry[=].resource = Inline-Instance-for-oncology-report-example-6
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:9a9f9a4a-52e3-4738-bd0b-a25374bbf358"
* entry[=].resource = Inline-Instance-for-oncology-report-example-7
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:58828523-8893-45fc-973b-16290366c5e5"
* entry[=].resource = Inline-Instance-for-oncology-report-example-8
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:2c28b23f-3e9f-4c03-8c8f-0e76bc5dc9c2"
* entry[=].resource = Inline-Instance-for-oncology-report-example-9
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:41bebbe5-e06f-4867-aa22-7c06db69dbd1"
* entry[=].resource = Inline-Instance-for-oncology-report-example-10
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:1642f190-e2c6-4999-8040-b9b2a70618bf"
* entry[=].resource = Inline-Instance-for-oncology-report-example-11
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:c3587931-242f-4129-93f9-be24500c8f29"
* entry[=].resource = Inline-Instance-for-oncology-report-example-12
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:41695fc0-1fd5-4cc8-95e6-82b2848a5cb6"
* entry[=].resource = Inline-Instance-for-oncology-report-example-13
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:58eb14f6-4059-4168-86a9-155ae61d30e2"
* entry[=].resource = Inline-Instance-for-oncology-report-example-14
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:1a71e80f-b044-4a91-80e1-eadbe5a53dca"
* entry[=].resource = Inline-Instance-for-oncology-report-example-15
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:6a80003f-822d-489e-8286-1f1dcba56dfa"
* entry[=].resource = Inline-Instance-for-oncology-report-example-16
* entry[=].request.method = #POST
* entry[=].request.url = "DiagnosticReport"

Instance: Inline-Instance-for-oncology-report-example-1
InstanceOf: Organization
Usage: #inline
* identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/organization"
* identifier.value = "CEGAT"
* name = "CEGAT"

Instance: Inline-Instance-for-oncology-report-example-10
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:1787 "CDKN2A"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_000077.4:c.9_32del
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:0000159 "deletion"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_000068.1:p.Ala4_Pro11del
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_000077.4 "NM_000077.4"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "AGGCTCCATGCTGCTCCCCGCCGCC"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.0536 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 112 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-11
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:9949 "RECQL4"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_004260.4:c.2086C>T 
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:1000002 "substitution"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_004251.4:p.Arg696Cys 
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_004260.4 "NM_004260.4"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "G"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.2568 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 148 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-12
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:10483 "RYR1"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_000540.3:c.4964G>A
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:1000002 "substitution"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_000531.2:p.Arg1655Leu
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_000540.2 "NM_000540.3"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "G"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.2151 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 93 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-13
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:10519 "SACS"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_014363.5:c.12118G>A
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:1000002 "substitution"
//KP - unable to validate what the right p. is
//* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
//* component[=].valueCodeableConcept.text = "p.A4040T" //$HGVS#p.A4040T
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_014363.5 "NM_014363.5"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "C"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.3333 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 60 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-14
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:11086 "SLIT2"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_004787.3:c.1290C>A
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:1000002 "substitution"
//kp - unable to validate p.
//* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
//* component[=].valueCodeableConcept.text = "p.N430K" //$HGVS#p.N430K
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_004787.3 "NM_004787.3"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "C"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.2642 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 53 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-15
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:11100 "SMARCA4"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_003072.5:c.2372C>T
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:1000002 "substitution"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_003063.2:p.Ala791Val
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_003072.5 "NM_003072.5"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "C"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.1938 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 160 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-16
InstanceOf: GenomicReport
Usage: #inline
* identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/reportID"
* identifier.value = "42867"
* status = #final
* code = $LNC#51969-4 "Genetic analysis report"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* issued = "2019-09-15T11:35:05.722-04:00"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* specimen = Reference(urn:uuid:a2041c83-b73d-4fc8-9466-4ba4a92da516)
* result[0] = Reference(urn:uuid:dac358c3-403a-4dbb-b478-4259aed882ae)
* result[+] = Reference(urn:uuid:1d773d66-cec7-44a2-b92a-46d00adeae00)
* result[+] = Reference(urn:uuid:842d9ab9-d940-4f0c-adf9-e5c528f5c0e5)
* result[+] = Reference(urn:uuid:9a9f9a4a-52e3-4738-bd0b-a25374bbf358)
* result[+] = Reference(urn:uuid:58828523-8893-45fc-973b-16290366c5e5)
* result[+] = Reference(urn:uuid:2c28b23f-3e9f-4c03-8c8f-0e76bc5dc9c2)
* result[+] = Reference(urn:uuid:41bebbe5-e06f-4867-aa22-7c06db69dbd1)
* result[+] = Reference(urn:uuid:1642f190-e2c6-4999-8040-b9b2a70618bf)
* result[+] = Reference(urn:uuid:c3587931-242f-4129-93f9-be24500c8f29)
* result[+] = Reference(urn:uuid:41695fc0-1fd5-4cc8-95e6-82b2848a5cb6)
* result[+] = Reference(urn:uuid:58eb14f6-4059-4168-86a9-155ae61d30e2)
* result[+] = Reference(urn:uuid:1a71e80f-b044-4a91-80e1-eadbe5a53dca)

Instance: Inline-Instance-for-oncology-report-example-2
InstanceOf: Patient
Usage: #inline
* identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/patID"
* identifier.value = "11111"

Instance: Inline-Instance-for-oncology-report-example-3
InstanceOf: Specimen
Usage: #inline
* identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* identifier.value = "UNKNOWN"
* type = $SPECIMENTYPE#TUMOR "Tumor"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* collection.method.text = "Biopsy"
* collection.bodySite = $ICD10CM#C16.0

Instance: Inline-Instance-for-oncology-report-example-4
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:8975 "PIK3CA"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_006218.4:c.3140A>G "NM_006218.4:c.3140A>G"
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:1000002 "substitution"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_006209.2:p.His1047Arg "NP_006209.2:p.His1047Arg"
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_006218.3 "NM_006218.4"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "A"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.2188 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 64 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-5
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:7989 "NRAS"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_002524.4:c.34G>T "NM_002524.4:c.34G>T"
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:1000002 "substitution"
//kp - unable to validate p.
// * component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
// * component[=].valueCodeableConcept.text = "p.G12C" //$HGVS#p.G12C
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_002524.4 "NM_002524.4"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "C"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.1793 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 145 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-6
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:16712 "FBXW7"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_001349798.2:c.1394G>A "NM_001349798.2:c.1394G>A"
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:1000002 "substitution"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_001336727.1:p.Arg465His "NP_001336727.1:p.Arg465His"
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_001349798.2 "NM_001349798.2"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "C"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.1053 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 57 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-7
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:7133 "KMT2D"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_003482.3:c.7900_7901delCA "NM_003482.3:c.7900_7901delCA"
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:0000159 "deletion"
// kp unable to validation p.
// * component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
// * component[=].valueCodeableConcept.text = "p.Q2634Afs*20" //$HGVS#p.Q2634Afs*20
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_003482.3 "NM_003482.3"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "CTG"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.188 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 117 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-8
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:8975 "PIK3CA"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_006218.3:c.333G>T "NM_006218.3:c.333G>T"
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:1000002 "substitution"
// kp unable to validation p.
// * component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
// * component[=].valueCodeableConcept.text = "p.K111N" //$HGVS#p.K111N
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_006218.3 "NM_006218.3"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "G"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.1471 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 68 '1' "reads per base pair"

Instance: Inline-Instance-for-oncology-report-example-9
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:f7a438e6-f484-453d-97e8-aa4d51008648)
* effectiveDateTime = "2023-03-05"
* performer = Reference(urn:uuid:fc16d84c-8584-4e1d-baae-64e2f95bfe17)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://molit.eu/fhir/genomics/NamingSystem/cegat/tissueID"
* specimen.identifier.value = "UNKNOWN"
* component[0].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48018-6 "Gene studied [ID]"
* component[=].valueCodeableConcept = $HGNCID#HGNC:6126 "IRS2"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept = $HGVS#NM_003749.2:c.3960C>T "NM_003749.2:c.3960C>T"
* component[+].code = $LNC#48019-4 "DNA change type"
* component[=].valueCodeableConcept = $SEQONT#SO:1000002 "substitution"
//kp unable to validate p.
// * component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
// * component[=].valueCodeableConcept.text = "p.=" //$HGVS#p.=
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_003749.2 "NM_003749.2"
* component[+].code = $LNC#69547-8 "Genomic ref allele [ID]"
* component[=].valueString = "G"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.1343 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 134 '1' "reads per base pair"


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-oncologyexamples-r4-withGrouping.fsh

Instance: bundle-oncologyexamples-r4-withGrouping
InstanceOf: Bundle
Description: "Example bundle with an oncology report with SNVs, TMB, MSI, and therapy matches, including some groupings of observations"
Usage: #example
* type = #transaction
* entry[+].fullUrl = "urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-1
* entry[=].request.method = #POST
* entry[=].request.url = "Patient"
* entry[+].fullUrl = "urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-2
* entry[=].request.method = #POST
* entry[=].request.url = "Practitioner"
* entry[+].fullUrl = "urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-3
* entry[=].request.method = #POST
* entry[=].request.url = "Specimen"
* entry[+].fullUrl = "urn:uuid:40e85eda-4d31-4bd2-a5a0-c39d63f9824f"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-4
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:64b91716-6431-41bd-8fd4-7ac99bebdee9"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-5
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-6
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:772de7d7-e1bd-4183-bdb5-cea8eda317cd"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-7
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:b33eb7ec-d83e-49d2-b0f3-729d024ebe01"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-8
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:1bf9642d-fd1c-4819-8c5d-44fae4bad524"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-9
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:1301943b-55db-4adf-8355-3635f7268a1b"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-10
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:05713d1c-0df5-46d5-a5a9-2085eef56c20"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-11
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:294f1401-e2eb-4df1-b6d2-09e1b7a6eca4"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-12
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12f"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-13
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12e"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-14
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:ad7a499b-a43b-4cc8-bf46-cac62d9ecae5"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-withGrouping-15
* entry[=].request.method = #POST
* entry[=].request.url = "DiagnosticReport"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-1
InstanceOf: Patient
Usage: #inline
* identifier.value = "1007"
* name.given[+] = "Carrot"
* name.given[+] = "John"
* name.given[+] = "Mr"
* gender = #male
* birthDate = "1962-12-31"
* generalPractitioner = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-10
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* method = $LNC#LA26811-2 "Computational analysis"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* derivedFrom = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d)
* component[+].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[+].code = $LNC#93044-6 "Level of Evidence"
* component[=].valueCodeableConcept = $LNC#LA30200-2 "Very strong evidence pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.text = "Polycythemia vera (PV)"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "AT9283"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-11
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* method = $LNC#LA26811-2 "Computational analysis"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* derivedFrom = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[+].code = $LNC#93044-6 "Level of Evidence"
* component[=].valueCodeableConcept = $LNC#LA30200-2 "Very strong evidence pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.text = "Polycythemia vera (PV)"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "SNX-5422"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-12
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* method = $LNC#LA26811-2 "Computational analysis"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* derivedFrom = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[+].code = $LNC#93044-6 "Level of Evidence"
* component[=].valueCodeableConcept = $LNC#LA30200-2 "Very strong evidence pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.text = "Polycythemia vera (PV)"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "TG02"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-13
InstanceOf: Observation
Usage: #inline
* status = #final
* category[+].coding = $OBSCAT#laboratory
* category[+].coding = $DIAGNOSTICSERVICE#GE
* code = $NCIT#C43359 "Group"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* hasMember[0] = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d) "SNV analysis for JAK2 variant"
* hasMember[+] = Reference(urn:uuid:772de7d7-e1bd-4183-bdb5-cea8eda317cd) "SNV analysis for KDR variant"
* hasMember[+] = Reference(urn:uuid:b33eb7ec-d83e-49d2-b0f3-729d024ebe01) "SNV analysis for ERBB4 variant"
* hasMember[+] = Reference(urn:uuid:40e85eda-4d31-4bd2-a5a0-c39d63f9824f) "TMB value analysis"
* hasMember[+] = Reference(urn:uuid:64b91716-6431-41bd-8fd4-7ac99bebdee9) "MSI status analysis"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-14
InstanceOf: Observation
Usage: #inline
* status = #final
* category[+].coding = $OBSCAT#laboratory
* category[+].coding = $DIAGNOSTICSERVICE#GE
* code = $NCIT#C43359 "Group"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* hasMember[0] = Reference(urn:uuid:1bf9642d-fd1c-4819-8c5d-44fae4bad524) "Therapy match result 1"
* hasMember[+] = Reference(urn:uuid:1301943b-55db-4adf-8355-3635f7268a1b) "Therapy match result 2"
* hasMember[+] = Reference(urn:uuid:05713d1c-0df5-46d5-a5a9-2085eef56c20) "Therapy match result 3"
* hasMember[+] = Reference(urn:uuid:294f1401-e2eb-4df1-b6d2-09e1b7a6eca4) "Therapy match result 4"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-15
InstanceOf: GenomicReport
Usage: #inline
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE
* code = $LNC#51969-4
  * text = "Genetic analysis report"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* result[0] = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12f) "Variants and other biomarker tests"
* result[+] = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d) "SNV analysis for JAK2 variant"
* result[+] = Reference(urn:uuid:772de7d7-e1bd-4183-bdb5-cea8eda317cd) "SNV analysis for KDR variant"
* result[+] = Reference(urn:uuid:b33eb7ec-d83e-49d2-b0f3-729d024ebe01) "SNV analysis for ERBB4 variant"
* result[+] = Reference(urn:uuid:40e85eda-4d31-4bd2-a5a0-c39d63f9824f) "TMB value analysis"
* result[+] = Reference(urn:uuid:64b91716-6431-41bd-8fd4-7ac99bebdee9) "MSI status analysis"
* result[+] = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12e) "Therapy Match Results"
* result[+] = Reference(urn:uuid:1bf9642d-fd1c-4819-8c5d-44fae4bad524) "Therapy match result 1"
* result[+] = Reference(urn:uuid:1301943b-55db-4adf-8355-3635f7268a1b) "Therapy match result 2"
* result[+] = Reference(urn:uuid:05713d1c-0df5-46d5-a5a9-2085eef56c20) "Therapy match result 3"
* result[+] = Reference(urn:uuid:294f1401-e2eb-4df1-b6d2-09e1b7a6eca4) "Therapy match result 4"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-2
InstanceOf: Practitioner
Usage: #inline
* identifier.value = "4654765876"
* name.given[0] = "Akerman"
* name.given[+] = "Philip"
* name.given[+] = "Prof"
* address.city = "Castro Valley"
* address.state = "CA"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-3
InstanceOf: Specimen
Usage: #inline
* identifier.value = "076e1948-2217-11e9-94d8-12e241dc1f66"
* status = #available
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* collection.collector = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-4
InstanceOf: MolecularBiomarker
Usage: #inline
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-biomarker"
* status = #final "final"
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#94076-7 "Mutations/Megabase [# Ratio] in Tumor"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueQuantity = 25.0 '1/1000000' "somatic variants per megabase of sequenced DNA"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[biomarker-category].code = TbdCodesCS#biomarker-category
* component[biomarker-category].valueCodeableConcept.coding = MolecularBiomarkerOntologyCS#molgen "molecular sequence adjacent category"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-5
InstanceOf: MolecularBiomarker
Usage: #inline
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-biomarker"
* status = #final "final"
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#81695-9 "Microsatellite instability [Interpretation] in Cancer specimen Qualitative"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueCodeableConcept = $LNC#LA14122-8 "Stable"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[biomarker-category].code = TbdCodesCS#biomarker-category
* component[biomarker-category].valueCodeableConcept.coding = MolecularBiomarkerOntologyCS#molgen "molecular sequence adjacent category"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-6
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[0].code = $LNC#81290-9 "Genomic DNA change (gHGVS)"
* component[=].valueCodeableConcept = $HGVS#NC_000009.12:g.5073770G>T "NC_000009.12:g.5073770G>T"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_004963.1:p.Val617Phe "NP_004963.1:p.Val617Phe"
* component[+].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6683-2 "Germline"
* component[+].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept.coding.system = "http://www.genenames.org"
* component[=].valueCodeableConcept.coding.display = "JAK2"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-7
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[0].code = $LNC#81290-9 "Genomic DNA change (gHGVS)"
* component[=].valueCodeableConcept = $HGVS#NC_000004.11:g.55972974T>A "NC_000004.11:g.55972974T>A"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_002244.1:p.Gln472His "NP_002244.1:p.Gln472His"
* component[+].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6683-2 "Germline"
* component[+].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept.coding.system = "http://www.genenames.org"
* component[=].valueCodeableConcept.coding.display = "KDR"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-8
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[0].code = $LNC#81290-9 "Genomic DNA change (gHGVS)"
* component[=].valueCodeableConcept = $HGVS#NC_000002.12:g.211713666_211713668del "NC_000002.12:g.211713666_211713668del"
* component[+].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6683-2 "Germline"
* component[+].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept.coding.system = "http://www.genenames.org"
* component[=].valueCodeableConcept.coding.display = "ERBB4"

Instance: Inline-Instance-for-oncologyexamples-r4-withGrouping-9
InstanceOf: TherapeuticImplication
Usage: #inline
* contained[+] = Inline-MedicationKnowledgeRuxolitinib
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-05-01"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* method = $LNC#LA26811-2 "Computational analysis"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* derivedFrom = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d)
* component[therapeutic-implication].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[evidence-level].valueCodeableConcept = $LNC#LA30200-2 "Very strong evidence pathogenic"
* component[phenotypic-treatment-context].valueCodeableConcept.text = "Polycythemia vera (PV)"
* component[medication-assessed]
  * valueCodeableConcept = $RXN#1193326 "Ruxolitinib"
  * extension[MedicationAssessedReference].valueReference = Reference(Inline-MedicationKnowledgeRuxolitinib)

Instance: Inline-MedicationKnowledgeRuxolitinib
InstanceOf: MedicationKnowledge
Usage: #inline
Description: "Simple MedicationKnowledge instance for Ruxolitinib"
* code.coding = $RXN#1193326 "Ruxolitinib"
* status = #active


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-oncologyexamples-r4.fsh

Instance: bundle-oncologyexamples-r4
InstanceOf: Bundle
Description: "Example bundle with an oncology report with SNVs, TMB, MSI, and therapy matches."
Usage: #example
* type = #transaction
* entry[0].fullUrl = "urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-1
* entry[=].request.method = #POST
* entry[=].request.url = "Patient"
* entry[+].fullUrl = "urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-2
* entry[=].request.method = #POST
* entry[=].request.url = "Practitioner"
* entry[+].fullUrl = "urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-3
* entry[=].request.method = #POST
* entry[=].request.url = "Specimen"
* entry[+].fullUrl = "urn:uuid:40e85eda-4d31-4bd2-a5a0-c39d63f9824f"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-4
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:64b91716-6431-41bd-8fd4-7ac99bebdee9"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-5
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-6
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:772de7d7-e1bd-4183-bdb5-cea8eda317cd"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-7
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:b33eb7ec-d83e-49d2-b0f3-729d024ebe01"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-8
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:1bf9642d-fd1c-4819-8c5d-44fae4bad524"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-9
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:1301943b-55db-4adf-8355-3635f7268a1b"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-10
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:05713d1c-0df5-46d5-a5a9-2085eef56c20"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-11
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:294f1401-e2eb-4df1-b6d2-09e1b7a6eca4"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-12
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:294f1401-e2eb-4df1-b6d2-09e1b7a6ecb6"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-14
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:ad7a499b-a43b-4cc8-bf46-cac62d9ecae5"
* entry[=].resource = Inline-Instance-for-oncologyexamples-r4-13
* entry[=].request.method = #POST
* entry[=].request.url = "DiagnosticReport"

Instance: Inline-Instance-for-oncologyexamples-r4-1
InstanceOf: Patient
Usage: #inline
* identifier.value = "1007"
* name.given[0] = "Carrot"
* name.given[+] = "John"
* name.given[+] = "Mr"
* gender = #male
* birthDate = "1962-12-31"
* generalPractitioner = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)

Instance: Inline-Instance-for-oncologyexamples-r4-10
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-04-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* method = $LNC#LA26811-2 "Computational analysis"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* derivedFrom = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[+].code = $LNC#93044-6 "Level of Evidence"
* component[=].valueCodeableConcept = $LNC#LA30200-2 "Very strong evidence pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.text = "Polycythemia vera (PV)"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "AT9283"

Instance: Inline-Instance-for-oncologyexamples-r4-11
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-04-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* method = $LNC#LA26811-2 "Computational analysis"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* derivedFrom = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[+].code = $LNC#93044-6 "Level of Evidence"
* component[=].valueCodeableConcept = $LNC#LA30200-2 "Very strong evidence pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.text = "Polycythemia vera (PV)"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "SNX-5422"

Instance: Inline-Instance-for-oncologyexamples-r4-12
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-04-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* method = $LNC#LA26811-2 "Computational analysis"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* derivedFrom = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[+].code = $LNC#93044-6 "Level of Evidence"
* component[=].valueCodeableConcept = $LNC#LA30200-2 "Very strong evidence pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.text = "Polycythemia vera (PV)"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "TG02"

Instance: Inline-Instance-for-oncologyexamples-r4-13
InstanceOf: GenomicReport
Usage: #inline
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE
* code = $LNC#51969-4
  * text = "Genetic analysis report"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* result[0] = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d) "SNV analysis for JAK2 variant"
* result[+] = Reference(urn:uuid:772de7d7-e1bd-4183-bdb5-cea8eda317cd) "SNV analysis for KDR variant"
* result[+] = Reference(urn:uuid:b33eb7ec-d83e-49d2-b0f3-729d024ebe01) "SNV analysis for ERBB4 variant"
* result[+] = Reference(urn:uuid:40e85eda-4d31-4bd2-a5a0-c39d63f9824f) "TMB value analysis"
* result[+] = Reference(urn:uuid:64b91716-6431-41bd-8fd4-7ac99bebdee9) "MSI status analysis"
* result[+] = Reference(urn:uuid:1bf9642d-fd1c-4819-8c5d-44fae4bad524) "Therapy match result 1"
* result[+] = Reference(urn:uuid:1301943b-55db-4adf-8355-3635f7268a1b) "Therapy match result 2"
* result[+] = Reference(urn:uuid:05713d1c-0df5-46d5-a5a9-2085eef56c20) "Therapy match result 3"
* result[+] = Reference(urn:uuid:294f1401-e2eb-4df1-b6d2-09e1b7a6eca4) "Therapy match result 4"

Instance: Inline-Instance-for-oncologyexamples-r4-2
InstanceOf: Practitioner
Usage: #inline
* identifier.value = "4654765876"
* name.given[0] = "Akerman"
* name.given[+] = "Philip"
* name.given[+] = "Prof"
* address.city = "Castro Valley"
* address.state = "CA"

Instance: Inline-Instance-for-oncologyexamples-r4-3
InstanceOf: Specimen
Usage: #inline
* identifier.value = "076e1948-2217-11e9-94d8-12e241dc1f66"
* status = #available
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* collection.collector = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)

Instance: Inline-Instance-for-oncologyexamples-r4-4
InstanceOf: MolecularBiomarker
Usage: #inline
* status = #final
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-biomarker"
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#94076-7 "Mutations/Megabase [# Ratio] in Tumor"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-04-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueQuantity = 25.0 '1/1000000' "somatic variants per megabase of sequenced DNA"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[biomarker-category].code = TbdCodesCS#biomarker-category
* component[biomarker-category].valueCodeableConcept.coding = MolecularBiomarkerOntologyCS#molgen "molecular sequence adjacent category"

Instance: Inline-Instance-for-oncologyexamples-r4-5
InstanceOf: MolecularBiomarker
Usage: #inline
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-biomarker"
* status = #final "final"
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#81695-9 "Microsatellite instability [Interpretation] in Cancer specimen Qualitative"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-04-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueCodeableConcept = $LNC#LA14122-8 "Stable"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[biomarker-category].code = TbdCodesCS#biomarker-category
* component[biomarker-category].valueCodeableConcept.coding = MolecularBiomarkerOntologyCS#molgen "molecular sequence adjacent category"

Instance: Inline-Instance-for-oncologyexamples-r4-6
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-04-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[0].code = $LNC#81290-9 "Genomic DNA change (gHGVS)"
* component[=].valueCodeableConcept = $HGVS#NC_000009.12:g.5073770G>T "NC_000009.12:g.5073770G>T"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_004963.1:p.Val617Phe "NP_004963.1:p.Val617Phe"
* component[+].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6683-2 "Germline"
* component[+].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept.coding.system = "http://www.genenames.org"
* component[=].valueCodeableConcept.coding.display = "JAK2"

Instance: Inline-Instance-for-oncologyexamples-r4-7
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-04-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[0].code = $LNC#81290-9 "Genomic DNA change (gHGVS)"
* component[=].valueCodeableConcept = $HGVS#NC_000004.11:g.55972974T>A "NC_000004.11:g.55972974T>A"
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_002244.1:p.Gln472His "NP_002244.1:p.Gln472His"
* component[+].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6683-2 "Germline"
* component[+].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept.coding.system = "http://www.genenames.org"
* component[=].valueCodeableConcept.coding.display = "KDR"

Instance: Inline-Instance-for-oncologyexamples-r4-8
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-04-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* component[0].code = $LNC#81290-9 "Genomic DNA change (gHGVS)"
* component[=].valueCodeableConcept = $HGVS#NC_000002.12:g.211713666_211713668del "NC_000002.12:g.211713666_211713668del"
* component[+].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept = $LNC#LA6683-2 "Germline"
* component[+].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept.coding.system = "http://www.genenames.org"
* component[=].valueCodeableConcept.coding.display = "ERBB4"

Instance: Inline-Instance-for-oncologyexamples-r4-9
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-04-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* method = $LNC#LA26811-2 "Computational analysis"
* specimen = Reference(urn:uuid:a5d6a6a9-4859-480e-85b2-56974fef3b9d)
* derivedFrom = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[+].code = $LNC#93044-6 "Level of Evidence"
* component[=].valueCodeableConcept = $LNC#LA30200-2 "Very strong evidence pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.text = "Polycythemia vera (PV)"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "Ruxolitinib"


Instance: Inline-Instance-for-oncologyexamples-r4-14
InstanceOf: TherapeuticImplication
Usage: #inline
* contained[+] = PlanDefRuxolitinib
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* subject = Reference(urn:uuid:d0b4affa-91d6-46d1-af01-b30d9f16ef6d)
* effectiveDateTime = "2023-04-05"
* performer = Reference(urn:uuid:a48256f8-db37-44e0-a0f6-d7af16c7c9ef)
* derivedFrom = Reference(urn:uuid:16fdd5fc-e665-48af-99e4-48d603f3e12d)
* component[therapy-assessed]
  * valueCodeableConcept.text = "Ruxolitinib"
  * extension[TherapyAssessedReference].valueReference = Reference(PlanDefRuxolitinib)

Instance: PlanDefRuxolitinib
InstanceOf: PlanDefinition
Usage: #inline
Description: "Plan Definition including Ruxolitinib (for example only)"
* status = #draft
* name = "RuxolitinibPlan"
* description = """
This is not a fully defined resource, just here as a place holder to show how a PlanDefinition could be packaged.
"""

/*
NOTE - Planned to use this ResearchStudy as an example of the 'therapy-assesed' extension above, but the publisher (as of Mar 19 2022)
       kept complaining that ResearchStudy was not a valid resource when building the example.

Instance: ResearchStudyNCT02928978
InstanceOf: ResearchStudy
Usage: #inline
Description: "Ruxolitinib for Premalignant Breast Disease (TBCRC 042)"
* title = "Ruxolitinib for Premalignant Breast Disease (TBCRC 042)"
* identifier[+].system = "http://clinicaltrials.gov"
* identifier[=].value = "NCT02928978"
* status = #active
* description = """
This study is evaluating how ruxolitinib affects premalignant breast cells. One half of the study participants will receive ruxolitinib for approximately 15 days, and the other half will receive a placebo (sugar pill) for approximately 15 days. Once study participants have completed their ruxolitinib or placebo, participants will undergo surgery to remove the premalignant breast tissue.
"""
*/

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-pgxexample.fsh

Instance: bundle-pgxexample
InstanceOf: Bundle
Description: "Example bundle with a PGx report and a variety of observations."
Usage: #example
* type = #transaction
* entry[0].fullUrl = "http://example.org/fhir/DiagnosticReport/pgx-report"
* entry[=].resource = pgx-report
* entry[=].request.method = #POST
* entry[=].request.url = "DiagnosticReport"
* entry[+].fullUrl = "http://example.org/fhir/Patient/ExamplePatient"
* entry[=].resource = ExamplePatient
* entry[=].request.method = #POST
* entry[=].request.url = "Patient"
* entry[+].fullUrl = "http://example.org/fhir/Specimen/ExampleSpecimen"
* entry[=].resource = ExampleSpecimen
* entry[=].request.method = #POST
* entry[=].request.url = "Specimen"
* entry[+].fullUrl = "http://example.org/fhir/Organization/ExampleLab"
* entry[=].resource = ExampleLab
* entry[=].request.method = #POST
* entry[=].request.url = "Organization"
* entry[+].fullUrl = "http://example.org/fhir/ServiceRequest/pgx-request"
* entry[=].resource = pgx-request
* entry[=].request.method = #POST
* entry[=].request.url = "ServiceRequest"
* entry[+].fullUrl = "http://example.org/fhir/Observation/med-impact-pgx-example"
* entry[=].resource = med-impact-pgx-example
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "http://example.org/fhir/Observation/genotype-pgx-example"
* entry[=].resource = genotype-pgx-example
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "http://example.org/fhir/Observation/haplotype-1-pgx-example"
* entry[=].resource = haplotype-1-pgx-example
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "http://example.org/fhir/Observation/haplotype-2-pgx-example"
* entry[=].resource = haplotype-2-pgx-example
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "http://example.org/fhir/Observation/variant-1-pgx-example"
* entry[=].resource = variant-1-pgx-example
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "http://example.org/fhir/Observation/variant-2-pgx-example"
* entry[=].resource = variant-2-pgx-example
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "http://example.org/fhir/MolecularSequence/sequence-1-pgx-example"
* entry[=].resource = sequence-1-pgx-example
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "http://example.org/fhir/MolecularSequence/sequence-2-pgx-example"
* entry[=].resource = sequence-2-pgx-example
* entry[=].request.method = #POST
* entry[=].request.url = "MolecularSequence"
* entry[+].fullUrl = "http://example.org/fhir/Task/med-usage-impact-task-pgx-example"
* entry[=].resource = med-usage-impact-task-pgx-example
* entry[=].request.method = #POST
* entry[=].request.url = "Task"
* entry[+].fullUrl = "http://example.org/fhir/MedicationStatement/current-med-example"
* entry[=].resource = current-med-example
* entry[=].request.method = #POST
* entry[=].request.url = "MedicationStatement"

Instance: pgx-report
InstanceOf: GenomicReport
Usage: #inline
* extension.url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/recommended-action"
* extension.valueReference = Reference(med-usage-impact-task-pgx-example)
* basedOn = Reference(pgx-request)
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE
* code = $LNC#51969-4
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2016"
* issued = "2018-03-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* result[0] = Reference(med-impact-pgx-example) "genetics impact for medication metabolism accessment"
* result[+] = Reference(genotype-pgx-example)
* result[+] = Reference(haplotype-1-pgx-example)
* result[+] = Reference(haplotype-2-pgx-example)
* result[+] = Reference(variant-1-pgx-example)
* result[+] = Reference(variant-2-pgx-example)

Instance: pgx-request
InstanceOf: ServiceRequest
Usage: #inline
* status = #active
* intent = #original-order
* code = $LNC#51969-4
* subject = Reference(ExamplePatient)
* reasonCode.text = "Worried about family planning"

Instance: med-impact-pgx-example
InstanceOf: TherapeuticImplication
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2018"
* issued = "2018-03-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(genotype-pgx-example)
* component[0].code = TbdCodesCS#therapeutic-implication
* component[=].valueCodeableConcept = $LNC#LA25390-8 "Rapid metabolizer"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept = $RXN#103 "mercaptopurine"

Instance: genotype-pgx-example
InstanceOf: Genotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84413-4 "genotype display name"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2018"
* issued = "2018-03-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $PHARMVAR#"CYP2C9 "CYP2C9 *4/*35B"
* specimen = Reference(ExampleSpecimen)
* derivedFrom[0] = Reference(haplotype-1-pgx-example)
* derivedFrom[+] = Reference(haplotype-2-pgx-example)
* component.code = $LNC#48018-6 "Gene studied ID"
* component.valueCodeableConcept = $HGNCID#HGNC:2623 "CYP2C9"

Instance: haplotype-1-pgx-example
InstanceOf: Haplotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype Name"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2018"
* issued = "2018-03-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $PHARMVAR#"CYP2C9 "CYP2C9 *35B"
* specimen = Reference(ExampleSpecimen)
* derivedFrom = Reference(variant-1-pgx-example)
* component.code = $LNC#48018-6 "Gene studied ID"
* component.valueCodeableConcept = $HGNCID#HGNC:2623 "CYP2C9"

Instance: haplotype-2-pgx-example
InstanceOf: Haplotype
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype Name"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2018"
* issued = "2018-03-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $PHARMVAR#"CYP2C9 "CYP2C9 *4"
* specimen = Reference(ExampleSpecimen)
* derivedFrom = Reference(variant-2-pgx-example)
* component.code = $LNC#48018-6 "Gene studied ID"
* component.valueCodeableConcept = $HGNCID#HGNC:2623 "CYP2C9"

Instance: variant-1-pgx-example
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2018"
* issued = "2018-03-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* specimen = Reference(ExampleSpecimen)
* derivedFrom = Reference(sequence-1-pgx-example)
* component[0].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept = $HGNCID#HGNC:2623 "CYP2C9"
* component[+].code = $LNC#81252-9
* component[=].valueCodeableConcept = $SNP#rs12345
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 55227976
* component[=].valueRange.high.value = 55227977

Instance: variant-2-pgx-example
InstanceOf: Variant
Usage: #inline
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2018"
* issued = "2018-03-06T00:00:00-05:00"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* specimen = Reference(ExampleSpecimen)
* derivedFrom = Reference(sequence-2-pgx-example)
* component[0].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept = $HGNCID#HGNC:2623 "CYP2C9"
* component[+].code = $LNC#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NG_007726.3
* component[+].code = $LNC#81252-9
* component[=].valueCodeableConcept = $SNP#rs233455
* component[+].code = $LNC#69547-8 "Ref nucleotide"
* component[=].valueString = "T"
* component[+].code = $LNC#69551-0 "Alt allele"
* component[=].valueString = "G"
* component[+].code = $LNC#81254-5
* component[=].valueRange.low.value = 55227978
* component[=].valueRange.high.value = 55227979

Instance: sequence-1-pgx-example
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.orientation = #sense
* referenceSeq.referenceSeqId = $NCBIREFSEQ#NG-007726.3
* referenceSeq.windowStart = 55227970
* referenceSeq.windowEnd = 55227980

Instance: sequence-2-pgx-example
InstanceOf: MolecularSequence
Usage: #inline
* type = #dna
* coordinateSystem = 0
* referenceSeq.orientation = #sense
* referenceSeq.referenceSeqId = $NCBIREFSEQ#NG-007726.3
* referenceSeq.windowStart = 55227970
* referenceSeq.windowEnd = 55227980

Instance: current-med-example
InstanceOf: MedicationStatement
Usage: #inline
* status = #unknown
* medicationCodeableConcept = $RXN#103 "mercaptopurine"
* subject = Reference(ExamplePatient)


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/bundle-sequence-phase-relation-CYP2C19.fsh

Instance: bundle-sequence-phase-relation-CYP2C19
InstanceOf: Bundle
Description: "Example Sequence Phase Relation showing a report with two variants in cis"
Usage: #example
* type = #transaction
* entry[0].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe60"
* entry[=].resource = dv-5c7e9e1ca8252
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:20ac0aeb-6bd4-4e92-a891-d44a807bfe01"
* entry[=].resource = dv-5c7e9e1ca82e4
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:20ac0aeb-6bd4-4e92-a891-d44a807bfe02"
* entry[=].resource = sid-5c7e9e1ca8a29
* entry[=].request.method = #POST
* entry[=].request.url = "Observation"
* entry[+].fullUrl = "urn:uuid:20ac0aeb-6bd4-4e92-a891-d44a807bfe03"
* entry[=].resource = sequence-phase-relation-CYP2C19
* entry[=].request.method = #POST
* entry[=].request.url = "DiagnosticReport"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeab"
* entry[=].resource = performingLab
* entry[=].request.method = #POST
* entry[=].request.url = "Organization"
* entry[+].fullUrl = "urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfeac"
* entry[=].resource = patient
* entry[=].request.method = #POST
* entry[=].request.url = "Patient"

Instance: sequence-phase-relation-CYP2C19
InstanceOf: GenomicReport
Usage: #inline
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE
//* code = $LNC#81254-5
//  * text = "Genetic analysis report"
* result[0] = Reference(urn:uuid:19ac0aeb-6bd4-4e92-a891-d44a807bfe60)
* result[+] = Reference(urn:uuid:20ac0aeb-6bd4-4e92-a891-d44a807bfe01)
* result[+] = Reference(urn:uuid:20ac0aeb-6bd4-4e92-a891-d44a807bfe02)


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/CGOpsRequests.fsh

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-subject-variants.html

Instance: FindSubjectVariantsOutput
InstanceOf: Parameters
Description: "Example Output of 'Find Subject Variants' Operation"

* parameter[0].name = "variants"
* parameter[=].part[0].name = "rangeItem"
* parameter[=].part[=].valueString = "NC_000017.11:39688093-39728660"
* parameter[=].part[+].name = "presence"
* parameter[=].part[=].valueBoolean = false
* parameter[+].name = "variants"
* parameter[=].part[0].name = "rangeItem"
* parameter[=].part[=].valueString = "NC_000007.14:140713327-140924929"
* parameter[=].part[+].name = "presence"
* parameter[=].part[=].valueBoolean = true
* parameter[=].part[+].name = "variant"
* parameter[=].part[=].resource.resourceType = "Observation"
* parameter[=].part[=].resource.id = "dv-5a7f781e83514"
* parameter[=].part[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* parameter[=].part[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Variant</div>"
* parameter[=].part[=].resource.text.status = #generated
* parameter[=].part[=].resource.status = #final
* parameter[=].part[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].part[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].part[=].resource.code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.code.coding.code = #69548-6
* parameter[=].part[=].resource.code.coding.display = "Genetic variant assessment"
* parameter[=].part[=].resource.subject.reference = "Patient/HG00403"
* parameter[=].part[=].resource.effectiveDateTime = "2023-07-01"
* parameter[=].part[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter[=].part[=].resource.valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.valueCodeableConcept.coding.code = #LA9633-4
* parameter[=].part[=].resource.valueCodeableConcept.coding.display = "Present"
* parameter[=].part[=].resource.component[0].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #48002-0
* parameter[=].part[=].resource.component[=].code.coding.display = "Genomic source class [Type]"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.code = #LA6684-0
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.display = "Somatic"
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #48013-7
* parameter[=].part[=].resource.component[=].code.coding.display = "Genomic reference sequence ID"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.system = "http://www.ncbi.nlm.nih.gov/refseq"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.code = #NC_000007.14
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #81258-6
* parameter[=].part[=].resource.component[=].code.coding.display = "Sample VAF"
* parameter[=].part[=].resource.component[=].valueQuantity.value = 0.1793
* parameter[=].part[=].resource.component[=].valueQuantity.code = #1
* parameter[=].part[=].resource.component[=].valueQuantity.system = "http://unitsofmeasure.org"
* parameter[=].part[=].resource.component[=].valueQuantity.unit = "relative frequency of a particular allele in the specimen"
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #69547-8
* parameter[=].part[=].resource.component[=].code.coding.display = "Genomic Ref allele [ID]"
* parameter[=].part[=].resource.component[=].valueString = "A"
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #69551-0
* parameter[=].part[=].resource.component[=].code.coding.display = "Genomic Alt allele [ID]"
* parameter[=].part[=].resource.component[=].valueString = "T"
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #92822-6
* parameter[=].part[=].resource.component[=].code.coding.display = "Genomic coord system"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.code = #LA30100-4
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.display = "0-based interval counting"
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #81254-5
* parameter[=].part[=].resource.component[=].code.text = "Genomic allele start-end"
* parameter[=].part[=].resource.component[=].valueRange.low.value = 140753335

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-subject-specific-variants.html

Instance: FindSubjectSpecificVariantsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindSubjectSpecificVariants' Operation"
Usage: #example
* parameter[0].name = "variants"
* parameter[=].part[0].name = "variantItem"
* parameter[=].part[=].valueString = "NM_001354609.2:c.1799T>A"
* parameter[=].part[+].name = "presence"
* parameter[=].part[=].valueBoolean = false
* parameter[+].name = "variants"
* parameter[=].part[0].name = "variantItem"
* parameter[=].part[=].valueString = "NM_007294.4:c.5559C>A"
* parameter[=].part[+].name = "presence"
* parameter[=].part[=].valueBoolean = false
* parameter[+].name = "variants"
* parameter[=].part[0].name = "variantItem"
* parameter[=].part[=].valueString = "NM_001354609.2:c.1802="
* parameter[=].part[+].name = "presence"
* parameter[=].part[=].valueBoolean = false
* parameter[+].name = "variants"
* parameter[=].part[0].name = "variantItem"
* parameter[=].part[=].valueString = "NC_000001.10:g.12225351T>A"
* parameter[=].part[+].name = "presence"
* parameter[=].part[=].valueBoolean = false
* parameter[+].name = "variants"
* parameter[=].part[0].name = "variantItem"
* parameter[=].part[=].valueString = "NM_000038.6:c.55G>T"
* parameter[=].part[+].name = "presence"
* parameter[=].part[=].valueBoolean = true
* parameter[=].part[+].name = "variant"
* parameter[=].part[=].resource.resourceType = "Observation"
* parameter[=].part[=].resource.id = "dv-5a7f781e83514"
* parameter[=].part[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* parameter[=].part[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Variant</div>"
* parameter[=].part[=].resource.text.status = #generated
* parameter[=].part[=].resource.status = #final
* parameter[=].part[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].part[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].part[=].resource.code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.code.coding.code = #69548-6
* parameter[=].part[=].resource.code.coding.display = "Genetic variant assessment"
* parameter[=].part[=].resource.subject.reference = "Patient/HG00403"
* parameter[=].part[=].resource.effectiveDateTime = "2023-07-01"
* parameter[=].part[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter[=].part[=].resource.valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.valueCodeableConcept.coding.code = #LA9633-4
* parameter[=].part[=].resource.valueCodeableConcept.coding.display = "Present"
* parameter[=].part[=].resource.component[0].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #48002-0
* parameter[=].part[=].resource.component[=].code.coding.display = "Genomic source class [Type]"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.code = #LA6683-2
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.display = "Germline"
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #48013-7
* parameter[=].part[=].resource.component[=].code.coding.display = "Genomic reference sequence ID"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.system = "http://www.ncbi.nlm.nih.gov/refseq"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.code = #NC_000005.9
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #53034-5
* parameter[=].part[=].resource.component[=].code.coding.display = "Allelic state"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.code = #LA6706-1
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.display = "heterozygous"
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #69547-8
* parameter[=].part[=].resource.component[=].code.coding.display = "Genomic Ref allele [ID]"
* parameter[=].part[=].resource.component[=].valueString = "G"
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #69551-0
* parameter[=].part[=].resource.component[=].code.coding.display = "Genomic Alt allele [ID]"
* parameter[=].part[=].resource.component[=].valueString = "T"
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #92822-6
* parameter[=].part[=].resource.component[=].code.coding.display = "Genomic coord system"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.code = #LA30100-4
* parameter[=].part[=].resource.component[=].valueCodeableConcept.coding.display = "0-based interval counting"
* parameter[=].part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component[=].code.coding.code = #81254-5
* parameter[=].part[=].resource.component[=].code.text = "Genomic allele start-end"
* parameter[=].part[=].resource.component[=].valueRange.low.value = 112090641

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-subject-structural-intersecting-variants.html

Instance: FindSubjectStructuralIntersectingVariantsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindSubjectStructuralIntersectingVariants' Operation"
Usage: #example
* parameter.name = "variants"
* parameter.part[0].name = "rangeItem"
* parameter.part[=].valueString = "NC_000007.13:55242414-55242513"
* parameter.part[+].name = "presence"
* parameter.part[=].valueBoolean = true
* parameter.part[+].name = "variant"
* parameter.part[=].resource.resourceType = "Observation"
* parameter.part[=].resource.id = "dv-fff9d40035a04"
* parameter.part[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* parameter.part[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Variant</div>"
* parameter.part[=].resource.text.status = #generated
* parameter.part[=].resource.status = #final
* parameter.part[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter.part[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter.part[=].resource.code.coding.system = "http://loinc.org"
* parameter.part[=].resource.code.coding.code = #69548-6
* parameter.part[=].resource.code.coding.display = "Genetic variant assessment"
* parameter.part[=].resource.subject.reference = "Patient/HG00403"
* parameter.part[=].resource.effectiveDateTime = "2023-07-01"
* parameter.part[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter.part[=].resource.valueCodeableConcept.coding.system = "http://loinc.org"
* parameter.part[=].resource.valueCodeableConcept.coding.code = #LA9633-4
* parameter.part[=].resource.valueCodeableConcept.coding.display = "present"
* parameter.part[=].resource.component[0].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #48019-4
* parameter.part[=].resource.component[=].code.coding.display = "DNA Change Type"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.system = "http://www.sequenceontology.org"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.code = #SO:0000159
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.display = "deletion"
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #48013-7
* parameter.part[=].resource.component[=].code.coding.display = "Genomic reference sequence ID"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.system = "http://www.ncbi.nlm.nih.gov/refseq"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.code = #NC_000007.13
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #48002-0
* parameter.part[=].resource.component[=].code.coding.display = "Genomic Source Class"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.code = #LA6684-0
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.display = "Somatic"
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #69547-8
* parameter.part[=].resource.component[=].code.coding.display = "Genomic Ref allele [ID]"
* parameter.part[=].resource.component[=].valueString = "G"
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #92822-6
* parameter.part[=].resource.component[=].code.coding.display = "Genomic coord system"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.code = #LA30100-4
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.display = "0-based interval counting"
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #81302-2
* parameter.part[=].resource.component[=].code.text = "Variant inner start and end"
* parameter.part[=].resource.component[=].valueRange.high.value = 55250000
* parameter.part[=].resource.component[=].valueRange.low.value = 55240000

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-subject-structural-subsuming-variants.html

Instance: FindSubjectStructuralSubsumingVariantsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindSubjectStructuralSubsumingVariants' Operation"
Usage: #example
* parameter.name = "variants"
* parameter.part[0].name = "rangeItem"
* parameter.part[=].valueString = "NC_000007.14:116672195-116798386"
* parameter.part[+].name = "presence"
* parameter.part[=].valueBoolean = true
* parameter.part[+].name = "variant"
* parameter.part[=].resource.resourceType = "Observation"
* parameter.part[=].resource.id = "dv-64c82675925d4"
* parameter.part[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* parameter.part[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Variant</div>"
* parameter.part[=].resource.text.status = #generated
* parameter.part[=].resource.status = #final
* parameter.part[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter.part[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter.part[=].resource.code.coding.system = "http://loinc.org"
* parameter.part[=].resource.code.coding.code = #69548-6
* parameter.part[=].resource.code.coding.display = "Genetic variant assessment"
* parameter.part[=].resource.subject.reference = "Patient/HG00403"
* parameter.part[=].resource.effectiveDateTime = "2023-07-01"
* parameter.part[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter.part[=].resource.valueCodeableConcept.coding.system = "http://loinc.org"
* parameter.part[=].resource.valueCodeableConcept.coding.code = #LA9633-4
* parameter.part[=].resource.valueCodeableConcept.coding.display = "present"
* parameter.part[=].resource.component[0].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #48019-4
* parameter.part[=].resource.component[=].code.coding.display = "DNA Change Type"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.system = "http://www.sequenceontology.org"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.code = #SO:0001019
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.display = "copy_number_variation"
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #48013-7
* parameter.part[=].resource.component[=].code.coding.display = "Genomic reference sequence ID"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.system = "http://www.ncbi.nlm.nih.gov/refseq"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.code = #NC_000007.14
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #48002-0
* parameter.part[=].resource.component[=].code.coding.display = "Genomic Source Class"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.code = #LA6684-0
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.display = "Somatic"
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #82155-3
* parameter.part[=].resource.component[=].code.coding.display = "Genomic Structural Variant copy Number"
* parameter.part[=].resource.component[=].valueQuantity.value = 5
* parameter.part[=].resource.component[=].valueQuantity.code = #1
* parameter.part[=].resource.component[=].valueQuantity.system = $UCUM
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #69547-8
* parameter.part[=].resource.component[=].code.coding.display = "Genomic Ref allele [ID]"
* parameter.part[=].resource.component[=].valueString = "T"
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #92822-6
* parameter.part[=].resource.component[=].code.coding.display = "Genomic coord system"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.code = #LA30100-4
* parameter.part[=].resource.component[=].valueCodeableConcept.coding.display = "0-based interval counting"
* parameter.part[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component[=].code.coding.code = #81302-2
* parameter.part[=].resource.component[=].code.text = "Variant inner start and end"
* parameter.part[=].resource.component[=].valueRange.high.value = 116800000
* parameter.part[=].resource.component[=].valueRange.low.value = 116600000

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-subject-haplotypes.html

Instance: FindSubjectHaplotypesOutput
InstanceOf: Parameters
Description: "Example Output of 'FindSubjectHaplotypes' Operation"
Usage: #example
* parameter[0].name = "haplotypes"
* parameter[=].part[0].name = "geneItem"
* parameter[=].part[=].valueString = "http://www.genenames.org|HGNC:2625"
* parameter[=].part[+].name = "genotype"
* parameter[=].part[=].resource.resourceType = "Observation"
* parameter[=].part[=].resource.id = "GenotypeCYP2D6"
* parameter[=].part[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genotype"
* parameter[=].part[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Genotype</div>"
* parameter[=].part[=].resource.text.status = #generated
* parameter[=].part[=].resource.status = #final
* parameter[=].part[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].part[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].part[=].resource.code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.code.coding.code = #84413-4
* parameter[=].part[=].resource.code.coding.display = "Genotype display name"
* parameter[=].part[=].resource.subject.reference = "Patient/CGPatientExample01"
* parameter[=].part[=].resource.effectiveDateTime = "2023-07-01"
* parameter[=].part[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter[=].part[=].resource.valueCodeableConcept.coding.system = "http://www.pharmvar.org"
* parameter[=].part[=].resource.valueCodeableConcept.coding.code = #CYP2D6_*1/*1
* parameter[=].part[=].resource.valueCodeableConcept.coding.display = "CYP2D6 *1/*1"
* parameter[=].part[=].resource.specimen.reference = "Specimen/GenomicSpecimenExample01"
* parameter[=].part[=].resource.component.code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component.code.coding.code = #48018-6
* parameter[=].part[=].resource.component.code.coding.display = "Gene studied [ID]"
* parameter[=].part[=].resource.component.valueCodeableConcept.coding.system = "http://www.genenames.org"
* parameter[=].part[=].resource.component.valueCodeableConcept.coding.code = #HGNC:2625
* parameter[=].part[=].resource.component.valueCodeableConcept.coding.display = "CYP2D6"
* parameter[+].name = "haplotypes"
* parameter[=].part[0].name = "geneItem"
* parameter[=].part[=].valueString = "http://www.genenames.org|HGNC:2621"
* parameter[=].part[+].name = "genotype"
* parameter[=].part[=].resource.resourceType = "Observation"
* parameter[=].part[=].resource.id = "GenotypeCYP2C19"
* parameter[=].part[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genotype"
* parameter[=].part[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Genotype</div>"
* parameter[=].part[=].resource.text.status = #generated
* parameter[=].part[=].resource.status = #final
* parameter[=].part[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].part[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].part[=].resource.code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.code.coding.code = #84413-4
* parameter[=].part[=].resource.code.coding.display = "Genotype display name"
* parameter[=].part[=].resource.subject.reference = "Patient/CGPatientExample01"
* parameter[=].part[=].resource.effectiveDateTime = "2023-07-01"
* parameter[=].part[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter[=].part[=].resource.valueCodeableConcept.coding.system = "http://www.pharmvar.org"
* parameter[=].part[=].resource.valueCodeableConcept.coding.code = #CYP2C19_*1/*2
* parameter[=].part[=].resource.valueCodeableConcept.coding.display = "CYP2C19 *1/*2"
* parameter[=].part[=].resource.specimen.reference = "Specimen/GenomicSpecimenExample01"
* parameter[=].part[=].resource.component.code.coding.system = "http://loinc.org"
* parameter[=].part[=].resource.component.code.coding.code = #48018-6
* parameter[=].part[=].resource.component.code.coding.display = "Gene studied [ID]"
* parameter[=].part[=].resource.component.valueCodeableConcept.coding.system = "http://www.genenames.org"
* parameter[=].part[=].resource.component.valueCodeableConcept.coding.code = #HGNC:2621
* parameter[=].part[=].resource.component.valueCodeableConcept.coding.display = "CYP2C19"

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-subject-specific-haplotypes.html

Instance: FindSubjectSpecificHaplotypesOutput
InstanceOf: Parameters
Description: "Example Output of 'FindSubjectSpecificHaplotypes' Operation"
Usage: #example
* parameter.name = "haplotypes"
* parameter.part[0].name = "haplotypeItem"
* parameter.part[=].valueString = "HLA-A*23"
* parameter.part[+].name = "presence"
* parameter.part[=].valueBoolean = true
* parameter.part[+].name = "genotype"
* parameter.part[=].resource.resourceType = "Observation"
* parameter.part[=].resource.id = "Genotype7"
* parameter.part[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genotype"
* parameter.part[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Genotype</div>"
* parameter.part[=].resource.text.status = #generated
* parameter.part[=].resource.status = #final
* parameter.part[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter.part[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter.part[=].resource.code.coding.system = "http://loinc.org"
* parameter.part[=].resource.code.coding.code = #84413-4
* parameter.part[=].resource.code.coding.display = "Genotype display name"
* parameter.part[=].resource.subject.reference = "Patient/CGPatientExample01"
* parameter.part[=].resource.effectiveDateTime = "2023-07-01"
* parameter.part[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter.part[=].resource.valueCodeableConcept.coding.system = "http://glstring.org"
* parameter.part[=].resource.valueCodeableConcept.coding.version = "1.0"
* parameter.part[=].resource.valueCodeableConcept.coding.code = #hla#3.33.0#HLA-A*02:69+HLA-A*23:30|HLA-A*02:302+HLA-A*23:26/HLA-A*23:39
* parameter.part[=].resource.valueCodeableConcept.coding.display = "hla#3.33.0#HLA-A*02:69+HLA-A*23:30|HLA-A*02:302+HLA-A*23:26/HLA-A*23:39"
* parameter.part[=].resource.specimen.reference = "Specimen/GenomicSpecimenExample01"
* parameter.part[=].resource.component.code.coding.system = "http://loinc.org"
* parameter.part[=].resource.component.code.coding.code = #48018-6
* parameter.part[=].resource.component.code.coding.display = "Gene studied [ID]"
* parameter.part[=].resource.component.valueCodeableConcept.coding.system = "http://www.genenames.org"
* parameter.part[=].resource.component.valueCodeableConcept.coding.code = #HGNC:4931
* parameter.part[=].resource.component.valueCodeableConcept.coding.display = "HLA-A"

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-subject-tx-implications.html

Instance: FindSubjectTxImplicationsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindSubjectTxImplications' Operation"
Usage: #example
* parameter.name = "implication"
* parameter[=].resource.resourceType = "Observation"
* parameter[=].resource.id = "TherapeuticImplicationExample"
* parameter[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/therapeutic-implication"
* parameter[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Therapeutic Implication</div>"
* parameter[=].resource.text.status = #generated
* parameter[=].resource.status = #final
* parameter[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].resource.code.coding.system = "http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs"
* parameter[=].resource.code.coding.code = #therapeutic-implication
* parameter[=].resource.subject.reference = "Patient/HG00403"
* parameter[=].resource.effectiveDateTime = "2023-07-01"
* parameter[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter[=].resource.derivedFrom.reference = "Observation/VariantExample"
* parameter[=].resource.component[0].code.coding.system = "http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs"
* parameter[=].resource.component[=].code.coding.code = #therapeutic-implication
* parameter[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].valueCodeableConcept.coding.code = #LA6677-4
* parameter[=].resource.component[=].valueCodeableConcept.coding.display = "Responsive"
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #51963-7
* parameter[=].resource.component[=].code.coding.display = "Medication assessed"
* parameter[=].resource.component[=].valueCodeableConcept.coding.system = "http://www.nlm.nih.gov/research/umls/rxnorm"
* parameter[=].resource.component[=].valueCodeableConcept.coding.code = #480167
* parameter[=].resource.component[=].valueCodeableConcept.coding.display = "Lapatinib"
* parameter[+].name = "variant"
* parameter[=].resource.resourceType = "Observation"
* parameter[=].resource.id = "variantExample"
* parameter[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* parameter[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Variant</div>"
* parameter[=].resource.text.status = #generated
* parameter[=].resource.status = #final
* parameter[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].resource.code.coding.system = "http://loinc.org"
* parameter[=].resource.code.coding.code = #69548-6
* parameter[=].resource.code.coding.display = "Genetic variant assessment"
* parameter[=].resource.subject.reference = "Patient/HG00403"
* parameter[=].resource.effectiveDateTime = "2023-07-01"
* parameter[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter[=].resource.valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].resource.valueCodeableConcept.coding.code = #LA9633-4
* parameter[=].resource.valueCodeableConcept.coding.display = "Present"
* parameter[=].resource.component[0].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #48002-0
* parameter[=].resource.component[=].code.coding.display = "Genomic source class [Type]"
* parameter[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].valueCodeableConcept.coding.code = #LA6684-0
* parameter[=].resource.component[=].valueCodeableConcept.coding.display = "Somatic"
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #48013-7
* parameter[=].resource.component[=].code.coding.display = "Genomic reference sequence ID"
* parameter[=].resource.component[=].valueCodeableConcept.coding.system = "http://www.ncbi.nlm.nih.gov/refseq"
* parameter[=].resource.component[=].valueCodeableConcept.coding.code = #NC_000017.11
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #69547-8
* parameter[=].resource.component[=].code.coding.display = "Genomic Ref allele [ID]"
* parameter[=].resource.component[=].valueString = "ATTGAGGGAAAACACAT"
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #69551-0
* parameter[=].resource.component[=].code.coding.display = "Genomic Alt allele [ID]"
* parameter[=].resource.component[=].valueString = "T"
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #92822-6
* parameter[=].resource.component[=].code.coding.display = "Genomic coord system"
* parameter[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].valueCodeableConcept.coding.code = #LA30100-4
* parameter[=].resource.component[=].valueCodeableConcept.coding.display = "0-based interval counting"
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #81254-5
* parameter[=].resource.component[=].code.text = "Genomic allele start-end"
* parameter[=].resource.component[=].valueRange.low.value = 39723965

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-subject-dx-implications.html

Instance: FindSubjectDxImplicationsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindSubjectDxImplications' Operation"
Usage: #example
* parameter.name = "implication"
* parameter[=].resource.resourceType = "Observation"
* parameter[=].resource.id = "DiagnosticImplicationExample"
* parameter[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/diagnostic-implication"
* parameter[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Pathogenic - Malignant hyperthermia</div>"
* parameter[=].resource.text.status = #generated
* parameter[=].resource.status = #final
* parameter[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].resource.code.coding.system = "http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs"
* parameter[=].resource.code.coding.code = #diagnostic-implication
* parameter[=].resource.subject.reference = "Patient/HG00403"
* parameter[=].resource.effectiveDateTime = "2023-04-01"
* parameter[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter[=].resource.derivedFrom.reference = "Observation/VariantExample2"
* parameter[=].resource.component[0].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #53037-8
* parameter[=].resource.component[=].code.text = "Genetic variation clinical significance"
* parameter[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].valueCodeableConcept.coding.code = #LA6668-3
* parameter[=].resource.component[=].valueCodeableConcept.coding.display = "Pathogenic"
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #81259-4
* parameter[=].resource.component[=].code.text = "Predicted phenotype"
* parameter[=].resource.component[=].valueCodeableConcept.coding.system = "http://snomed.info/sct"
* parameter[=].resource.component[=].valueCodeableConcept.coding.code = #405501007
* parameter[=].resource.component[=].valueCodeableConcept.coding.display = "Malignant hyperthermia"
* parameter[+].name = "variant"
* parameter[=].resource.resourceType = "Observation"
* parameter[=].resource.id = "variantExample2"
* parameter[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* parameter[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>NC_000019.10:g.38499669C>T</div>"
* parameter[=].resource.text.status = #generated
* parameter[=].resource.status = #final
* parameter[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].resource.code.coding.system = "http://loinc.org"
* parameter[=].resource.code.coding.code = #69548-6
* parameter[=].resource.code.coding.display = "Genetic variant assessment"
* parameter[=].resource.subject.reference = "Patient/HG00403"
* parameter[=].resource.effectiveDateTime = "2023-04-01"
* parameter[=].resource.performer.reference = "Organization/ExampleOrg"
* parameter[=].resource.valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].resource.valueCodeableConcept.coding.code = #LA9633-4
* parameter[=].resource.valueCodeableConcept.coding.display = "Present"
* parameter[=].resource.component[0].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #48002-0
* parameter[=].resource.component[=].code.coding.display = "Genomic source class [Type]"
* parameter[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].valueCodeableConcept.coding.code = #LA6683-2
* parameter[=].resource.component[=].valueCodeableConcept.coding.display = "Germline"
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #48013-7
* parameter[=].resource.component[=].code.coding.display = "Genomic reference sequence ID"
* parameter[=].resource.component[=].valueCodeableConcept.coding.system = "http://www.ncbi.nlm.nih.gov/refseq"
* parameter[=].resource.component[=].valueCodeableConcept.coding.code = #NC_000019.10
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #69547-8
* parameter[=].resource.component[=].code.coding.display = "Genomic Ref allele [ID]"
* parameter[=].resource.component[=].valueString = "C"
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #69551-0
* parameter[=].resource.component[=].code.coding.display = "Genomic Alt allele [ID]"
* parameter[=].resource.component[=].valueString = "T"
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #92822-6
* parameter[=].resource.component[=].code.coding.display = "Genomic coord system"
* parameter[=].resource.component[=].valueCodeableConcept.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].valueCodeableConcept.coding.code = #LA30100-4
* parameter[=].resource.component[=].valueCodeableConcept.coding.display = "0-based interval counting"
* parameter[=].resource.component[+].code.coding.system = "http://loinc.org"
* parameter[=].resource.component[=].code.coding.code = #81254-5
* parameter[=].resource.component[=].code.text = "Genomic allele start-end"
* parameter[=].resource.component[=].valueRange.low.value = 38499669

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-subject-molec-conseq.html

Instance: FindSubjectMolecConseqOutput
InstanceOf: Parameters
Description: "Example Output of 'FindSubjectMolecConseq' Operation"
Usage: #example
* parameter.name = "consequence"
* parameter[=].resource.resourceType = "Observation"
* parameter[=].resource.id = "MC1"
* parameter[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-consequence"
* parameter[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>NM_001395525.1:c.-281+2T>C - intronic_variant:loss_of_function_variant</div>"
* parameter[=].resource.text.status = #generated
* parameter[=].resource.status = #final
* parameter[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].resource.code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* parameter[=].resource.subject = Reference(Patient/HG00403)
* parameter[=].resource.effectiveDateTime = "2023-04-01"
* parameter[=].resource.performer = Reference(Organization/ExampleOrg)
* parameter[=].resource.derivedFrom = Reference(Observation/variantExampleMC)
* parameter[=].resource.interpretation = http://example.org/pcingola.github.io/SnpEff/se_inputoutput/\#impact-prediction#HIGH "High"
//kp - Error while processing 'NM_001395525.1:c.-281+2T>C': Intronic variants are not yet supported
// * parameter[=].resource.component[0].code = http://loinc.org#48004-6
// * parameter[=].resource.component[0].code.text = "coding HGVS"
// * parameter[=].resource.component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_001395525.1:c.-281+2T>C "NM_001395525.1:c.-281+2T>C"
* parameter[=].resource.component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* parameter[=].resource.component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_001395525.1 "NM_001395525.1"
* parameter[=].resource.component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* parameter[=].resource.component[=].valueCodeableConcept.coding[0] = http://www.sequenceontology.org#SO:0001575 "splice_donor_variant"
* parameter[=].resource.component[=].valueCodeableConcept.coding[1] = http://www.sequenceontology.org#SO:0001627 "intron_variant"
* parameter[=].resource.component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#functional-effect
* parameter[=].resource.component[=].valueCodeableConcept = http://www.sequenceontology.org#SO_0002054 "loss_of_function_variant"
* parameter[+].name = "consequence"
* parameter[=].resource.resourceType = "Observation"
* parameter[=].resource.id = "MC2"
* parameter[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-consequence"
* parameter[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>NM_001366781.1:c.90T>C - synonymous_variant</div>"
* parameter[=].resource.text.status = #generated
* parameter[=].resource.status = #final
* parameter[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].resource.code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* parameter[=].resource.subject = Reference(Patient/HG00403)
* parameter[=].resource.effectiveDateTime = "2023-04-01"
* parameter[=].resource.performer = Reference(Organization/ExampleOrg)
* parameter[=].resource.derivedFrom = Reference(Observation/variantExampleMC)
* parameter[=].resource.interpretation = http://example.org/pcingola.github.io/SnpEff/se_inputoutput/\#impact-prediction#LOW "Low"
* parameter[=].resource.component[0].code = http://loinc.org#48004-6
* parameter[=].resource.component[0].code.text = "coding HGVS"
* parameter[=].resource.component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_001366781.1:c.90T>C "NM_001366781.1:c.90T>C"
// * parameter[=].resource.component[+].code = http://loinc.org#48005-3 "Amino acid change (pHGVS)"
// * parameter[=].resource.component[=].valueCodeableConcept = http://varnomen.hgvs.org#p.Cys30Cys "p.Cys30Cys"
* parameter[=].resource.component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* parameter[=].resource.component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_001366781.1 "NM_001366781.1"
* parameter[=].resource.component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* parameter[=].resource.component[=].valueCodeableConcept = http://www.sequenceontology.org#SO:0001819 "synonymous_variant"
* parameter[+].name = "consequence"
* parameter[=].resource.resourceType = "Observation"
* parameter[=].resource.id = "MC3"
* parameter[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-consequence"
* parameter[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Molecular Consequence</div>"
* parameter[=].resource.text.status = #generated
* parameter[=].resource.status = #final
* parameter[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].resource.code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* parameter[=].resource.subject = Reference(Patient/HG00403)
* parameter[=].resource.effectiveDateTime = "2023-04-01"
* parameter[=].resource.performer = Reference(Organization/ExampleOrg)
* parameter[=].resource.derivedFrom = Reference(Observation/variantExampleMC)
* parameter[=].resource.interpretation = http://example.org/pcingola.github.io/SnpEff/se_inputoutput/\#impact-prediction#MOD "Modifier"
* parameter[=].resource.component[0].code = http://loinc.org#48004-6
* parameter[=].resource.component[0].code.text = "coding HGVS"
* parameter[=].resource.component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_001395543.1:c.-171T>C "NM_001395543.1:c.-171T>C"
* parameter[=].resource.component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* parameter[=].resource.component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_001395543.1 "NM_001395543.1"
* parameter[=].resource.component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* parameter[=].resource.component[=].valueCodeableConcept = http://www.sequenceontology.org#SO:0001623 "5_prime_UTR_variant"
* parameter[+].name = "consequence"
* parameter[=].resource.resourceType = "Observation"
* parameter[=].resource.id = "MC4"
* parameter[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-consequence"
* parameter[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Molecular Consequence</div>"
* parameter[=].resource.text.status = #generated
* parameter[=].resource.status = #final
* parameter[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].resource.code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* parameter[=].resource.subject = Reference(Patient/HG00403)
* parameter[=].resource.effectiveDateTime = "2023-04-01"
* parameter[=].resource.performer = Reference(Organization/ExampleOrg)
* parameter[=].resource.derivedFrom = Reference(Observation/variantExampleMC)
* parameter[=].resource.interpretation = http://example.org/pcingola.github.io/SnpEff/se_inputoutput/\#impact-prediction#MOD "Modifier"
//kp - Error while processing 'NM_001395525.1:c.-281+2T>C': Intronic variants are not yet supported
// * parameter[=].resource.component[0].code = http://loinc.org#48004-6
// * parameter[=].resource.component[0].code.text = "coding HGVS"
// * parameter[=].resource.component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_001395544.1:c.-21-3742T>C "NM_001395544.1:c.-21-3742T>C"
* parameter[=].resource.component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* parameter[=].resource.component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_001395544.1 "NM_001395544.1"
* parameter[=].resource.component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* parameter[=].resource.component[=].valueCodeableConcept = http://www.sequenceontology.org#SO:0001627 "intron_variant"
* parameter[+].name = "variant"
* parameter[=].resource.resourceType = "Observation"
* parameter[=].resource.id = "variantExampleMC"
* parameter[=].resource.meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* parameter[=].resource.text.div = "<div xmlns='http://www.w3.org/1999/xhtml'>Variant</div>"
* parameter[=].resource.text.status = #generated
* parameter[=].resource.status = #final
* parameter[=].resource.category[+].coding = $OBSCAT#laboratory
* parameter[=].resource.category[+].coding = $DIAGNOSTICSERVICE#GE
* parameter[=].resource.code = $LNC#69548-6 "Genetic variant assessment"
* parameter[=].resource.subject = Reference(Patient/HG00403)
* parameter[=].resource.effectiveDateTime = "2023-04-01"
* parameter[=].resource.performer = Reference(Organization/ExampleOrg)
* parameter[=].resource.valueCodeableConcept = $LNC#LA9633-4 "Present"
* parameter[=].resource.component[+].code = $LNC#48002-0 "Genomic source class"
* parameter[=].resource.component[=].valueCodeableConcept.coding = $LNC#LA6683-2 "Germline"
* parameter[=].resource.component[+].code = http://loinc.org#48013-7 "Genomic reference sequence ID"
* parameter[=].resource.component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NC_000001.10
* parameter[=].resource.component[+].code = $LNC#53034-5 "Allelic state"
* parameter[=].resource.component[=].valueCodeableConcept = $LNC#LA6706-1 "Heterozygous"
* parameter[=].resource.component[+].code = http://loinc.org#81252-9 "Discrete genetic variant"
* parameter[=].resource.component[=].valueCodeableConcept = $HGVS#NC_000001.10:g.86852621A>G "NC_000001.10:g.86852621A>G"
* parameter[=].resource.component[+].code = http://loinc.org#81258-6 "Sample VAF"
* parameter[=].resource.component[=].valueQuantity = 0.6 '1' "relative frequency of a particular allele in the specimen"
* parameter[=].resource.component[+].code = http://loinc.org#69547-8 "Genomic Ref allele [ID]"
* parameter[=].resource.component[=].valueString = "A"
* parameter[=].resource.component[+].code = http://loinc.org#69551-0 "Genomic Alt allele [ID]"
* parameter[=].resource.component[=].valueString = "G"
* parameter[=].resource.component[+].code = http://loinc.org#92822-6 "Genomic coord system"
* parameter[=].resource.component[=].valueCodeableConcept = http://loinc.org#LA30100-4 "0-based interval counting"
* parameter[=].resource.component[+].code = http://loinc.org#81254-5 "Genomic allele start-end"
* parameter[=].resource.component[=].valueRange.low.value = 86852620

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-population-specific-variants.html

Instance: FindANYPopulationSpecificVariantsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindPopulationSpecificVariants' Operation, based on 'OR' logic"
Usage: #example
* parameter[0].name = "variants"
* parameter[=].part[0].name = "variantItem"
* parameter[=].part[=].valueString = "NC_000010.10:g.121551307G>A"
* parameter[=].part[+].name = "numerator"
* parameter[=].part[=].valueQuantity.value = 7
* parameter[+].name = "variants"
* parameter[=].part[0].name = "variantItem"
* parameter[=].part[=].valueString = "NC_000011.10:g.8263343T>C"
* parameter[=].part[+].name = "numerator"
* parameter[=].part[=].valueQuantity.value = 1
* parameter[+].name = "variants"
* parameter[=].part[0].name = "variantItem"
* parameter[=].part[=].valueString = "NC_000001.10:g.198742263G>A"
* parameter[=].part[+].name = "numerator"
* parameter[=].part[=].valueQuantity.value = 3

Instance: FindALLPopulationSpecificVariantsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindPopulationSpecificVariants' Operation, based on 'AND' logic"
Usage: #example
* parameter.name = "variants"
* parameter.part[0].name = "variantItem"
* parameter.part[=].valueString = "NC_000010.10:g.121551307G>A AND NC_000011.10:g.8263343T>C AND NC_000001.10:g.198742263G>A"
* parameter.part[+].name = "numerator"
* parameter.part[=].valueQuantity.value = 2

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-population-structural-intersecting-variants.html

Instance: FindPopulationStructuralIntersectingVariantsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindPopulationStructuralIntersectingVariants' Operation"
Usage: #example
* parameter[0].name = "variants"
* parameter[=].part[0].name = "rangeItem"
* parameter[=].part[=].valueString = "NC_000017.11:43044294-43125364"
* parameter[=].part[+].name = "numerator"
* parameter[=].part[=].valueQuantity.value = 17
* parameter[=].part[+].name = "subject"
* parameter[=].part[=].valueString = "patient/12345"
* parameter[=].part[+].name = "subject"
* parameter[=].part[=].valueString = "patient/23456"
* parameter[=].part[+].name = "subject"
* parameter[=].part[=].valueString = "..."
* parameter[+].name = "variants"
* parameter[=].part[0].name = "rangeItem"
* parameter[=].part[=].valueString = "NC_000013.11:32315507-32400268"
* parameter[=].part[+].name = "numerator"
* parameter[=].part[=].valueQuantity.value = 4
* parameter[=].part[+].name = "subject"
* parameter[=].part[=].valueString = "patient/34567"
* parameter[=].part[+].name = "subject"
* parameter[=].part[=].valueString = "patient/45678"
* parameter[=].part[+].name = "subject"
* parameter[=].part[=].valueString = "..."

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-population-structural-subsuming-variants.html

Instance: FindPopulationStructuralSubsumingVariantsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindPopulationStructuralSubsumingVariants' Operation"
Usage: #example
* parameter.name = "variants"
* parameter.part[0].name = "rangeItem"
* parameter.part[=].valueString = "NC_000022.10:42522500-42526812"
* parameter.part[+].name = "numerator"
* parameter.part[=].valueQuantity.value = 113
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "patient/12345"
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "patient/23456"
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "..."

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-population-specific-haplotypes.html

Instance: FindPopulationSpecificHaplotypesOutput
InstanceOf: Parameters
Description: "Example Output of 'FindPopulationSpecificHaplotypes' Operation"
Usage: #example
* parameter.name = "haplotypes"
* parameter.part[0].name = "haplotypeItem"
* parameter.part[=].valueString = "A*1 and B*8 and DRB1*03 and DRB3*01 and A*31 and  B*35 and DRB1*04 and DRB4*01:02"
* parameter.part[+].name = "numerator"
* parameter.part[=].valueQuantity.value = 0
* parameter.part[+].name = "denominator"
* parameter.part[=].valueQuantity.value = 7534

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-population-tx-implications.html

Instance: FindPopulationTxImplicationsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindPopulationTxImplications' Operation"
Usage: #example
* parameter.name = "implications"
* parameter.part[0].name = "numerator"
* parameter.part[=].valueQuantity.value = 23
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "patient/12345"
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "patient/23456"
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "..."

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-population-dx-implications.html

Instance: FindPopulationDxImplicationsOutput
InstanceOf: Parameters
Description: "Example Output of 'FindPopulationDxImplications' Operation"
Usage: #example
* parameter.name = "implications"
* parameter.part[0].name = "numerator"
* parameter.part[=].valueQuantity.value = 23
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "patient/12345"
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "patient/23456"
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "..."

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-population-molec-conseq.html

Instance: FindPopulationMolecConseqOutput
InstanceOf: Parameters
Description: "Example Output of 'FindPopulationMolecConseq' Operation"
Usage: #example
* parameter.name = "consequences"
* parameter.part[0].name = "numerator"
* parameter.part[=].valueQuantity.value = 23
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "patient/12345"
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "patient/23456"
* parameter.part[+].name = "subject"
* parameter.part[=].valueString = "..."

//http://build.fhir.org/ig/HL7/genomics-reporting/OperationDefinition-find-study-metadata.html

Instance: FindStudyMetadataOutput
InstanceOf: Parameters
Description: "Example Output of 'FindStudyMetadata' Operation"
Usage: #example
* parameter.name = "tests"
* parameter.part[0].name = "testId"
* parameter.part[=].valueString = "test08736abc87"
* parameter.part[+].name = "testDate"
* parameter.part[=].valueDateTime = "2020-07-13"
* parameter.part[+].name = "specimenId"
* parameter.part[=].valueString = "specimen9652adbg69"
* parameter.part[+].name = "genomicBuild"
* parameter.part[=].valueCodeableConcept = http://loinc.org#LA26806-2 "GRCh38"
* parameter.part[+].name = "dnaChangeType"
* parameter.part[=].valueCodeableConcept.coding[0] = http://www.sequenceontology.org#SO:0001483 "SNV"
* parameter.part[=].valueCodeableConcept.coding[+] = http://www.sequenceontology.org#SO:0002007 "MNV"
* parameter.part[=].valueCodeableConcept.coding[+] = http://www.sequenceontology.org#SO:1000032 "delins"
* parameter.part[=].valueCodeableConcept.coding[+] = http://www.sequenceontology.org#SO:0001019 "copy_number_variation"
* parameter.part[+].name = "regionStudied"
* parameter.part[=].valueString = "['NC_000007.14:55019016-55019365', 'NC_000007.14:55142285-55142437', 'NC_000007.14:55143304-55143488', 'NC_000007.14:55146605-55146740', 'NC_000007.14:55151293-55151362', 'NC_000007.14:55152545-55152664', 'NC_000007.14:55154010-55154152', 'NC_000007.14:55155829-55155946', 'NC_000007.14:55156532-55156659', 'NC_000007.14:55156758-55156832', 'NC_000007.14:55157662-55157753', 'NC_000007.14:55160138-55160338', 'NC_000007.14:55161498-55161631', 'NC_000007.14:55163732-55163823', 'NC_000007.14:55165279-55165437', 'NC_000007.14:55171174-55171213', 'NC_000007.14:55172982-55173124', 'NC_000007.14:55173920-55174043', 'NC_000007.14:55174721-55174820', 'NC_000007.14:55181292-55181478', 'NC_000007.14:55191718-55191874', 'NC_000007.14:55192765-55192841', 'NC_000007.14:55198716-55198863', 'NC_000007.14:55200315-55200413', 'NC_000007.14:55201187-55201355', 'NC_000007.14:55201734-55201782', 'NC_000007.14:55202516-55202625', 'NC_000007.14:55205255-55211628']"
* parameter.part[+].name = "uncallableRegions"
* parameter.part[=].valueString = "['NC_000007.14:55173910-55174053', 'NC_000007.14:55174711-55174830', 'NC_000007.14:55181282-55181488']"


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/diagnosticreport-hla-glstring-r4.fsh

Instance: diagnosticreport-hla-glstring-r4 
InstanceOf: GenomicReport
Description: "Example GenomicReport including an HLA glstring"
Usage: #example
* basedOn[0] = Reference(servicerequest-hla-a-r4)
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE "Genetics"
* code.coding[0] = $LNC#51969-4 "Genetic analysis report"
* code.coding[+] = $HGNCID#HGNC:588 "Histocompatibility complex (HLA)"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2023-06-01"
* performer = Reference(ExampleLab)
* specimen = Reference(specimen-hla-r4)
* result[0] = Reference(genotype-hla-a-glstring-r4)
* result[+] = Reference(haplotype-hla-a-1-r4)

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/ExampleReusableResources.fsh

Instance: ExamplePatient
InstanceOf: Patient
Description: "Patient example"
* text.status = #additional
* text.div = "<div xmlns=\"http://www.w3.org/1999/xhtml\">This would contain patient identifiers, demographics, etc.</div>"

Instance: ExampleSpecimen
InstanceOf: Specimen
Description: "Specimen example"
* subject = Reference(ExamplePatient)

Instance: ExampleLab
InstanceOf: Organization
Description: "Organization (lab) example"
* name = "Some lab"

Instance: ExampleServiceRequest
InstanceOf: ServiceRequest
Description: "ServiceRequest (order) example"
* status = #active
* intent = #original-order
* code.coding = $LNC#51969-4 "Genetic analysis report"
* subject = Reference(ExamplePatient)
* reasonCode.text = "Worried about family planning"

Instance: MedicationStatementWarfarin
InstanceOf: MedicationStatement
Description: "MedicationStatement for Warfarin"
* status = #unknown
* medicationCodeableConcept.coding = $RXN#11289 "Warfarin"
* subject = Reference(ExamplePatient)


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/genomic-study-examples.fsh

Instance: lungMass
InstanceOf: GenomicStudy
Description: "Example of solid tumor whole exome sequencing panel"
Usage: #example
* extension[GenomicStudyAnalysisExt][+].valueReference = Reference(lungMass-analysis1)
* extension[GenomicStudyAnalysisExt][+].valueReference = Reference(lungMass-analysis2)
* status = #completed
* code.text = "Solid tumor whole exome sequencing panel"
* category = $OBSCAT#laboratory
* subject = Reference(Patient/genomicPatient)
* performedDateTime = "2019-03-01"
* basedOn = Reference(ServiceRequest/genomicServiceRequest)
* asserter = Reference(Practitioner/practitioner02)
* reasonCode = $SCT#309529002 "Lung mass (finding)"
* note.text = "For technical reasons, PIK3CB was deemed uncallable."

Instance: lungMass-analysis1
InstanceOf: GenomicStudyAnalysis
Description: "Example of Sequence analysis of the entire coding region"
Usage: #example
* extension[GenomicStudyAnalysisMethodType][+].valueCodeableConcept = $GSMETHODTYPE#sequence-analysis-of-the-entire-coding-region "Sequence analysis of the entire coding region"
* extension[GenomicStudyAnalysisGenomicSourceClass].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* extension[GenomicStudyAnalysisChangeType][+].valueCodeableConcept = $SEQONT#SO:0001483 "SNV"
* extension[GenomicStudyAnalysisChangeType][+].valueCodeableConcept = $SEQONT#SO:0002007 "MNV"
* extension[GenomicStudyAnalysisChangeType][+].valueCodeableConcept = $SEQONT#SO:1000032 "delins"
* extension[GenomicStudyAnalysisGenomeBuild][+].valueCodeableConcept = $LNC#LA26806-2 "GRCh38"
* extension[GenomicStudyAnalysisSpecimen][+].valueReference = Reference(Specimen/genomicSpecimen)
* extension[GenomicStudyAnalysisFocus][+].valueReference = Reference(Patient/genomicPatient)
* extension[GenomicStudyAnalysisDevice][+].extension[device].valueReference = Reference(Device/triodenovo-software)
* extension[GenomicStudyAnalysisRegions]
  * extension[studied][+].valueReference = Reference(DocumentReference/WES-FullSequencedRegion-GRCh38)
  * extension[uncalled][+].valueReference = Reference(DocumentReference/UncallableRegions) 
  * extension[called][+].valueReference = Reference(DocumentReference/SimpleVariantAnalysis-called)
* extension[GenomicStudyAnalysisOutput][+].extension[type].valueCodeableConcept = $GSFILETYPE#vcf "VCF"
* extension[GenomicStudyAnalysisOutput][=].extension[file].valueReference = Reference(DocumentReference/genomicVCFfile-simple)
* instantiatesUri = "https://pubmed.ncbi.nlm.nih.gov/33927380/"
* status = #completed
* category = $OBSCAT#laboratory
* performedDateTime = "2019-03-01T01:01:10-06:00"
* note.text = "For technical reasons, PIK3CB was deemed uncallable using this method."
* performer.actor = Reference(Practitioner/practitioner02)
* performer.function = $PARTICIPATIONTYPE#PRF "Performer"
* subject = Reference(Patient/genomicPatient)

Instance: lungMass-analysis2
InstanceOf: GenomicStudyAnalysis
Description: "Example of Deletion/duplication analysis"
Usage: #example
* extension[GenomicStudyAnalysisMethodType][+].valueCodeableConcept = $GSMETHODTYPE#deletion-duplication-analysis "Deletion/duplication analysis"
* extension[GenomicStudyAnalysisGenomicSourceClass].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* extension[GenomicStudyAnalysisChangeType][+].valueCodeableConcept = $SEQONT#SO:0001019 "CNV"
* extension[GenomicStudyAnalysisGenomeBuild][+].valueCodeableConcept = $LNC#LA26806-2 "GRCh38"
* extension[GenomicStudyAnalysisSpecimen][+].valueReference = Reference(Specimen/genomicSpecimen)
* extension[GenomicStudyAnalysisFocus][+].valueReference = Reference(Patient/genomicPatient)
* extension[GenomicStudyAnalysisRegions]
  * extension[studied][+].valueReference = Reference(DocumentReference/WES-FullSequencedRegion-GRCh38)
  * extension[called][+].valueReference = Reference(DocumentReference/CNVAnalysis-called)
* extension[GenomicStudyAnalysisOutput][+].extension[type].valueCodeableConcept = $GSFILETYPE#vcf "VCF"
* extension[GenomicStudyAnalysisOutput][=].extension[file].valueReference = Reference(DocumentReference/genomicVCFfile-cnv)
* instantiatesUri = "https://pubmed.ncbi.nlm.nih.gov/33927380/"
* status = #completed
* category = $OBSCAT#laboratory
* performedDateTime = "2019-03-01T01:01:10-06:00"
* note.text = "For technical reasons, PIK3CB was deemed uncallable using this method."
* performer.actor = Reference(Practitioner/practitioner02)
* performer.function = $PARTICIPATIONTYPE#PRF "Performer"
* subject = Reference(Patient/genomicPatient)

Instance: genomicSpecimen
InstanceOf: Specimen
Description: "Example of lung specimen obtained by biopsy"
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/specimens"
* identifier.value = "4"
* status = #available
* type = $SCT#122610009 "Specimen from lung obtained by biopsy (specimen)"
* subject = Reference(Patient/genomicPatient) "John Doe"
* receivedTime = "2019-03-01T01:01:01Z"
* request = Reference(ServiceRequest/genomicServiceRequest)
* collection.collector = Reference(Practitioner/practitioner02)
* collection.collectedDateTime = "2019-03-01T01:01:00Z"
* collection.quantity.value = 1
* collection.quantity.unit = "mm2"
* collection.method = $SCT#129314006 "Biopsy - action"

Instance: genomicPatient
InstanceOf: Patient
Description: "Example patient"
Usage: #example
* identifier.use = #temp
* identifier.type = $IDTYPE#MR "Medical record number"
* identifier.system = "http://www.somesystemabc.net/identifiers/persons"
* identifier.value = "11117"
* identifier.period.start = "2021-01-01"
* identifier.assigner.display = "General Hospital"
* active = true
* name.use = #official
* name.family = "Doe"
* name.given[0] = "John"
* name.given[+] = "Junior"
* gender = #unknown
* birthDate = "2001-01-01"

Instance: practitioner02
InstanceOf: Practitioner
Description: "Example practitioner"
Usage: #example
* identifier.use = #temp
* identifier.type = $IDTYPE#PRN "Provider number"
* identifier.system = "http://www.somesystemabc.net/identifiers/persons"
* identifier.value = "11116"
* identifier.period.start = "2021-01-01"
* identifier.assigner.display = "Child Hospital"
* active = true
* name.family = "Doel"
* name.given = "Jane"
* name.prefix = "Dr"

Instance: genomicServiceRequest
InstanceOf: ServiceRequest
Description: "Example of Molecular genetic test"
Usage: #example
* identifier.type = $IDTYPE#LACSN
* identifier.type.text = "Laboratory Accession ID"
* identifier.system = "http://www.somesystemabc.net/identifiers/serviceRequests"
* identifier.value = "111111112"
* status = #active
* intent = #plan
* code = $SCT#405825005 "Molecular genetic test (procedure)"
* subject = Reference(Patient/genomicPatient)

Instance: WES-FullSequencedRegion-GRCh38
InstanceOf: GenomicDataFile
Description: "WES_FullSequencedRegion_GRCh38: A sample Document Reference instance representing a BED file that may be used as input or output of a genomic analysis pipeline."
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/files"
* identifier.value = "11117"
* status = #current
* docStatus = #preliminary
* description = "WES_FullSequencedRegion_GRCh38: A sample Document Reference instance representing a BED file that may be used as input or output of a genomic analysis pipeline."
* securityLabel = $CONFIDENTIALITY#R "Restricted"
* content.attachment.url = "https://chat.fhir.org/user_uploads/10155/BILYJerATC59WlG15J316BEf/WES_FullSequencedRegion_GRCh38.bed"
* content.attachment.title = "WES_FullSequencedRegion_GRCh38"
* content.attachment.creation = "2019-03-01T01:02:01+01:00"

Instance: CNVAnalysis-called
InstanceOf: GenomicDataFile
Description: "CNVAnalysis_called: A sample Document Reference instance representing a BED file that may be used as input or output of a genomic analysis pipeline."
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/files"
* identifier.value = "11120"
* status = #current
* docStatus = #preliminary
* description = "CNVAnalysis_called: A sample Document Reference instance representing a BED file that may be used as input or output of a genomic analysis pipeline."
* securityLabel = $CONFIDENTIALITY#R "Restricted"
* content.attachment.url = "https://chat.fhir.org/user_uploads/10155/Hnv3LtumKn-1QjeyS2KVuw4R/CNVAnalysis_called.bed"
* content.attachment.title = "CNVAnalysis_called"
* content.attachment.creation = "2019-03-01T01:02:01+01:00"

Instance: genomicVCFfile-cnv
InstanceOf: GenomicDataFile
Description: "genomicVCFfile_cnv: A sample Document Reference instance representing a VCF file that may be used as input or output of a genomic analysis pipeline."
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/files"
* identifier.value = "11121"
* status = #current
* docStatus = #preliminary
* description = "genomicVCFfile_cnv: A sample Document Reference instance representing a VCF file that may be used as input or output of a genomic analysis pipeline."
* securityLabel = $CONFIDENTIALITY#R "Restricted"
* content.attachment.url = "https://chat.fhir.org/user_uploads/10155/dxn3s6R1Y-Cv0h-1bTCu-JvY/genomicVCFFile_cnv.vcf"
* content.attachment.title = "genomicVCFfile_cnv"
* content.attachment.creation = "2019-03-01T01:02:01+01:00"

Instance: genomicVCFfile-simple
InstanceOf: GenomicDataFile
Description: "genomicVCFfile_simple: A sample Document Reference instance representing a VCF file that may be used as input or output of a genomic analysis pipeline."
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/files"
* identifier.value = "11119"
* status = #current
* docStatus = #preliminary
* description = "genomicVCFfile_simple: A sample Document Reference instance representing a VCF file that may be used as input or output of a genomic analysis pipeline."
* securityLabel = $CONFIDENTIALITY#R "Restricted"
* content.attachment.url = "https://chat.fhir.org/user_uploads/10155/gdL90-np7lJjGwUxeARapUHB/genomicVCFFile_simple.vcf"
* content.attachment.title = "genomicVCFfile_simple"
* content.attachment.creation = "2019-03-01T01:02:01+01:00"

Instance: SimpleVariantAnalysis-called
InstanceOf: GenomicDataFile
Description: "SimpleVariantAnalysis_called: A sample Document Reference instance representing a BED file that may be used as input or output of a genomic analysis pipeline."
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/files"
* identifier.value = "11118"
* status = #current
* docStatus = #preliminary
* description = "SimpleVariantAnalysis_called: A sample Document Reference instance representing a BED file that may be used as input or output of a genomic analysis pipeline."
* securityLabel = $CONFIDENTIALITY#R "Restricted"
* content.attachment.url = "https://chat.fhir.org/user_uploads/10155/jQZWWKs4JO8ZfhfrbsCbbMUA/SimpleVariantAnalysis_called.bed"
* content.attachment.title = "SimpleVariantAnalysis_called"
* content.attachment.creation = "2019-03-01T01:02:01+01:00"

Instance: UncallableRegions
InstanceOf: GenomicDataFile
Description: "Representing a BED file that represents uncallable regions"
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/files"
* identifier.value = "11125"
* status = #current
* docStatus = #preliminary
* description = "Representing a BED file that represents uncallable regions"
* securityLabel = $CONFIDENTIALITY#R "Restricted"
* content.attachment.url = "https://chat.fhir.org/user_uploads/10155/jQZWWKs4JO8ZfhfrbsCbbMUA/uncallable_regions.bed"
* content.attachment.title = "Uncalleable Regions"
* content.attachment.creation = "2019-03-01T01:02:01+01:00"


Instance: genomicstudy-trio2
InstanceOf: GenomicStudy
Description: "GenomicStudy example of a trio analysis"
Usage: #example
* extension[GenomicStudyAnalysisExt][+].valueReference = Reference(genomicstudyanalysis-trio2)
* extension[GenomicStudyReferrerExt].valueReference = Reference(Practitioner/practitioner02)
* identifier[+]
  * use = #temp
  * system = "http://www.somesystemabc.net/identifiers/genomicstudies"
  * value = "urn:uuid:1111-1111-1111-1113"
* status = #completed
* code.text = "Trio Analysis"
* category = $OBSCAT#laboratory
* subject = Reference(Patient/denovoChild)
* performedDateTime = "2023-10-01"
* asserter = Reference(Practitioner/practitioner02)
* note.text = "De novo mutation study of the patient. The Prenatal Trio Whole Exome Sequencing (Prenatal Trio WES) test is ordered by a physician and must be accompanied with a consent form and detailed clinical information. In general, the test is used when prenatal imaging detects an anomaly that strongly suggests that there is an underlying genetic etiology. Prenatal Trio WES is often considered only after fetal chromosome microarray analysis has been non-diagnostic."

Instance: genomicstudyanalysis-trio2
InstanceOf: GenomicStudyAnalysis
Description: "GenomicStudy Analysis example of a trio analysis"
Usage: #example
* extension[GenomicStudyAnalysisGenomeBuild][+].valueCodeableConcept = $LNC#LA26806-2 "GRCh38"
* extension[GenomicStudyAnalysisFocus][+].valueReference = Reference(Patient/denovoFather)
* extension[GenomicStudyAnalysisFocus][+].valueReference = Reference(Patient/denovoMother)
* extension[GenomicStudyAnalysisTitle].valueString = "Parental Sequence Variation Detection Using Next Generation Sequencing"
* extension[GenomicStudyAnalysisProtocolPerformed].valueReference = Reference(Procedure/SequencingProcedure)
* extension[GenomicStudyAnalysisInput][+]
  * extension[file].valueReference = Reference(DocumentReference/genomicFileProbandBAM)
  * extension[type].valueCodeableConcept = GenomicStudyDataFormatCS#bam "BAM"
* extension[GenomicStudyAnalysisInput][+]
  * extension[file].valueReference = Reference(DocumentReference/genomicFileMotherBAM)
  * extension[type].valueCodeableConcept = GenomicStudyDataFormatCS#bam "BAM"
* extension[GenomicStudyAnalysisInput][+]
  * extension[file].valueReference = Reference(DocumentReference/genomicFileFatherBAM)
  * extension[type].valueCodeableConcept = GenomicStudyDataFormatCS#bam "BAM"
* extension[GenomicStudyAnalysisOutput][+]
  * extension[file].valueReference = Reference(DocumentReference/genomicFileProbandVCF)
  * extension[type].valueCodeableConcept = GenomicStudyDataFormatCS#vcf "VCF"
* status = #completed
* category = $OBSCAT#laboratory
* performedDateTime = "2023-10-01"
* instantiatesUri = "https://pubmed.ncbi.nlm.nih.gov/33927380/"
* subject = Reference(Patient/denovoChild)

Instance: triodenovo-software
InstanceOf: Device
Description: "Triodenovo Software"
Usage: #example
* id = "triodenovo-software"
* identifier[+]
  * system = "http://www.somesystemabc.net/identifiers/devices"
  * value = "11112"
* status = #active
* manufacturer = "Vanderbilt Genetics Institute"
* deviceName[+]
  * name = "Triodenovo Software"
  * type = #user-friendly-name
* type = $SCT#706687001 "Software (physical object)"
* version[+].value = "0.06"

Instance: denovoChild
InstanceOf: Patient
Description: "Child of a trio"
Usage: #example
* identifier[+]
  * use = #temp
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#MR "Medical record number"
  * system = "http://www.somesystemabc.net/identifiers/persons"
  * value = "1234-11111"
* name[+]
  * use = #official
  * family = "Denovo"
  * given[+] = "Child"

Instance: denovoFather
InstanceOf: Patient
Description: "Father of a trio"
Usage: #example
* identifier[+]
  * use = #temp
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#MR "Medical record number"
  * system = "http://www.somesystemabc.net/identifiers/persons"
  * value = "6789-11111"
* name[+]
  * use = #official
  * family = "Denovo"
  * given[+] = "Father"

Instance: denovoMother
InstanceOf: Patient
Description: "Mother of a trio"
Usage: #example
* identifier[+]
  * use = #temp
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#MR "Medical record number"
  * system = "http://www.somesystemabc.net/identifiers/persons"
  * value = "0987-11111"
* name[+]
  * use = #official
  * family = "Denovo"
  * given[+] = "Mother"

Instance: genomicFileProbandBAM
InstanceOf: GenomicDataFile
Description: "A sample Document Reference instance representing a generic genomic file that may ber used as input or output of a genomic analysis pipeline"
Usage: #example
* identifier[+]
  * system = "http://www.somesystemabc.net/identifiers/files"
  * value = "11118"
* status = #current
* docStatus = #preliminary
* subject = Reference(Patient/denovoChild)
* description = "A sample Document Reference instance representing a generic genomic file that may ber used as input or output of a genomic analysis pipeline."
* securityLabel[+] = http://terminology.hl7.org/CodeSystem/v3-Confidentiality#R "Restricted"
* content[+].attachment
  * url = "http://www.somesystemabc.net/identifiers/files/11111"
  * title = "Probands BAM file"

Instance: genomicFileMotherBAM
InstanceOf: GenomicDataFile
Description: "A sample Document Reference instance representing a generic genomic file that may ber used as input or output of a genomic analysis pipeline"
Usage: #example
* identifier[+]
  * system = "http://www.somesystemabc.net/identifiers/files"
  * value = "11125"
* status = #current
* docStatus = #preliminary
* subject = Reference(Patient/denovoMother)
* description = "A sample Document Reference instance representing a generic genomic file that may ber used as input or output of a genomic analysis pipeline."
* securityLabel[+] = http://terminology.hl7.org/CodeSystem/v3-Confidentiality#R "Restricted"
* content[+].attachment
  * url = "http://www.somesystemabc.net/identifiers/files/11125"
  * title = "Mothers BAM file"

Instance: genomicFileFatherBAM
InstanceOf: GenomicDataFile
Description: "A sample Document Reference instance representing a generic genomic file that may ber used as input or output of a genomic analysis pipeline"
Usage: #example
* identifier[+]
  * system = "http://www.somesystemabc.net/identifiers/files"
  * value = "11132"
* status = #current
* docStatus = #preliminary
* subject = Reference(Patient/denovoFather)
* description = "A sample Document Reference instance representing a generic genomic file that may ber used as input or output of a genomic analysis pipeline."
* securityLabel[+] = http://terminology.hl7.org/CodeSystem/v3-Confidentiality#R "Restricted"
* content[+].attachment
  * url = "http://www.somesystemabc.net/identifiers/files/11132"
  * title = "Fathers BAM file"

Instance: genomicFileProbandVCF
InstanceOf: GenomicDataFile
Description: "A sample Document Reference instance representing a generic genomic file that may ber used as input or output of a genomic analysis pipeline"
Usage: #example
* identifier[+]
  * system = "http://www.somesystemabc.net/identifiers/files"
  * value = "11150"
* status = #current
* docStatus = #preliminary
* subject = Reference(Patient/denovoChild)
* description = "A sample Document Reference instance representing a generic genomic file that may ber used as input or output of a genomic analysis pipeline."
* securityLabel[+] = http://terminology.hl7.org/CodeSystem/v3-Confidentiality#R "Restricted"
* content[+].attachment
  * url = "http://www.somesystemabc.net/identifiers/files/11150"
  * title = "Probands VCF file"

Instance: SequencingProcedure
InstanceOf: Procedure
Description: "Very simple producure representing a genomic study's protocol"
Usage: #example
* status = #completed
* code = $SCT#443968007 "Sequencing of entire coding region of gene (procedure)"
* subject = Reference(Patient/denovoChild)
* performer.actor = Reference(Practitioner/practitioner02)


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/genotype-hla-a-glstring-r4.fsh

Instance: genotype-hla-a-glstring-r4
InstanceOf: Genotype
Description: "Genotype example: Sequence-based typing of HLA-A"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code.coding[0] = $LNC#84413-4 "Genotype display name"
* code.coding[+] = $LNC#57290-9 "HLA-A [Type] by High resolution"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2018-07-26"
* performer.type = "Organization"
* performer.display = "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "1.0"
* valueCodeableConcept.coding = $GLSTRING#hla#3.31.0#HLA-A*03:01:01:01+HLA-A*30:01:01
* method.text = "PACBIO Sequel"
* specimen = Reference(specimen-hla-r4)
* derivedFrom[0] = Reference(haplotype-hla-a-1-r4)
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4931 "HLA-A"

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/haplotype-hla-a-1-r4.fsh

Instance: haplotype-hla-a-1-r4
InstanceOf: Haplotype
Description: "Example Haplotype Sequence-based typing of HLA-A"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code.coding[0] = $LNC#84414-2 "Haplotype name"
* code.coding[+] = $LNC#57290-9 "HLA-A [Type] by High resolution"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2018-07-26"
* performer.type = "Organization"
* performer.display = "aTypingLab, Inc"
* valueCodeableConcept.coding.version = "3.31.0"
* valueCodeableConcept.coding = $HLAALLELE#HLA-A*03:01:01:01
* specimen = Reference(specimen-hla-r4)
* component.code = $LNC#48018-6 "Gene studied [ID]"
* component.valueCodeableConcept = $HGNCID#HGNC:4931 "HLA-A"
  * text = "HLA-A"

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/med-usage-impact-task-pgx-example.fsh

Instance: med-usage-impact-task-pgx-example
InstanceOf: MedicationRecommendation
Description: "Example MedicationRecommendation"
Usage: #inline
* status = #requested
* intent = #proposal
* code = $LNC#LA26423-6 "Increase dose"
  * text = "Any Increase does Recommendation Text from the provider"
* description = "Any Increase does Recommendation Text from the provider"
* focus = Reference(current-med-example)
* for = Reference(ExamplePatient)
* reasonReference = Reference(med-impact-pgx-example)

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/molec-conseq1.fsh

Instance: molec-conseq1
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence that includes Loss of Function prediction"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(Patient/HG00403)
* effectiveDateTime = "2023-06-01"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(Observation/variant-with-molec-consequences)
* interpretation = http://example.org/pcingola.github.io/SnpEff/se_inputoutput/\#impact-prediction#HIGH "High"
//KP - Error while processing 'NM_001395525.1:c.-281+2T>C': Intronic variants are not yet supported
// * component[0].code = http://loinc.org#48004-6
// * component[0].code.text = "coding HGVS"
// * component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_001395525.1:c.-281+2T>C "NM_001395525.1:c.-281+2T>C"
* component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_001395525.1 "NM_001395525.1"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept.coding[0] = http://www.sequenceontology.org#SO:0001575 "splice_donor_variant"
* component[=].valueCodeableConcept.coding[1] = http://www.sequenceontology.org#SO:0001627 "intron_variant"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#functional-effect
* component[=].valueCodeableConcept = http://www.sequenceontology.org#SO_0002054 "loss_of_function_variant"


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/molec-conseq2.fsh

Instance: molec-conseq2
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(Patient/HG00403)
* effectiveDateTime = "2023-06-01"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(Observation/variant-with-molec-consequences)
* interpretation = http://example.org/pcingola.github.io/SnpEff/se_inputoutput/\#impact-prediction#LOW "Low"
* component[0].code = http://loinc.org#48004-6
* component[0].code.text = "coding HGVS"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_001366781.1:c.90T>C "NM_001366781.1:c.90T>C"
//kp - unable to determine pHGVS
// * component[+].code = http://loinc.org#48005-3 "Amino acid change (pHGVS)"
// * component[=].valueCodeableConcept = http://varnomen.hgvs.org#p.Cys30Cys "p.Cys30Cys"
* component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_001366781.1 "NM_001366781.1"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept = http://www.sequenceontology.org#SO:0001819 "synonymous_variant"

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/molec-conseq3.fsh

Instance: molec-conseq3
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(Patient/HG00403)
* effectiveDateTime = "2023-06-01"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(Observation/variant-with-molec-consequences)
* interpretation = http://example.org/pcingola.github.io/SnpEff/se_inputoutput/\#impact-prediction#MOD "Modifier"
* component[0].code = http://loinc.org#48004-6
* component[0].code.text = "coding HGVS"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_001395543.1:c.-171T>C "NM_001395543.1:c.-171T>C"
* component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_001395543.1 "NM_001395543.1"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept = http://www.sequenceontology.org#SO:0001623 "5_prime_UTR_variant"


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/molec-conseq4.fsh

Instance: molec-conseq4
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(Patient/HG00403)
* effectiveDateTime = "2023-06-01"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(Observation/variant-with-molec-consequences)
* interpretation = http://example.org/pcingola.github.io/SnpEff/se_inputoutput/\#impact-prediction#MOD "Modifier"
//kp - Error while processing 'NM_001395544.1:c.-21-3742T>C': Intronic variants are not yet supported
// * component[0].code = http://loinc.org#48004-6
// * component[0].code.text = "coding HGVS"
// * component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_001395544.1:c.-21-3742T>C "NM_001395544.1:c.-21-3742T>C"
* component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_001395544.1 "NM_001395544.1"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept = http://www.sequenceontology.org#SO:0001627 "intron_variant"



---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/obs-idh-ex.fsh

Instance: obs-idh-ex
InstanceOf: DiagnosticImplication
Description: "Example DiagnosticImplication - Pathogenic for CF"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* subject = Reference(Patient/HG00403)
* effectiveDateTime = "2023-06-01"
* performer = Reference(ExampleLab)
* derivedFrom = Reference(SNVexample)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA6668-3 "Pathogenic"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.ncbi.nlm.nih.gov/clinvar/variation/53685/#clinical-assertions"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.coding[0] = $MONDO#MONDO:0009061 "cystic fibrosis"
* component[=].valueCodeableConcept.text = "Cystic fibrosis"
* component[+].code = TbdCodesCS#condition-inheritance
* component[=].valueCodeableConcept = $HPO#HP:0000007 "Autosomal recessive inheritance"

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/RepeatExpansionExamples.fsh

Instance: RepeatExpansion
InstanceOf: Variant
Description: "Repeat Expansion Variant"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2023-06-01"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"

* component[0].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept = $HGNCID#HGNC:8565 "PABPN1"

* component[+].code = $LNC#48001-2 "Cytogenetic (chromosome) location"
* component[=].valueCodeableConcept.text = "chr14"

* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_004643.4 "NM_004643.4"

* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept.coding = $HGVS#NM_004643.3:c.3GGC[14] "NM_004643.3:c.3GGC[14]"

* component[+].code = $LNC#81254-5 "Genomic allele start-end"
* component[=].valueRange.low.value = 3

* component[+].code = $LNC#81252-9 "Discrete genetic variant"
* component[=].valueCodeableConcept.coding = $CLINVAR#503634 "NM_004643.3(PABPN1):c.3GGC[11] (p.Ala11_Gly12insAlaAlaAlaAla)"

* component[+].code = TbdCodesCS#repeat-motif "Repeat Expansion Motif"
* component[=].valueString = "GGC"
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 1

* component[+].code = TbdCodesCS#repeat-number "Number of Repeat Expansions"
* component[=].valueQuantity.value = 11
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 1


Instance: MultipleRepeatExpansions
InstanceOf: Variant
Description: "Repeat Expansion Variant with multiple motifs"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2023-06-01"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"

* component[0].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept = $HGNCID#HGNC:3775 "FMR1"

* component[+].code = $LNC#48001-2 "Cytogenetic (chromosome) location"
* component[=].valueCodeableConcept.text = "chrx"

* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_002024.5 "NM_002024.5"

* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept.coding = $HGVS#NM_002024.5:c.-128_-69GGC[10]GGA[1]GGC[9]GGA[1]GGC[10] "NM_002024.5:c.-128_-69GGC[10]GGA[1]GGC[9]GGA[1]GGC[10]"

* component[+].code = $LNC#81254-5 "Genomic allele start-end"
* component[=].valueRange.low.value = 3

* component[+].code = TbdCodesCS#repeat-motif "Repeat Expansion Motif"
* component[=].valueString = "GGC"
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 1

* component[+].code = TbdCodesCS#repeat-number "Number of Repeat Expansions"
* component[=].valueQuantity.value = 10
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 1

* component[+].code = TbdCodesCS#repeat-motif "Repeat Expansion Motif"
* component[=].valueString = "GGA"
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 2

* component[+].code = TbdCodesCS#repeat-number "Number of Repeat Expansions"
* component[=].valueQuantity.value = 1
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 2

* component[+].code = TbdCodesCS#repeat-motif "Repeat Expansion Motif"
* component[=].valueString = "GGC"
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 3

* component[+].code = TbdCodesCS#repeat-number "Number of Repeat Expansions"
* component[=].valueQuantity.value = 9
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 3

* component[+].code = TbdCodesCS#repeat-motif "Repeat Expansion Motif"
* component[=].valueString = "GGA"
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 4

* component[+].code = TbdCodesCS#repeat-number "Number of Repeat Expansions"
* component[=].valueQuantity.value = 1
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 4

* component[+].code = TbdCodesCS#repeat-motif "Repeat Expansion Motif"
* component[=].valueString = "GGC"
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 5

* component[+].code = TbdCodesCS#repeat-number "Number of Repeat Expansions"
* component[=].valueQuantity.value = 10
* component[=].extension[RepeatMotifOrder].valuePositiveInt = 5


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/servicerequest-hla-a-r4.fsh

Instance: servicerequest-hla-a-r4
InstanceOf: ServiceRequest
Description: "Example ServiceRequest for genomics"
Usage: #example
* identifier.system = "http:/aDonorRegistry.com"
* identifier.value = "1234.5678"
* status = #completed
* intent = #order
* category = $SCT#108252007 "Laboratory procedure"
* priority = #routine
* code = $LNC#13298-5 "HLA-A [Type]"
* subject.identifier.value = "001^24680"
* requester.type = "Organization"
* requester.display = "aDonorRegistry"
* performer.type = "Organization"
* performer.display = "aTypingLab, Inc"
* specimen.type = "Specimen"
* specimen.display = "buccal swab: 123456789"

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/SMN1SMN2_CNVexample.fsh


//examples are based on consultation with a New Born Sequencing lab director and the following https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7846873/ https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7713720/ https://pubmed.ncbi.nlm.nih.gov/29614695/ https://pubmed.ncbi.nlm.nih.gov/39051410/ 
//Workflow is population based screening for SMN which is a PCR test for SMN1 homozygous deletion then SMN2 to help in determining how soon to treat
Instance: obs1-interact-smn1-smn2
InstanceOf: Variant
Description: "SMN1 CNV companion for SMN1/SMN2 CNV example"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* status = #final "Final"
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
//* code.coding[1].system = "http://loinc.org"
//* code.coding[1].code = #35462-1
//* code.coding[1].display = "SMN1 gene targeted mutation analysis in Blood or Tissue by Molecular genetics method"
* method = $LNC#LA26418-6 "PCR"
* subject.display = "A Newborn"
* effectiveDateTime = "2019-04-01"
* performer = Reference(ExampleOrg)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:11117 "SMN1" 
* component[coding-change-type].valueCodeableConcept = $SEQONT#SO:0001019 "copy_number_variation"
* component[genomic-source-class].valueCodeableConcept = $LNC#LA6683-2 "Germline"
* component[copy-number].valueQuantity.value = 0
* component[copy-number].valueQuantity.code = #1
* component[copy-number].valueQuantity.system = $UCUM
* component[genomic-ref-seq].valueCodeableConcept.coding = $NCBIREFSEQ#NC_000005.10
* component[inner-start-end].valueRange.low.value = 70924941
* component[inner-start-end].valueRange.high.value = 70966375


Instance: obs2-interact-smn1-smn2
InstanceOf: Variant
Description: "SMN2 CNV companion for SMN1/SMN2 CNV example"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* status = #final "Final"
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
//* code.coding[+].system = "http://loinc.org"
//* code.coding[=].code = #54449-4
//* code.coding[=].display = "SMN2 gene targeted mutation analysis in Blood or Tissue by Molecular genetics method"
* method = $LNC#LA26418-6 "PCR"
* subject.display = "A Newborn"
* effectiveDateTime = "2019-04-01"
* performer = Reference(ExampleOrg)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:11118 "SMN2" 
* component[coding-change-type].valueCodeableConcept = $SEQONT#SO:0001019 "copy_number_variation"
* component[genomic-source-class].valueCodeableConcept = $LNC#LA6683-2 "Germline"
* component[copy-number].valueQuantity.value = 2
* component[copy-number].valueQuantity.code = #1
* component[copy-number].valueQuantity.system = $UCUM

* component[genomic-ref-seq].valueCodeableConcept.coding = $NCBIREFSEQ#NC_000005.10
* component[inner-start-end].valueRange.low.value = 70049523
* component[inner-start-end].valueRange.high.value = 70090528



Instance: therapuDrug1-interact-smn1-smn2
InstanceOf: TherapeuticImplication
Description: "Antisense Oligonucelotide example of Therapeutic Implication for SMN1/SMN2 CNV example"
* category[labCategory] = $OBSCAT#laboratory
* code = TbdCodesCS#therapeutic-implication
* subject.display = "A Newborn"
* effectiveDateTime = "2019-04-01"
* performer[0] = Reference(ExampleOrg)
* component[+].code.coding = $LNC#81259-4 // "Phenotypic treatment context"
* component[=].valueCodeableConcept.coding[0] = $SCT#5262007 "Spinal muscular atrophy (disorder)"
* component[=].valueCodeableConcept.text = "Spinal muscular atrophy (SMA)"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "nusinersen (antisense oligonucleotide)"
* derivedFrom[0] = Reference(obs1-interact-smn1-smn2)
* derivedFrom[+] = Reference(obs2-interact-smn1-smn2)
* status = #final
* component[+].code = TbdCodesCS#conclusion-string
* component[=].valueString = "Spinal muscular atrophy (SMA) is an autosomal recessive disease characterized by the degeneration of alpha motor neurons in the spinal cord, leading to muscular atrophy. SMA is caused by deletions or mutations in the survival motor neuron 1 gene (SMN1). In humans, a nearly identical copy gene, SMN2, is present. Because SMN2 has been shown to decrease disease severity in a dose-dependent manner, SMN2 copy number is predictive of disease severity...The overarching recommendation is that all infants with two or three copies of SMN2 should receive immediate treatment"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://pubmed.ncbi.nlm.nih.gov/29614695/"

Instance: therapuDrug2-interact-smn1-smn2
InstanceOf: TherapeuticImplication
Description: "Transgene example of Therapeutic Implication for SMN1/SMN2 CNV example"
* category[labCategory] = $OBSCAT#laboratory
* code = TbdCodesCS#therapeutic-implication
* subject.display = "A Newborn"
* effectiveDateTime = "2019-04-01"
* performer[0] = Reference(ExampleOrg)
* component[+].code.coding = $LNC#81259-4 // "Phenotypic treatment context"
* component[=].valueCodeableConcept.coding[0] = $SCT#5262007 "Spinal muscular atrophy (disorder)"
* component[=].valueCodeableConcept.text = "Spinal muscular atrophy (SMA)"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "onasemnogene abeparvovec (transgene)"
* derivedFrom[0] = Reference(obs1-interact-smn1-smn2)
* derivedFrom[+] = Reference(obs2-interact-smn1-smn2)
* status = #final
* component[+].code = TbdCodesCS#conclusion-string
* component[=].valueString = "Spinal muscular atrophy (SMA) is an autosomal recessive disease characterized by the degeneration of alpha motor neurons in the spinal cord, leading to muscular atrophy. SMA is caused by deletions or mutations in the survival motor neuron 1 gene (SMN1). In humans, a nearly identical copy gene, SMN2, is present. Because SMN2 has been shown to decrease disease severity in a dose-dependent manner, SMN2 copy number is predictive of disease severity...The overarching recommendation is that all infants with two or three copies of SMN2 should receive immediate treatment"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://pubmed.ncbi.nlm.nih.gov/29614695/"

Instance: therapuDrug3-interact-smn1-smn2
InstanceOf: TherapeuticImplication
Description: "Small molecule example of Therapeutic Implication for SMN1/SMN2 CNV example"
* category[labCategory] = $OBSCAT#laboratory
* code = TbdCodesCS#therapeutic-implication
* subject.display = "A Newborn"
* effectiveDateTime = "2019-04-01"
* performer[0] = Reference(ExampleOrg)
* component[+].code.coding = $LNC#81259-4 // "Phenotypic treatment context"
* component[=].valueCodeableConcept.coding[0] = $SCT#5262007 "Spinal muscular atrophy (disorder)"
* component[=].valueCodeableConcept.text = "Spinal muscular atrophy (SMA)"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "risdiplam (small molecule)"
* derivedFrom[0] = Reference(obs1-interact-smn1-smn2)
* derivedFrom[+] = Reference(obs2-interact-smn1-smn2)
* status = #final
* component[+].code = TbdCodesCS#conclusion-string
* component[=].valueString = "Spinal muscular atrophy (SMA) is an autosomal recessive disease characterized by the degeneration of alpha motor neurons in the spinal cord, leading to muscular atrophy. SMA is caused by deletions or mutations in the survival motor neuron 1 gene (SMN1). In humans, a nearly identical copy gene, SMN2, is present. Because SMN2 has been shown to decrease disease severity in a dose-dependent manner, SMN2 copy number is predictive of disease severity...The overarching recommendation is that all infants with two or three copies of SMN2 should receive immediate treatment"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://pubmed.ncbi.nlm.nih.gov/29614695/"


Instance: diagnosticImplication-interact-smn1-smn2
InstanceOf: DiagnosticImplication
Description: "Diagnostic Implication for SMN1/SMN2 CNV example"
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* subject.display = "A Newborn"
* effectiveDateTime = "2019-04-01"
* performer[0] = Reference(ExampleOrg)
* method = $LNC#LA26811-2 "Computational analysis"
* derivedFrom = Reference(obs1-interact-smn1-smn2)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA6668-3 "Pathogenic"
* component[+].code = $LNC#93044-6 "Level of Evidence"
* component[=].valueCodeableConcept = $LNC#LA30200-2 "Very strong evidence pathogenic"
* component[+].code = $LNC#81259-4
* component[=].valueCodeableConcept.coding[0] = $SCT#5262007 "Spinal muscular atrophy (disorder)"
* component[=].valueCodeableConcept.text = "Spinal muscular atrophy (SMA)"

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/SNVexample.fsh

Instance: SNVexample
InstanceOf: Variant
Description: "Example Variant"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(ExamplePatient)
* effectiveDateTime = "2023-06-01"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* method = $LNC#LA26398-0 "Sequencing"
* specimen.identifier.system = "http://slk-kliniken.de/fhir/namingSystem/tissueID"
* specimen.identifier.value = "16-123456-23"
* component[0].code = $LNC#48018-6 "Gene studied ID"
* component[=].valueCodeableConcept = $HGNCID#HGNC:644 "AR"
* component[+].code = $LNC#48001-2 "Cytogenetic (chromosome) location"
* component[=].valueCodeableConcept.text = "chrX"
* component[+].code = $LNC#62374-4 "Human reference sequence assembly version"
* component[=].valueCodeableConcept = $LNC#LA14029-5 "GRCh37"
* component[+].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept.coding = $LNC#LA6684-0 "Somatic"
* component[+].code = $LNC#48004-6 "DNA change (c.HGVS)"
* component[=].valueCodeableConcept.coding = $HGVS#NM_000044.6:c.7G>A
* component[+].code = $LNC#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = $HGVS#NP_000035.2:p.Val3Met "NP_000035.2:p.Val3Met"
* component[+].code = $LNC#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = $NCBIREFSEQ#NM_000044.3 "NM_000044.6"
* component[+].code = $LNC#81258-6 "Sample VAF"
* component[=].valueQuantity.value = 0.44
* component[=].valueQuantity.system = "http://unitsofmeasure.org"
* component[=].valueQuantity.unit = "decimal"
* component[+].code = $LNC#82121-5 "Allelic read depth"
* component[=].valueQuantity = 120 '1' "reads per base pair"

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/somaticReportPDFexamples.fsh

//Instance: bundle-somaticReportPDFdraftExamples
//InstanceOf: Bundle
//Description: "data instances for somatic report."
//// Usage: #os1dddpe = #collection
//* type = #collection
//* entry[+].resource = NOTCH1-uncertain-var
//* entry[+].resource = ZFHX3-uncertain-var
//* entry[+].resource = MSH2-del-var
//* entry[+].resource = NTHL1-snv-var
//* entry[+].resource = STAG2-insertion-var
//* entry[+].resource = ATR-insertion-var
//* entry[+].resource = EGFR-L858R-var
//* entry[+].resource = ROS1-Fusion-var
//* entry[+].resource = NOTCH1-significance
//* entry[+].resource = ZFHX3-significance
//* entry[+].resource = MSH2-del-disease
//* entry[+].resource = NTHL1-snv-disease
//* entry[+].resource = STAG2-insertion-significance
//* entry[+].resource = ATR-insertion-significance
//* entry[+].resource = EGFR-L858R-significance
//* entry[+].resource = ROS1-Fusion-disease
//* entry[+].resource = ROS1-Fusion-therapuTrial
//* entry[+].resource = ROS1-Fusion-therapuDrug
//* entry[+].resource = EGFR-L858R-therapuDrug1
//* entry[+].resource = EGFR-L858R-therapuDrug2
//* entry[+].resource = TMB-therapuDrug
//* entry[+].resource = NOTCH1-uncertain-molc
//* entry[+].resource = ZFHX3-uncertain-molc
//* entry[+].resource = MSH2-del-molc
//* entry[+].resource = NTHL1-snv-molc
//* entry[+].resource = STAG2-insertion-molc
//* entry[+].resource = ATR-insertion-molc
//* entry[+].resource = EGFR-L858R-molc
//* entry[+].resource = ROS1-Fusion
//* entry[+].resource = somaticStudy
//* entry[+].resource = TMBExample
//* entry[+].resource = MSIExample
//* entry[+].resource = PDL1Example
//* entry[+].resource = HLA-A-haplotype1
//* entry[+].resource = HLA-A-haplotype2
//* entry[+].resource = HLA-B-haplotype1
//* entry[+].resource = HLA-B-haplotype2
//* entry[+].resource = HLA-C-haplotype1
//* entry[+].resource = HLA-C-haplotype2
//* entry[+].resource = somaticReport
//* entry[+].resource = somaticServiceRequest
//* entry[+].resource = orderingPractitioner
//* entry[+].resource = pathologistPractitioner
//* entry[+].resource = supervisorPractitioner
//* entry[+].resource = performingOrganization

Instance: NOTCH1-uncertain-var
InstanceOf: Variant
Description: "Example for NOTCH1 uncertain"
//* id = "VariantExample1"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* status = #final "Final"
* category[labCategory] = $OBSCAT#laboratory
* method = $LNC#LA26398-0 "Sequencing"
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
//* interpretation = $SCT#10828004 "Positive (qualifier value)"
* component[representative-protein-hgvs].valueCodeableConcept = $HGVS#NP_060087.3:p.A1931T
* component[representative-transcript-ref-seq].valueCodeableConcept = $NCBIREFSEQ#NM_017617.5
* component[allelic-read-depth].valueQuantity = 221 '1' "reads per base pair"
//* component[coding-change-type].valueCodeableConcept = 
* component[representative-coding-hgvs].valueCodeableConcept = $HGVS#NM_017617.5:c.5791G>A
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:7881 "NOTCH1"
* component[alt-allele].valueString = "T"
* component[chromosome-identifier].valueCodeableConcept =  $LNC#LA21262-3 "Chromosome 9"
* component[exact-start-end][0].valueRange.low.value = 139395147
* component[exact-start-end][0].valueRange.high.value = 139395147
* component[genomic-ref-seq].valueCodeableConcept = $NCBIREFSEQ#NC_000009.11
* component[ref-allele].valueString = "C"
//* component[variation-code].valueCodeableConcept = 
* component[genomic-hgvs].valueCodeableConcept = $HGVS#NC_000009.11:g.139395147C>T
* component[genomic-source-class].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[sample-allelic-frequency][0].valueQuantity.value = 0.15
* component[sample-allelic-frequency][0].valueQuantity.system = "http://unitsofmeasure.org"
* component[sample-allelic-frequency][0].valueQuantity.unit = "decimal"

Instance: ZFHX3-uncertain-var
InstanceOf: Variant
Description: "Example for ZFHX3 uncertain"
//* id = "VariantExample1"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* status = #final "Final"
* category[labCategory] = $OBSCAT#laboratory
* method = $LNC#LA26398-0 "Sequencing"
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
//* interpretation = $SCT#10828004 "Positive (qualifier value)"
* component[representative-protein-hgvs].valueCodeableConcept = $HGVS#NP_008816.3:p.G585S
* component[representative-transcript-ref-seq].valueCodeableConcept = $NCBIREFSEQ#NM_006885.4
* component[allelic-read-depth].valueQuantity = 212 '1' "reads per base pair"
//* component[coding-change-type].valueCodeableConcept = 
* component[representative-coding-hgvs].valueCodeableConcept = $HGVS#NM_006885.4:c.1753G>A
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:777 "ZFHX3"
* component[alt-allele].valueString = "T"
* component[chromosome-identifier].valueCodeableConcept =  $LNC#LA21269-8 "Chromosome 16"
* component[exact-start-end][0].valueRange.low.value = 72992292
* component[exact-start-end][0].valueRange.high.value = 72992292
* component[genomic-ref-seq].valueCodeableConcept = $NCBIREFSEQ#NC_000016.9
* component[ref-allele].valueString = "C"
//* component[variation-code].valueCodeableConcept = 
* component[genomic-hgvs].valueCodeableConcept = $HGVS#NC_000016.9:g.72992292C>T
* component[genomic-source-class].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[sample-allelic-frequency][0].valueQuantity.value = 0.10
* component[sample-allelic-frequency][0].valueQuantity.system = "http://unitsofmeasure.org"
* component[sample-allelic-frequency][0].valueQuantity.unit = "decimal"

Instance: MSH2-del-var
InstanceOf: Variant
Description: "Example for MSH2 deletion"
//* id = "VariantExample1"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* status = #final "Final"
* category[labCategory] = $OBSCAT#laboratory
* method = $LNC#LA26398-0 "Sequencing"
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
//* interpretation = $SCT#10828004 "Positive (qualifier value)"
//kp - Error while processing 'NP_000242.1:p.Val611fs': Frameshift variants are not supported
// * component[representative-protein-hgvs].valueCodeableConcept = $HGVS#NP_000242.1:p.Val611fs
* component[representative-transcript-ref-seq].valueCodeableConcept = $NCBIREFSEQ#NM_006885.4
//* component[allelic-read-depth].valueQuantity =
* component[coding-change-type].valueCodeableConcept = http://www.sequenceontology.org#SO:0000159 "deletion"
//* component[representative-coding-hgvs].valueCodeableConcept = 
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:7235 "MSH2"
//* component[alt-allele].valueString = 
//* component[chromosome-identifier].valueCodeableConcept =  
//* component[exact-start-end][0].valueRange.low.value = 
//* component[exact-start-end][0].valueRange.high.value = 
//* component[genomic-ref-seq].valueCodeableConcept = 
//* component[ref-allele].valueString = 
* component[variation-code].valueCodeableConcept = $CLINVAR#1780982 "NM_000251.3(MSH2):c.1832del (p.Val611fs)"
//* component[genomic-hgvs].valueCodeableConcept = 
* component[genomic-source-class].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
//* component[sample-allelic-frequency][0].valueQuantity.value = 
//* component[sample-allelic-frequency][0].valueQuantity.system = 
//* component[sample-allelic-frequency][0].valueQuantity.unit = 

//NTHL1_snv
Instance: NTHL1-snv-var
InstanceOf: Variant
Description: "Example for NTHL1 SNV"
//* id = "VariantExample1"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* status = #final "Final"
* category[labCategory] = $OBSCAT#laboratory
* method = $LNC#LA26398-0 "Sequencing"

* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
//* interpretation = $SCT#10828004 "Positive (qualifier value)"
* component[representative-protein-hgvs].valueCodeableConcept = $HGVS#NP_002519.2:p.Trp243Ter
//* component[representative-transcript-ref-seq].valueCodeableConcept = 
//* component[allelic-read-depth].valueQuantity =
* component[coding-change-type].valueCodeableConcept = http://www.sequenceontology.org#SO:0001483 "SNV"
* component[representative-coding-hgvs].valueCodeableConcept = $HGVS#NM_002528.7:c.728G>A
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:8028 "NTHL1"
//* component[alt-allele].valueString = 
//* component[chromosome-identifier].valueCodeableConcept =  
//* component[exact-start-end][0].valueRange.low.value = 
//* component[exact-start-end][0].valueRange.high.value = 
//* component[genomic-ref-seq].valueCodeableConcept = 
//* component[ref-allele].valueString = 
* component[variation-code].valueCodeableConcept = $CLINVAR#962538 "NM_002528.7(NTHL1):c.728G>A (p.Trp243Ter)"
* component[genomic-hgvs].valueCodeableConcept = $HGVS#NC_000016.10:g.2040196C>T
* component[genomic-source-class].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
//* component[sample-allelic-frequency][0].valueQuantity.value = 
//* component[sample-allelic-frequency][0].valueQuantity.system = 
//* component[sample-allelic-frequency][0].valueQuantity.unit =

Instance: STAG2-insertion-var
InstanceOf: Variant
Description: "Example for STAG2 insertion"
//* id = "VariantExample1"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* status = #final "Final"
* category[labCategory] = $OBSCAT#laboratory
* method = $LNC#LA26398-0 "Sequencing"
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
//* interpretation = $SCT#10828004 "Positive (qualifier value)"
* component[representative-protein-hgvs].valueCodeableConcept = $HGVS#NP_006594.3:p.S1178*
* component[representative-transcript-ref-seq].valueCodeableConcept = $NCBIREFSEQ#NM_006603.5
* component[allelic-read-depth].valueQuantity = 88 '1' "reads per base pair"
* component[coding-change-type].valueCodeableConcept = http://www.sequenceontology.org#SO:0000667 "insertion"
* component[representative-coding-hgvs].valueCodeableConcept = $HGVS#NM_006603.5:c.3530_3531insGTGACTATTAATAT
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:11355 "STAG2"
* component[alt-allele].valueString = "GTGACTA"
* component[chromosome-identifier].valueCodeableConcept =  $LNC#LA21276-3 "Chromosome X"
* component[exact-start-end][0].valueRange.low.value = 123227930
* component[exact-start-end][0].valueRange.high.value = 123227930
* component[genomic-ref-seq].valueCodeableConcept = $NCBIREFSEQ#NC_000023.10
//* component[ref-allele].valueString = ""
//* component[variation-code].valueCodeableConcept = 
* component[genomic-hgvs].valueCodeableConcept = $HGVS#NC_000023.10:g.123227930_123227931insGTGACTATTAATAT
* component[genomic-source-class].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[sample-allelic-frequency][0].valueQuantity.value = 0.08
* component[sample-allelic-frequency][0].valueQuantity.system = "http://unitsofmeasure.org"
* component[sample-allelic-frequency][0].valueQuantity.unit = "decimal"

Instance: ATR-insertion-var
InstanceOf: Variant
Description: "Example for ATR insertion"
//* id = "VariantExample1"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* status = #final "Final"
* category[labCategory] = $OBSCAT#laboratory
* method = $LNC#LA26398-0 "Sequencing"
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
//* interpretation = $SCT#10828004 "Positive (qualifier value)"
//kp - Error while processing 'NP_001175.2:p.R960fs*2': Frameshift variants are not supported
// * component[representative-protein-hgvs].valueCodeableConcept = $HGVS#NP_001175.2:p.R960fs*2
* component[representative-transcript-ref-seq].valueCodeableConcept = $NCBIREFSEQ#NM_001184.4
* component[allelic-read-depth].valueQuantity = 131 '1' "reads per base pair"
* component[coding-change-type].valueCodeableConcept = http://www.sequenceontology.org#SO:0000667 "insertion"
* component[representative-coding-hgvs].valueCodeableConcept = $HGVS#NM_001184.4:c.2878_2879insAGTAA
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:882 "ATR"
* component[alt-allele].valueString = "TTACT"
* component[chromosome-identifier].valueCodeableConcept =  $LNC#LA21256-5 "Chromosome 3"
* component[exact-start-end][0].valueRange.low.value = 142269071
* component[exact-start-end][0].valueRange.high.value = 142269071
* component[genomic-ref-seq].valueCodeableConcept = $NCBIREFSEQ#NC_000003.11
//* component[ref-allele].valueString = ""
//* component[variation-code].valueCodeableConcept = 
* component[genomic-hgvs].valueCodeableConcept = $HGVS#NC_000003.11:g.142269071_142269072insTTACT
* component[genomic-source-class].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[sample-allelic-frequency][0].valueQuantity.value = 0.075
* component[sample-allelic-frequency][0].valueQuantity.system = "http://unitsofmeasure.org"
* component[sample-allelic-frequency][0].valueQuantity.unit = "decimal"
* component[conclusion-string].valueString = "Gene: ATR; Exon: 14; Nucleotide: NM_001184.4:c.2878_2879insAGTAA; Genomic Location: NC_000003.11:g.142269071_142269072insTTACT; Amino acid: NP_001175.2:p.R960fs*2; Function: loss; Assessment: Likely Pathogenic; Classification: Tier 2C; Allele Fraction: 5.34% (of 131 reads); Variation: Insertion; Interpretation: The protein encoded by this gene is a serine/threonine kinase and DNA damage sensor, activating cell cycle checkpoint signaling upon DNA stress. The encoded protein can phosphorylate and activate several proteins involved ... more"

Instance: EGFR-L858R-var
InstanceOf: Variant
Description: "Example for EGFR L858R"
//* id = "VariantExample1"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* status = #final "Final"
* category[labCategory] = $OBSCAT#laboratory
* method = $LNC#LA26398-0 "Sequencing"
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
//* interpretation = $SCT#10828004 "Positive (qualifier value)"
* component[variation-code].valueCodeableConcept.text = "EGFR L858R"
* component[representative-protein-hgvs].valueCodeableConcept = $HGVS#NP_005219.2:p.Leu858Arg
//* component[representative-transcript-ref-seq].valueCodeableConcept = 
//* component[allelic-read-depth].valueQuantity = 
* component[coding-change-type].valueCodeableConcept = http://www.sequenceontology.org#SO:0001483 "SNV"
* component[representative-coding-hgvs].valueCodeableConcept = $HGVS#NM_005228.4:c.2573T>G
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:3236 "EGFR"
//* component[alt-allele].valueString = 
//* component[chromosome-identifier].valueCodeableConcept =  
//* component[exact-start-end][0].valueRange.low.value = 
//* component[exact-start-end][0].valueRange.high.value = 
//* component[genomic-ref-seq].valueCodeableConcept = 
//* component[ref-allele].valueString = ""
//* component[variation-code].valueCodeableConcept = 
* component[genomic-hgvs].valueCodeableConcept = $HGVS#NC_000007.13:g.55259515T>G
* component[genomic-source-class].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
* component[sample-allelic-frequency][0].valueQuantity.value = 0.066
* component[sample-allelic-frequency][0].valueQuantity.system = "http://unitsofmeasure.org"
* component[sample-allelic-frequency][0].valueQuantity.unit = "decimal"

Instance: ROS1-Fusion-var
InstanceOf: Variant
Description: "Example for ROS1 Fusion but as typically reported in PDF, without necessarily declaring a particular fusion partner"
//* id = "VariantExample1"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
* status = #final "Final"
* category[labCategory] = $OBSCAT#laboratory
* method = $LNC#LA26398-0 "Sequencing"
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[variation-code].valueCodeableConcept.text = "ROS1 Fusion"
//* interpretation = $SCT#10828004 "Positive (qualifier value)"
//* component[representative-protein-hgvs].valueCodeableConcept =
//* component[representative-transcript-ref-seq].valueCodeableConcept = 
//* component[allelic-read-depth].valueQuantity = 
//* component[coding-change-type].valueCodeableConcept = ?fusion?
//* component[representative-coding-hgvs].valueCodeableConcept = 
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:10261 "ROS1"
//* component[alt-allele].valueString = 
//* component[chromosome-identifier].valueCodeableConcept =  
//* component[exact-start-end][0].valueRange.low.value = 
//* component[exact-start-end][0].valueRange.high.value = 
//* component[genomic-ref-seq].valueCodeableConcept = 
//* component[ref-allele].valueString = ""
//* component[variation-code].valueCodeableConcept = 
//* component[genomic-hgvs].valueCodeableConcept = 
* component[genomic-source-class].valueCodeableConcept = $LNC#LA6684-0 "Somatic"
//* component[sample-allelic-frequency][0].valueQuantity.value = 0.075
//* component[sample-allelic-frequency][0].valueQuantity.system = "http://unitsofmeasure.org"
//* component[sample-allelic-frequency][0].valueQuantity.unit = "decimal"

Instance: NOTCH1-significance
InstanceOf: DiagnosticImplication
Description: "Example DiagnosticImplication - NOTCH1 uncertain significance"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* derivedFrom = Reference(NOTCH1-uncertain-var)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA26333-7 "Uncertain Significance"


Instance: ZFHX3-significance
InstanceOf: DiagnosticImplication
Description: "Example DiagnosticImplication - ZFHX3 uncertain significance"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* derivedFrom = Reference(ZFHX3-uncertain-var)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA26333-7 "Uncertain Significance"

//MSH2_del 
//699346009 | Hereditary cancer-predisposing syndrome (disorder) SNOMED
//LA6668-3 Pathogenic
Instance: MSH2-del-disease
InstanceOf: DiagnosticImplication
Description: "Example DiagnosticImplication - MSH2 deletion Hereditary cancer-predisposing syndrome"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* derivedFrom = Reference(MSH2-del-var)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA6668-3 "Pathogenic"
* component[+].code = $LNC#81259-4 "Associated phenotype"
* component[=].valueCodeableConcept.coding[0] = $SCT#699346009 "Hereditary cancer-predisposing syndrome (disorder)"
* component[=].valueCodeableConcept.text = "Hereditary cancer-predisposing syndrome"

//NTHL1_snv  
//699346009 | Hereditary cancer-predisposing syndrome (disorder) SNOMED
//LA6668-3 Pathogenic
Instance: NTHL1-snv-disease
InstanceOf: DiagnosticImplication
Description: "Example DiagnosticImplication - NTHL1 SNV Hereditary cancer-predisposing syndrome"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* derivedFrom = Reference(NTHL1-snv-var)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA6668-3 "Pathogenic"
* component[+].code = $LNC#81259-4 "Associated phenotype"
* component[=].valueCodeableConcept.coding[0] = $SCT#699346009 "Hereditary cancer-predisposing syndrome (disorder)"
* component[=].valueCodeableConcept.text = "Hereditary cancer-predisposing syndrome"

//STAG2_insertion
//LA26332-9^Likely Pathogenic
//https://www.ncbi.nlm.nih.gov/gene/545#:~:text=ATR%20may%20function%20as%20an,radiation%20when%20ATM%20is%20deficient.
Instance: STAG2-insertion-significance
InstanceOf: DiagnosticImplication
Description: "Example DiagnosticImplication - STAG2 likely pathogenic"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* derivedFrom = Reference(STAG2-insertion-var)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA26332-9 "Likely Pathogenic"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.ncbi.nlm.nih.gov/gene/545#:~:text=ATR%20may%20function%20as%20an,radiation%20when%20ATM%20is%20deficient"

//ATR insertion 
//LA26332-9^Likely Pathogenic
//Tier 2C
//https://www.jmdjournal.org/cms/attachment/ee43a71b-81de-4cb3-ac5e-2fb9a7d41491/gr2.jpg
Instance: ATR-insertion-significance
InstanceOf: DiagnosticImplication
Description: "Example DiagnosticImplication - ATR likely pathogenic"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* derivedFrom = Reference(ATR-insertion-var)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA26332-9 "Likely Pathogenic"
* component[+].code = $LNC#93044-6 "Level of evidence"
//* component[=].valueCodeableConcept = codesystem#code "display"
* component[=].valueCodeableConcept.text = "Tier 2C"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.jmdjournal.org/cms/attachment/ee43a71b-81de-4cb3-ac5e-2fb9a7d41491/gr2.jpg"
* component[+].code = TbdCodesCS#conclusion-string
* component[=].valueString = "The protein encoded by this gene is a serine/threonine kinase and DNA damage sensor, activating cell cycle checkpoint signaling upon DNA stress. The encoded protein can phosphorylate and activate several proteins involved ... more"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.ncbi.nlm.nih.gov/gene/545#:~:text=ATR%20may%20function%20as%20an,radiation%20when%20ATM%20is%20deficient"

//EGFR L858R
//LA6668-3 Pathogenic
Instance: EGFR-L858R-significance
InstanceOf: DiagnosticImplication
Description: "Example DiagnosticImplication - EGFR pathogenic"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* derivedFrom = Reference(EGFR-L858R-var)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA6668-3 "Pathogenic"


Instance: ROS1-Fusion-disease
InstanceOf: DiagnosticImplication
Description: "Example DiagnosticImplication - ROS1 disease"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = TbdCodesCS#diagnostic-implication "Diagnostic Implication"
* derivedFrom = Reference(ROS1-Fusion-var)
* component[0].code = $LNC#53037-8 "Genetic variation clinical significance [Imp]"
* component[=].valueCodeableConcept = $LNC#LA6668-3 "Pathogenic"



Instance: ROS1-Fusion-therapuTrial
InstanceOf: TherapeuticImplication
Description: "Example of ROS1 Fusion Therapeutic Implication for Clinical Trial"
//* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/therapeutic-implication"
* category[labCategory] = $OBSCAT#laboratory
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
//* component[0].code.coding = TbdCodesCS#predicted-therapeutic-implication "Predicted Therapeutic Implication"
//* component[=].valueCodeableConcept.coding = $SCT#399223003 "Patient eligible for clinical trial"
* component[+].code.coding = $LNC#81259-4 // "Phenotypic treatment context"
//#HP:0003002
* component[=].valueCodeableConcept.coding[0] = $SCT#254637007 "Non-small cell lung cancer (disorder)"
* component[=].valueCodeableConcept.text = "Non-small cell lung cancer"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "Taletrectinib"
* component[+].code.coding = TbdCodesCS#associated-therapy "Associated Therapy"
* component[=].valueCodeableConcept.coding = $NCTTRIAL#NCT04919811 "NCT04919811"
* component[=].valueCodeableConcept.text = "A Single-Arm, Open-Label, Multicenter Phase 2 Study to Evaluate the Efficacy and Safety of Taletrectinib in Patients With Advanced or Metastatic ROS1 Positive NSCLC and Other Solid Tumors"
* derivedFrom = Reference(ROS1-Fusion-var)
* status = #final
* component[+].code = $LNC#93044-6 "Level of evidence"
//* component[=].valueCodeableConcept = codesystem#code "display"
* component[=].valueCodeableConcept.text = "Tier I - Level A"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.jmdjournal.org/cms/attachment/ee43a71b-81de-4cb3-ac5e-2fb9a7d41491/gr2.jpg"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.cancer.net/cancer-types/lung-cancer-non-small-cell/types-treatment"


Instance: ROS1-Fusion-therapuDrug
InstanceOf: TherapeuticImplication
Description: "Example of ROS1 Fusion Therapeutic Implication for drug treatment"
//* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/therapeutic-implication"
* category[labCategory] = $OBSCAT#laboratory
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
//* component[0].code.coding = TbdCodesCS#predicted-therapeutic-implication "Predicted Therapeutic Implication"
* component[0].code.coding = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[=].valueCodeableConcept.text = "cancer sensitive to crizotinib"
* component[+].code.coding = $LNC#81259-4 // "Phenotypic treatment context"
* component[=].valueCodeableConcept.coding[0] = $SCT#254637007 "Non-small cell lung cancer (disorder)"
* component[=].valueCodeableConcept.text = "Non-small cell lung cancer"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "Crizotinib"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://civicdb.org/molecular-profiles/2562/summary"
* derivedFrom = Reference(ROS1-Fusion-var)
* status = #final
* component[+].code = TbdCodesCS#conclusion-string
* component[=].valueString = "Non-small cell lung cancer positive for ROS1 fusions are sensitive to crizotinib"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://civicdb.org/evidence/7282/summary"
* component[+].code = $LNC#93044-6 "Level of evidence"
//* component[=].valueCodeableConcept = codesystem#code "display"
* component[=].valueCodeableConcept.text = "Tier I - Level A"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.jmdjournal.org/cms/attachment/ee43a71b-81de-4cb3-ac5e-2fb9a7d41491/gr2.jpg"


Instance: TMB-therapuDrug
InstanceOf: TherapeuticImplication
Description: "Example of high TMB Therapeutic Implication for drug treatment"
* category[labCategory] = $OBSCAT#laboratory
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* component[0].code.coding = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[=].valueCodeableConcept.text = "cancer sensitive to pembrolizumab"
* component[+].code.coding = $LNC#81259-4 // "Phenotypic treatment context"
* component[=].valueCodeableConcept.coding[0] = $SCT#254637007 "Non-small cell lung cancer (disorder)"
* component[=].valueCodeableConcept.text = "Non-small cell lung cancer"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "Pembrolizumab"
* derivedFrom = Reference(TMBExample)
* status = #final
* component[+].code = TbdCodesCS#conclusion-string
* component[=].valueString = "Non-small cell lung cancer with high TMB is sensitive to pembrolizumab"

Instance: EGFR-L858R-therapuDrug1
InstanceOf: TherapeuticImplication
Description: "Example of EGFR-L858R Therapeutic Implication for drug treatment 1"
* category[labCategory] = $OBSCAT#laboratory
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* component[+].code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[+].code = $LNC#93044-6 "Level of evidence"
* component[=].valueCodeableConcept.text = "Tier I - Level A"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.jmdjournal.org/cms/attachment/ee43a71b-81de-4cb3-ac5e-2fb9a7d41491/gr2.jpg"
* component[+].code.coding = $LNC#81259-4 // "Phenotypic treatment context"
* component[=].valueCodeableConcept.coding[0] = $SCT#254637007 "Non-small cell lung cancer (disorder)"
* component[=].valueCodeableConcept.text = "Non-small cell lung cancer"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "Gefitinib"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.cancer.net/cancer-types/lung-cancer-non-small-cell/types-treatment"
* derivedFrom = Reference(EGFR-L858R-var)
* status = #final
* component[+].code = TbdCodesCS#conclusion-string
* component[=].valueString = "Non-small cell lung cancer with EGFR L858R mutation is sensitive to gefitinib"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://civicdb.org/evidence/2994/summary"


Instance: EGFR-L858R-therapuDrug2
InstanceOf: TherapeuticImplication
Description: "Example of EGFR-L858R Therapeutic Implication for drug treatment 2"
* category[labCategory] = $OBSCAT#laboratory
* code = TbdCodesCS#therapeutic-implication
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* component[+].code = TbdCodesCS#therapeutic-implication "Therapeutic Implication"
* component[=].valueCodeableConcept = $LNC#LA6677-4 "Responsive"
* component[+].code = $LNC#93044-6 "Level of evidence"
* component[=].valueCodeableConcept.text = "Tier I - Level A"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.jmdjournal.org/cms/attachment/ee43a71b-81de-4cb3-ac5e-2fb9a7d41491/gr2.jpg"
* component[+].code.coding = $LNC#81259-4 // "Phenotypic treatment context"
* component[=].valueCodeableConcept.coding[0] = $SCT#254637007 "Non-small cell lung cancer (disorder)"
* component[=].valueCodeableConcept.text = "Non-small cell lung cancer"
* component[+].code = $LNC#51963-7 "Medication assessed"
* component[=].valueCodeableConcept.text = "Erlotinib"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://www.cancer.net/cancer-types/lung-cancer-non-small-cell/types-treatment"
* derivedFrom = Reference(EGFR-L858R-var)
* status = #final
* component[+].code = TbdCodesCS#conclusion-string
* component[=].valueString = "Non-small cell lung cancer with EGFR L858R mutation is sensitive to erlotinib"
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.type = #citation
* component[=].extension[RelatedArtifactComponent].valueRelatedArtifact.url = "https://civicdb.org/evidence/2994/summary"

Instance: NOTCH1-uncertain-molc
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence NOTCH1 missense_variant"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(CGPatientExample01)
* derivedFrom = Reference(NOTCH1-uncertain-var)
* component[0].code = http://loinc.org#48004-6
* component[0].code.text = "coding HGVS"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_017617.5:c.5791G>A "NM_017617.5:c.5791G>A"

* component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_017617.5 "NM_017617.5"

* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept.coding[0] = http://www.sequenceontology.org#SO:0001583 "missense_variant"

* component[+].code = http://loinc.org#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NP_060087.3:p.A1931T "NP_060087.3:p.A1931T"

Instance: ZFHX3-uncertain-molc
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence ZFHX3 missense_variant"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(CGPatientExample01)
* derivedFrom = Reference(ZFHX3-uncertain-var)
* component[0].code = http://loinc.org#48004-6
* component[0].code.text = "coding HGVS"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_006885.4:c.1753G>A "NM_006885.4:c.1753G>A"
* component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_006885.4 "NM_006885.4"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept.coding[0] = http://www.sequenceontology.org#SO:0001583 "missense_variant"
* component[+].code = http://loinc.org#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NP_008816.3:p.G585S "NP_008816.3:p.G585S"

Instance: MSH2-del-molc
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence MSH2 frameshift"
Usage: #example
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(CGPatientExample01)
* derivedFrom = Reference(MSH2-del-var)
* component[0].code = http://loinc.org#48004-6
* component[0].code.text = "coding HGVS"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_000251.3:c.1832del "NM_000251.3:c.1832del"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept.coding[0] = http://www.sequenceontology.org#SO:0001589 "frameshift_variant"
//kp - Error while processing 'NP_000242.1:p.Val611fs': Frameshift variants are not supported
// * component[+].code = http://loinc.org#48005-3 "Amino acid change (pHGVS)"
// * component[=].valueCodeableConcept = http://varnomen.hgvs.org#NP_000242.1:p.Val611fs "NP_000242.1:p.Val611fs"

Instance: NTHL1-snv-molc
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence NTHL1 SNV variant with nonsense codon"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(CGPatientExample01)
* derivedFrom = Reference(NTHL1-snv-var)
* component[0].code = http://loinc.org#48004-6
* component[0].code.text = "coding HGVS"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_002528.7:c.728G>A "NM_002528.7:c.728G>A"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept.coding[0] = http://www.sequenceontology.org#SO:0001587 "stop_gained"
* component[=].valueCodeableConcept.text = "nonsense codon"
* component[+].code = http://loinc.org#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NP_002519.2:p.Trp243Ter "NP_002519.2:p.Trp243Ter"

Instance: STAG2-insertion-molc
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence STAG2 frameshift"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(CGPatientExample01)
* derivedFrom = Reference(STAG2-insertion-var)
* component[0].code = http://loinc.org#48004-6
* component[0].code.text = "coding HGVS"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_006603.5:c.3530_3531insGTGACTATTAATAT "NM_006603.5:c.3530_3531insGTGACTATTAATAT"
* component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_006603.5 "NM_006603.5"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept.coding[0] = http://www.sequenceontology.org#SO:0001589 "frameshift_variant"
* component[+].code = http://loinc.org#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NP_006594.3:p.S1178* "NP_006594.3:p.S1178*"

Instance: ATR-insertion-molc
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence ATR frameshift"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(CGPatientExample01)
* derivedFrom = Reference(ATR-insertion-var)
* component[0].code = http://loinc.org#48004-6
* component[0].code.text = "coding HGVS"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_001184.4:c.2878_2879insAGTAA "NM_001184.4:c.2878_2879insAGTAA"
* component[+].code = http://loinc.org#51958-7 "Transcript reference sequence [ID]"
* component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NM_001184.4 "NM_001184.4"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept.coding[0] = http://www.sequenceontology.org#SO:0001589 "frameshift_variant"
//kp Error while processing 'NP_001175.2:p.R960fs*2': Frameshift variants are not supported
// * component[+].code = http://loinc.org#48005-3 "Amino acid change (pHGVS)"
// * component[=].valueCodeableConcept = http://varnomen.hgvs.org#NP_001175.2:p.R960fs*2 "NP_001175.2:p.R960fs*2"
* component[+].code = TbdCodesCS#functional-effect
* component[=].valueCodeableConcept = http://www.sequenceontology.org#SO_0002054 "loss_of_function_variant"

Instance: EGFR-L858R-molc
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence EGFR L858R missense_variant"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(CGPatientExample01)
* derivedFrom = Reference(EGFR-L858R-var)
* component[0].code = http://loinc.org#48004-6
* component[0].code.text = "coding HGVS"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NM_005228.4:c.2573T>G "NM_005228.4:c.2573T>G"
* component[+].code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#feature-consequence
* component[=].valueCodeableConcept.coding[0] = http://www.sequenceontology.org#SO:0001583 "missense_variant"
* component[+].code = http://loinc.org#48005-3 "Amino acid change (pHGVS)"
* component[=].valueCodeableConcept = http://varnomen.hgvs.org#NP_005219.2:p.Leu858Arg "NP_005219.2:p.Leu858Arg"

Instance: ROS1-Fusion
InstanceOf: MolecularConsequence
Description: "Example Molecular Consequence ROS1 Fusion"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs#molecular-consequence
* subject = Reference(CGPatientExample01)
* derivedFrom = Reference(ROS1-Fusion-var)
* component[+].code = TbdCodesCS#functional-effect
* component[=].valueCodeableConcept = http://www.sequenceontology.org#SO:0001886 "transcript_fusion"

Instance: somaticPatient
InstanceOf: Patient
Description: "Somatic example Patient"
Usage: #example
* identifier.use = #temp
* identifier.type = $IDTYPE#MR "Medical record number"
* identifier.system = "http://www.somesystemabc.net/identifiers/persons"
* identifier.value = "1234567"
* identifier.period.start = "2021-01-01"
* identifier.assigner.display = "General Hospital"
* active = true
* name.use = #official
* name.family = "Hamsburg"
* name.given[0] = "Substance"
* name.given[+] = "Junior"
* gender = #male
* birthDate = "1987-09-01"

Instance: somaticStudy
InstanceOf: GenomicStudy
Description: "Example for somatic genetic study"
Usage: #example
* extension[0].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomic-study-analysis-ext"
* extension[=].valueReference = Reference(analysisTumorNormalDNA)
* extension[+].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomic-study-analysis-ext"
* extension[=].valueReference = Reference(analysisTumorRNA)
* status = #completed
* code.text = "My Test Cancer 700 Gene Panel"
* category = $OBSCAT#laboratory
* subject = Reference(somaticPatient)
* performedDateTime = "2023-02-02"
* reasonCode = $SCT#424993006 "Adenocarcinoma of lung, stage IV (disorder)"
* note.text = "I. Assay Overview: The Genomic Cancer Testing Assay is a comprehensive molecular examination designed to detect and quantify specific DNA and RNA alterations associated with malignancies. The test incorporates Next Generation Sequencing (NGS) technology to profile tumor samples, providing insights into the genomic alterations that drive tumor behavior and prognosis. II. Target Genes and Biomarkers: This assay evaluates an extensive panel of cancer-related genes, focusing on mutations, copy number variations, fusions, and select RNA expression levels known to be relevant in the oncology field. The analyzed genes are implicated in various cancer types and have established therapeutic, diagnostic, prognostic, or predictive relevance. Genes tested: ABL1, ATK1, ALK, APC, etc. III. Methodology: Sample Type: Formalin-fixed, paraffin-embedded (FFPE) tumor tissue. Sequencing Platform: Illumina-based NGS technology. Library Preparation: Amplification of specific target regions within the genes of interest. Bioinformatics Analysis: Employing advanced algorithms and databases to annotate and interpret the detected variants' clinical significance. IV. Assay Performance: Sensitivity: Detects variants with a mutant allele frequency (MAF) as low as 5%. Specificity: >99% for single nucleotide variants (SNVs) and indels. Reportable Range: Identification and quantification of SNVs, indels, fusions, amplifications, and RNA expression levels of specific genes. V. Clinical Interpretation: The report offers a concise clinical interpretation of identified genomic alterations, linking the detected variants to FDA-approved therapies, clinical trials, or the patient's potential response to specific treatments. VI. Limitations: The assay might not detect alterations present at low levels or in regions not covered by the test. Variants may be present in genes not included in the panel. The test does not assess epigenetic changes, like methylation status. False negatives and false positives, although rare, may occur due to the technical limitations of the methodology employed. VII. Quality Assurance: The laboratory follows CAP/CLIA guidelines, ensuring stringent quality control and assurance practices at each step, from sample reception to result reporting. VIII. Result Delivery: Results are delivered within 10-14 working days from the sample receipt. The report outlines detected genomic alterations, their clinical implications, and potential therapeutic options. IX. Contact: For further clarification or questions regarding the results, healthcare providers are encouraged to contact our molecular pathology board-certified experts."

Instance: analysisTumorNormalDNA
InstanceOf: GenomicStudyAnalysis
Description: "Example for Tumor Normal Genomics Study Analysis"
Usage: #example
* extension[GenomicStudyAnalysisRegions]
  * extension[studied][+].valueReference = Reference(DocumentReference/WES-FullSequencedRegion-GRCh38)
  * extension[studied][+].valueCodeableConcept = $HGNCID#HGNC:76 "ABL1"
  * extension[studied][+].valueCodeableConcept = $HGNCID#HGNC:427 "ALK"
  * extension[studied][+].valueCodeableConcept = $HGNCID#HGNC:583 "APC"
  * extension[uncalled][+].valueReference = Reference(DocumentReference/WES-UncallableRegions-GRCh38)
* extension[GenomicStudyAnalysisMethodType][+].valueCodeableConcept = $GSMETHODTYPE#sequence-analysis-of-the-entire-coding-region "Sequence analysis of the entire coding region"
* extension[GenomicStudyAnalysisMethodType][+].valueCodeableConcept = $GSMETHODTYPE#deletion-duplication-analysis "Deletion/duplication analysis"
* extension[GenomicStudyAnalysisChangeType][+].valueCodeableConcept = $SEQONT#SO:0001483 "SNV"
* extension[GenomicStudyAnalysisChangeType][+].valueCodeableConcept = $SEQONT#SO:0002007 "MNV"
* extension[GenomicStudyAnalysisChangeType][+].valueCodeableConcept = $SEQONT#SO:1000032 "delins"
* extension[GenomicStudyAnalysisChangeType][+].valueCodeableConcept = $SEQONT#SO:0001019 "CNV"
* extension[GenomicStudyAnalysisGenomeBuild].valueCodeableConcept = $LNC#LA26806-2 "GRCh38"
* extension[GenomicStudyAnalysisSpecimen][+].valueReference = Reference(Specimen/tumorSpecimen)
* extension[GenomicStudyAnalysisSpecimen][+].valueReference = Reference(Specimen/normalSpecimen)
* extension[GenomicStudyAnalysisFocus][+].valueReference = Reference(Patient/somaticPatient)
* extension[GenomicStudyAnalysisOutput][+]
  * extension[type].valueCodeableConcept = $GSFILETYPE#vcf "VCF"
  * extension[file].valueReference = Reference(DocumentReference/somaticVCFfile)
* status = #completed
* category = $OBSCAT#laboratory
* performedDateTime = "2023-02-02T01:01:10-06:00"
* note.text = "For technical reasons, BCR gene was deemed uncallable."
* subject = Reference(somaticPatient)

Instance: analysisTumorRNA
InstanceOf: GenomicStudyAnalysis
Description: "Example for just Tumor RNA Genomic Study Analysis"
Usage: #example
* extension[GenomicStudyAnalysisRegions].extension[studied][+].valueReference = Reference(DocumentReference/FullGenome-GRCh38)
* extension[GenomicStudyAnalysisMethodType].valueCodeableConcept = $GSMETHODTYPE#rna-analysis "RNA analysis"
* extension[GenomicStudyAnalysisChangeType][+].valueCodeableConcept = $SEQONT#SO:0001565 "gene_fusion"
* extension[GenomicStudyAnalysisChangeType][+].valueCodeableConcept = $SEQONT#SO:0001576 "transcript_variant"
* extension[GenomicStudyAnalysisSpecimen][+].valueReference = Reference(Specimen/tumorSpecimen)
* extension[GenomicStudyAnalysisFocus][+].valueReference = Reference(Patient/somaticPatient)
* status = #completed
* category = $OBSCAT#laboratory
* performedDateTime = "2023-02-02T01:01:10-06:00"
* subject = Reference(somaticPatient)

Instance: tumorSpecimen
InstanceOf: Specimen
Description: "Example for tumor specimen"
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/specimens"
* identifier.value = "222333"
* status = #available
* type = $SCT#122610009 "Specimen from lung obtained by biopsy (specimen)"
* subject = Reference(Patient/somaticPatient) "Hamsburg, Substance Jr"
* receivedTime = "2023-01-23T01:01:01Z"
* collection.collectedDateTime = "2023-01-01T01:01:00Z"
* collection.method = $SCT#129314006 "Biopsy - action"
* collection.bodySite = $SCT#3341006 "Right lung structure (body structure)"
* note.text = "Tumor: 20%"

Instance: normalSpecimen
InstanceOf: Specimen
Description: "Example for normal specimen"
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/specimens"
* identifier.value = "444555"
* status = #available
* type = $SCT#119297000 "Blood specimen (specimen)"
* subject = Reference(Patient/somaticPatient) "Hamsburg, Substance Jr"
* receivedTime = "2023-01-23T01:01:01Z"
* collection.collectedDateTime = "2023-01-01T01:01:00Z"

Instance: WES-UncallableRegions-GRCh38
InstanceOf: GenomicDataFile
Description: "Example for Genomic Data File WES-UncallableRegions-GRCh38"
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/files"
* identifier.value = "11146520"
* status = #current
* docStatus = #preliminary
* description = "Regions deemed uncallable (generally due to low coverage)."
* securityLabel = $CONFIDENTIALITY#R "Restricted"
* content.attachment.url = "https://chat.fhir.org/user_uploads/10155/WrHnNR_WvMcqM2aXQ3BRHxlA/WES-UncallableRegions-GRCh38.bed"
* content.attachment.title = "WES-UncallableRegions-GRCh38"

Instance: somaticVCFfile
InstanceOf: GenomicDataFile
Description: "Example for Genomic Data File somaticVCFfile"
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/files"
* identifier.value = "1134121"
* status = #current
* docStatus = #preliminary
* description = "VCF file containing identified variants"
* securityLabel = $CONFIDENTIALITY#R "Restricted"
* content.attachment.url = "https://chat.fhir.org/user_uploads/10155/Rp_GBGx6jz5reuS_Py7XQDjr/somaticVCFFile.vcf"
* content.attachment.title = "somatic VCF File"

Instance: FullGenome-GRCh38
InstanceOf: GenomicDataFile
Description: "Example for Genomic Data File FullGenome-GRCh38"
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/files"
* identifier.value = "1076146520"
* status = #current
* docStatus = #preliminary
* description = "Studied region for RNA sequencing."
* securityLabel = $CONFIDENTIALITY#R "Restricted"
* content.attachment.url = "https://chat.fhir.org/user_uploads/10155/ha97JexWz8T5PLHl7zoLsosd/Default.WGS.GRCh38.bed"
* content.attachment.title = "Full Genome"

Instance: TMBExample
InstanceOf: MolecularBiomarker
Description: "Example for Tumor Mutation Burden"
* id = "TMBExample"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-biomarker"
* status = #final "final"
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory].coding = $OBSCAT#laboratory
* category[mbCategory].coding = TbdCodesCS#biomarker-category
* category[geCategory].coding = $DIAGNOSTICSERVICE#GE
* code = $LNC#94076-7 "Mutations/Megabase [# Ratio] in Tumor"
* subject = Reference(somaticPatient)
* valueQuantity.value = 29.8
* component[biomarker-category].code = TbdCodesCS#biomarker-category
* component[biomarker-category].valueCodeableConcept.coding = MolecularBiomarkerOntologyCS#molgen "molecular sequence adjacent category"

Instance: MSIExample
InstanceOf: MolecularBiomarker
Description: "Example for MSI"
* id = "MSIExample"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-biomarker"
* status = #final "final"
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory].coding = $OBSCAT#laboratory
* category[mbCategory].coding = TbdCodesCS#biomarker-category
* category[geCategory].coding = $DIAGNOSTICSERVICE#GE
* code = $LNC#81695-9 "Microsatellite instability [Interpretation] in Cancer specimen Qualitative"
* subject = Reference(somaticPatient)
* valueCodeableConcept = $LNC#LA26203-2 "MSI-H"
* component[biomarker-category].code = TbdCodesCS#biomarker-category
* component[biomarker-category].valueCodeableConcept.coding = MolecularBiomarkerOntologyCS#molgen "molecular sequence adjacent category"

Instance: PDL1Example
InstanceOf: MolecularBiomarker
Description: "Example for PD-L1"
* id = "PDL1Example"
* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-biomarker"
* status = #final "final"
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory].coding = $OBSCAT#laboratory
* category[mbCategory].coding = TbdCodesCS#biomarker-category
* category[geCategory].coding = $DIAGNOSTICSERVICE#GE
* code = $LNC#85147-7 "PD-L1 by clone 22C3 in Tissue by Immune stain Report"
* subject = Reference(somaticPatient)
* valueCodeableConcept = $LNC#LA6576-8 "Positive"
* component[biomarker-category][0].valueCodeableConcept = $BIOMARKER#cellReceptorLigand "cell receptor ligand category"
* component[biomarker-category][1].valueCodeableConcept = $BIOMARKER#immuneStain "immune stain category"
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:17635 "CD274"
* note.text = "5% tumor proportion score"

Instance: HLA-A-haplotype1
InstanceOf: Haplotype
Description: "Example for HLA-A-haplotype1"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype Name"
* issued = "2023-02-01T00:00:00-05:00"
* valueCodeableConcept = $HLAALLELE#A*24:02 "HLA-A*24:02"
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:4931 "HLA-A"

Instance: HLA-A-haplotype2
InstanceOf: Haplotype
Description: "Example for HLA-A-haplotype2"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype Name"
* issued = "2023-02-01T00:00:00-05:00"
* valueCodeableConcept = $HLAALLELE#A*02:06 "HLA-A*02:06"
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:4931 "HLA-A"

Instance: HLA-B-haplotype1
InstanceOf: Haplotype
Description: "Example for HLA-B-haplotype1"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype Name"
* issued = "2023-02-01T00:00:00-05:00"
* valueCodeableConcept = $HLAALLELE#B*35:01 "HLA-B*35:01"
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:4932 "HLA-B"

Instance: HLA-B-haplotype2
InstanceOf: Haplotype
Description: "Example for HLA-B-haplotype2"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype Name"
* issued = "2023-02-01T00:00:00-05:00"
* valueCodeableConcept = $HLAALLELE#B*35:01 "HLA-B*35:01"
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:4932 "HLA-B"

Instance: HLA-C-haplotype1
InstanceOf: Haplotype
Description: "Example for HLA-C-haplotype1"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype Name"
* issued = "2023-02-01T00:00:00-05:00"
* valueCodeableConcept = $HLAALLELE#C*03:03 "HLA-C*03:03"
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:4933 "HLA-C"

Instance: HLA-C-haplotype2
InstanceOf: Haplotype
Description: "Example for HLA-C-haplotype2"
Usage: #example
* status = #final
* subject = Reference(Patient/somaticPatient)
* effectiveDateTime = "2023-02-01"
* performer[0] = Reference(pathologistPractitioner)
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#84414-2 "Haplotype Name"
* issued = "2023-02-01T00:00:00-05:00"
* valueCodeableConcept = $HLAALLELE#C*15:02 "HLA-C*15:02"
* component[gene-studied].valueCodeableConcept = $HGNCID#HGNC:4933 "HLA-C"

Instance: somaticReport
InstanceOf: GenomicReport
Description: "Example somatic GenomicReport"
Usage: #example
* extension[0].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomic-study-reference"
* extension[=].valueReference = Reference(somaticStudy) "somaticStudy"
* identifier.type.coding.code = #ACSN
* identifier.type.coding.display = "Accession ID"
* identifier.type.coding.system = "http://terminology.hl7.org/CodeSystem/v2-0203"
* identifier.value = "23-0000345"
* status = #final
* category[Genetics] = $DIAGNOSTICSERVICE#GE "Genetics"
* code.coding[0] = $LNC#51969-4 "Genetic analysis report"
* effectiveDateTime = "2023-02-02"
* basedOn = Reference(somaticServiceRequest)
* performer[0] = Reference(pathologistPractitioner)
* performer[+] = Reference(performingOrganization)
* resultsInterpreter[0] = Reference(supervisorPractitioner)
* resultsInterpreter[+] = Reference(performingOrganization)
* result[+] = Reference(NOTCH1-uncertain-var)
* result[+] = Reference(ZFHX3-uncertain-var)
* result[+] = Reference(MSH2-del-var)
* result[+] = Reference(NTHL1-snv-var)
* result[+] = Reference(STAG2-insertion-var)
* result[+] = Reference(ATR-insertion-var)
* result[+] = Reference(EGFR-L858R-var)
* result[+] = Reference(ROS1-Fusion-var)
* result[+] = Reference(NOTCH1-significance)
* result[+] = Reference(ZFHX3-significance)
* result[+] = Reference(MSH2-del-disease)
* result[+] = Reference(NTHL1-snv-disease)
* result[+] = Reference(STAG2-insertion-significance)
* result[+] = Reference(ATR-insertion-significance)
* result[+] = Reference(EGFR-L858R-significance)
* result[+] = Reference(ROS1-Fusion-disease)
* result[+] = Reference(ROS1-Fusion-therapuTrial)
* result[+] = Reference(ROS1-Fusion-therapuDrug)
* result[+] = Reference(EGFR-L858R-therapuDrug1)
* result[+] = Reference(EGFR-L858R-therapuDrug2)
* result[+] = Reference(TMB-therapuDrug)
* result[+] = Reference(NOTCH1-uncertain-molc)
* result[+] = Reference(ZFHX3-uncertain-molc)
* result[+] = Reference(MSH2-del-molc)
* result[+] = Reference(NTHL1-snv-molc)
* result[+] = Reference(STAG2-insertion-molc)
* result[+] = Reference(ATR-insertion-molc)
* result[+] = Reference(EGFR-L858R-molc)
* result[+] = Reference(ROS1-Fusion)
* result[+] = Reference(TMBExample)
* result[+] = Reference(MSIExample)
* result[+] = Reference(PDL1Example)
* result[+] = Reference(HLA-A-haplotype1)
* result[+] = Reference(HLA-A-haplotype2)
* result[+] = Reference(HLA-B-haplotype1)
* result[+] = Reference(HLA-B-haplotype2)
* result[+] = Reference(HLA-C-haplotype1)
* result[+] = Reference(HLA-C-haplotype2)
* conclusionCode[+] = $LNC#LA6576-8 "Positive"
* conclusion = "Patient is positive for Tier 1 / Level A variants, including variants with FDA-approved indications."

Instance: somaticServiceRequest
InstanceOf: ServiceRequest
Description: "Example for Somatic Testing Request"
Usage: #example
* identifier.type = $IDTYPE#LACSN
* identifier.type.text = "Laboratory Accession ID"
* identifier.system = "http://www.somesystemabc.net/identifiers/serviceRequests"
* identifier.value = "1111jh11112"
* status = #active
* intent = #plan
* code.text = "My Test Cancer 700 Gene Panel"
* subject = Reference(somaticPatient)
* reasonCode = $SCT#424993006 "Adenocarcinoma of lung, stage IV (disorder)"
* requester = Reference(orderingPractitioner)

Instance: orderingPractitioner
InstanceOf: Practitioner
Description: "Example of an ordering provider"
Usage: #example
* identifier.system = "http://hl7.org/fhir/sid/us-npi"
* identifier.value = "1245319599"
* active = true
* name.family = "Submitter"
* name.given = "Test"
* name.prefix = "Dr"
* address.line = "1000 SE Submitter Ave"
* address.city = "Los Angeles"
* address.state = "CA"
* address.postalCode = "90027"

Instance: pathologistPractitioner
InstanceOf: Practitioner
Description: "Example of a pathologist"
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/persons"
* identifier.value = "11d116"
* active = true
* name.family = "Dolin"
* name.given = "Test"
* name.prefix = "Dr"

Instance: supervisorPractitioner
InstanceOf: Practitioner
Description: "Example of a supervisor"
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/persons"
* identifier.value = "11d1x16"
* active = true
* name.family = "Chamala"
* name.given = "Test"
* name.prefix = "Dr"

Instance: performingOrganization
InstanceOf: Organization
Description: "Example of a Pathology lab"
Usage: #example
* identifier.system = "http://www.somesystemabc.net/identifiers/CLIA"
* identifier.value = "10000DLAB8"
* name = "My Test Pathology Laboratories"
* telecom[0].system = #phone
* telecom[=].value = "(100) 200-3000"
* telecom[+].system = #fax
* telecom[=].value = "(400) 500-6000"
* telecom[+].system = #url
* telecom[=].value = "http://www.example.com/mytestpathlabs"
* address.line = "1000 SE Lab Ave"
* address.city = "Los Angeles"
* address.state = "CA"
* address.postalCode = "90027"


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/specimen-hla-r4.fsh

Instance: specimen-hla-r4
InstanceOf: Specimen
Description: "Example HLA specimen"
Usage: #example
* identifier.system = "http://specimensrus.com"
* identifier.value = "001^123456789"
* type = $SCT#258564008 "Buccal smear sample"
* subject.display = "de-identified patient"
* receivedTime = "2018-05-01"
* collection.method = $SCT#418622002 "Taking oral swab (procedure)"
* collection.bodySite = $SCT#261063000 "Buccal space"

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/examples/variant-with-molec-consequences.fsh

Instance: variant-with-molec-consequences
InstanceOf: Variant
Description: "Variant with molecular consequences"
Usage: #example
* status = #final
* category[labCategory] = $OBSCAT#laboratory
* category[geCategory] = $DIAGNOSTICSERVICE#GE
* code = $LNC#69548-6 "Genetic variant assessment"
* subject = Reference(Patient/HG00403)
* effectiveDateTime = "2023-06-01"
* performer = Reference(ExampleLab)
* valueCodeableConcept = $LNC#LA9633-4 "Present"
* component[+].code = $LNC#48002-0 "Genomic source class"
* component[=].valueCodeableConcept.coding = $LNC#LA6683-2 "Germline"
* component[+].code = http://loinc.org#48013-7 "Genomic reference sequence ID"
* component[=].valueCodeableConcept = http://www.ncbi.nlm.nih.gov/refseq#NC_000001.10
* component[+].code = $LNC#53034-5 "Allelic state"
* component[=].valueCodeableConcept = $LNC#LA6706-1 "Heterozygous"
* component[+].code = http://loinc.org#81252-9 "Discrete genetic variant"
* component[=].valueCodeableConcept = $HGVS#NC_000001.10:g.86852621A>G "NC_000001.10:g.86852621A>G"
* component[+].code = http://loinc.org#81258-6 "Sample VAF"
* component[=].valueQuantity = 0.6 '1' "relative frequency of a particular allele in the specimen"
* component[+].code = http://loinc.org#69547-8 "Genomic Ref allele [ID]"
* component[=].valueString = "A"
* component[+].code = http://loinc.org#69551-0 "Genomic Alt allele [ID]"
* component[=].valueString = "G"
* component[+].code = http://loinc.org#92822-6 "Genomic coord system"
* component[=].valueCodeableConcept = http://loinc.org#LA30100-4 "0-based interval counting"
* component[+].code = http://loinc.org#81254-5 //"Variant exact start-end"
* component[=].code.text = "Variant exact start-end"
* component[=].valueRange.low.value = 86852620
* component[+].code = http://loinc.org#92821-8 //"Population allele frequency"
* component[=].code.text = "Population allele frequency"
* component[=].valueQuantity = 0.327084 '1'

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/Aliases.fsh

// Code Systems
Alias: $LNC = http://loinc.org
Alias: $SCT = http://snomed.info/sct
Alias: $UCUM = http://unitsofmeasure.org
Alias: $NCIT = http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl
Alias: $ICD10CM = http://hl7.org/fhir/sid/icd-10-cm
Alias: $RXN = http://www.nlm.nih.gov/research/umls/rxnorm
Alias: $GTR = http://www.ncbi.nlm.nih.gov/gtr
Alias: $CLINVAR = http://www.ncbi.nlm.nih.gov/clinvar
Alias: $IDTYPE = http://terminology.hl7.org/CodeSystem/v2-0203
Alias: $HGNCID = 	http://www.genenames.org
Alias: $HGNCGROUP = http://www.genenames.org/genegroup
Alias: $HGVS = http://varnomen.hgvs.org
Alias: $SPECIMENTYPE = http://terminology.hl7.org/CodeSystem/v2-0487
Alias: $SEQONT = http://www.sequenceontology.org
Alias: $OBSCAT = http://terminology.hl7.org/CodeSystem/observation-category
Alias: $OBSINT = http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation
Alias: $DIAGNOSTICSERVICE = http://terminology.hl7.org/CodeSystem/v2-0074
Alias: $TASKSTATUS = http://hl7.org/fhir/task-status
Alias: $TASKINTENT = http://hl7.org/fhir/task-intent
Alias: $REQUESTINTENT = http://hl7.org/fhir/request-intent
Alias: $REQUESTSTATUS = http://hl7.org/fhir/request-status
Alias: $RISKQUAL = 	http://terminology.hl7.org/CodeSystem/risk-probability
Alias: $NCTTRIAL = http://clinicaltrials.gov
Alias: $HPO = http://human-phenotype-ontology.org
Alias: $NCBIREFSEQ = http://www.ncbi.nlm.nih.gov/refseq
Alias: $HLAALLELE = http://www.ebi.ac.uk/ipd/imgt/hla
Alias: $GLSTRING = http://glstring.org
Alias: $PHARMVAR = http://www.pharmvar.org
Alias: $SNP = http://www.ncbi.nlm.nih.gov/projects/SNP
Alias: $ISCN = https://iscn.karger.com //defined in THO with https
Alias: $PARTICIPATIONTYPE = http://terminology.hl7.org/CodeSystem/v3-ParticipationType
Alias: $CONFIDENTIALITY = http://terminology.hl7.org/CodeSystem/v3-Confidentiality
Alias: $GSMETHODTYPE = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/genomic-study-method-type-cs
Alias: $GSFILETYPE = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/genomic-study-data-format-cs
Alias: $BIOMARKER = http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/molecular-biomarker-ontology-cs
Alias: $MONDO = http://purl.obolibrary.org/obo/mondo.owl

//Extensions
Alias: operationAllowedType = http://hl7.org/fhir/StructureDefinition/operationdefinition-profile


---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/CGCodeSystem.fsh

CodeSystem: SequencePhaseRelationshipCS
Id: sequence-phase-relationship-cs
Title: "Sequence Phase Relationship Codes"
Description: "Code System for specific types of relationships"
* ^url = "http://terminology.hl7.org/CodeSystem/sequence-phase-relationship-cs"
* ^caseSensitive = true
* ^experimental = false
* #Cis "Cis" "Phase is in cis (same chromosome)"
* #Trans "Trans" "Phase is in trans (different chromosomes)"
* #Indeterminate "Indeterminate" "Phase is unable to be determined"
* #Unknown "Unknown" "Phase is unknown"

CodeSystem: TbdCodesCS
Id: tbd-codes-cs
Title: "To Be Determined Codes"
Description: "These codes are 'TBD' codes, meaning they are important concepts but codes have not been identifed in available code systems. The CG WG anticpates these codes maturing over time. We may remove some codes as we refine the concepts, or identify suitable replacement codes from other code systems. As we approach normative and these concepts are validated, for codes that remain, we plan to request formal LOINC codes to replace these temporary codes."
* ^caseSensitive = true
* ^experimental = true
* #associated-therapy "Associated Therapy" "The non-medication therapy (procedure) associated with this implication."
* #molecular-consequence "Molecular Consequence" "The calculated or observed effect of a DNA variant on its downstream transcript and, if applicable, ensuing protein sequence."
* #feature-consequence "Feature Consequence" "The structural implications of a variant (e.g. the variant disrupts a regulatory region, the variant is an inframe insertion)."
* #diagnostic-implication "Diagnostic Implication" "An observation linking a genomic finding with a knowledge base, providing context that may aid in diagnosing a patient with a particular phenotype or condition."
* #therapeutic-implication "Therapeutic Implication" "An observation linking a genomic finding with a knowledge base, providing potential evidence of an interaction with a specified medication or non-medicinal therapy. Ramifications may include alterations in drug metabolism (or pharmacokinetics) that determine the concentration of the drug, prodrug, and/or break-down products over time; alterations in drug efficacy (or pharmacodynamics) that determine how effective a drug is at a given concentration; alterations that alter the risk of adverse drug events, or other types of implications that indicate altered responsiveness to other types of therapies."
* #functional-effect "Functional Effect" "The effect of a variant on downstream biological products or pathways (from Sequence Ontology)."
* #conclusion-string "Conclusion Text" "Clinical conclusion (interpretation) of the observation."
* #condition-inheritance "Condition Inheritance" "The transmission pattern of the condition/phenotype in a pedigree."
* #variant-confidence-status "Variant Confidence Status" "The confidence of a true positive variant call."
* #repeat-motif "Repeat Expansion Motif" "Nucleotides of a repeat expansion motif."
* #repeat-number "Number of Repeat Expansions" "Number of repeats of a repeat expansion."
* #biomarker-category "A characterization of a given biomarker observation."
* #protein-ref-seq "An identifier for the protein reference sequence."

CodeSystem: ClinVarEvidenceLevelCustomCS
Id: clinvar-evidence-level-custom-cs
Title: "ClinVar Evidence Level Example Codes"
Description: "ClinVar contains examples of evidence level concepts that are not conflated with clinical significance. These can be found on  ClinVar https://www.ncbi.nlm.nih.gov/clinvar/docs/review_status/ . These examples are informational only, for copyright information contact the relevant source."
* ^caseSensitive = true
* ^experimental = true
* #4-star	"4 star"	"Supported by practice guideline"
* #3-star	"3 star"	"Supported by expert panel review"
* #2-star	"2 star"	"Supported by submission by multiple-submitters with documentation of criteria for assertion"
* #1-star	"1 star"	"Single submitter providing interpretation and documentation of criteria for assertion"
* #no-star	"no star"	"submitted, no evidence"

CodeSystem: PharmGKBEvidenceLevelCustomCS
Id: pharmgkb-evidence-level-custom-cs
Title: "PharmGKB Evidence Level Example Codes"
Description: "PharmGKB contains examples of evidence level concepts that are not conflated with clinical significance. These can be found on PharmGKB https://www.pharmgkb.org/page/clinAnnLevels. These examples are informational only, for copyright information contact the relevant source."
* ^caseSensitive = true
* ^experimental = true
* #1A	"PGKB 1A"	"High level of evidence. Supported by a guideline or FDA label with variant specific prescribing guidance. Additionally, supported by at least one publication."
* #1B	"PGKB 1B"	"High level of evidence. But, NOT supported by a guideline or FDA label with variant specific prescribing guidance. Additionally, supported by at least one publication."
* #2A	"PGKB 2A"	"Moderate level of evidence. Supported by being a KNOWN pharmacogene on https://www.pharmgkb.org/vips. Also, found in multiple studies but may have a minority of studies that did not support the majority opinion. Supported by at least two indpendent publications."
* #2B	"PGKB 2B"	"Moderate level of evidence. NOT in the list of very important, KNOWN pharmacogenes. Found in multiple studies but may have a minority of studies that did not support the majority opinion. Supported by at least two indpendent publications."
* #3	"PGKB 3"	"Low-level of evidence. Where either the assocation is based on a single study, failed to be reproduced, or preliminary evidence."
* #4	"PGKB 4"	"The evidence does not support an association between the variant and the drug phenotype. (negative)"

CodeSystem: VariantConfidenceStatusCS
Id: variant-confidence-status-cs
Title: "Variant Confidence Status Codes"
Description: "A code that represents the confidence of a true positive variant call."
* ^url = "http://terminology.hl7.org/CodeSystem/variant-confidence-status-cs"
* ^caseSensitive = true
* ^experimental = true
* #high	        "High"	        "High confidence true positive variant call. Suitable for clinical reporting."
* #intermediate	"Intermediate"	"Candidate true positive variant call. Unable to confirm without additional testing."
* #low	        "Low"	        "High confidence false positive variant call."

CodeSystem: CodedAnnotationTypesCS
Id: coded-annotation-types-cs
Title: "Coded Annotation Type Codes"
Description: "Code System for specific types of coded annotations"
* ^caseSensitive = true
* ^experimental = true
* #test-disclaimer "Test Disclaimer" "Test disclaimer with the report"
* #test-methodology "Test Methodology" "Additional details regarding the test methdology"
* #result-confirmation "Result Confirmation" "Additional details regarding the confirmation of the results"

CodeSystem: MolecularBiomarkerOntologyCS
Id: molecular-biomarker-ontology-cs
Title: "Molecular Biomarker Ontology Codes"
Description: "This (experimental) code system provides for a draft categorization of biomarkers along several axes. Based on ballot feedback, the committee anticipates significant revisions."
* ^status = #active
* ^content = #complete
* ^version = "1.0.0"
* ^caseSensitive = true
* ^experimental = true
* ^hierarchyMeaning = #is-a
* #_physiologyBiomarkerCategory "Molecular biomarker physiologic role category" "A category based on the normal bodily function of the biomarker."
  * #antibody "antibody category" "Molecular biomarker physiologic role is an antibody."
  * #antgen "antigen category" "Molecular biomarker physiologic role is an antigen."
  * #cellReceptor "cell receptor category" "Molecular biomarker physiologic role is a cell receptor."
  * #cellReceptorLigand "cell receptor ligand category" "Molecular biomarker physiologic role is a cell receptor ligand."
* #_moleculeTypeBiomarkerCategory "Molecular biomarker molecular type category" "A category based on the type of biomarker molecule."
  * #carbohydrate "carbohydrate category" "Molecular biomarker molecule type is a carbohydrate."
  * #lipid "lipid category" "Molecular biomarker molecule type is a lipid."
  * #nucleicAcid "nucleic acid category" "Molecular biomarker molecule type is a nucleic acid."
  * #protein "protein category" "Molecular biomarker molecule type is a protein. Includes single or multi-subunit proteins with or without post-translational modifications."
* #_methodBiomarkerCategory "Molecular biomarker method type category" "A category based on the measurement ascertainment method of the biomarker. (Methods are drawn from LOINC (https://loinc.org/kb/users-guide/major-parts-of-a-loinc-term/#type-of-method-6th-part) and NCBI Genetic Testing Registry (https://www.ncbi.nlm.nih.gov/gtr/))."
  * #enzymeAssay "enzyme assay category" "Molecular biomarker method is enzyme assay."
  * #flowCytometry "flow cytometry category" "Molecular biomarker method is flow cytometry."
  * #immuneStain "immune stain category" "Molecular biomarker method is immune stain."
  * #immunoassay "immunoassay category" "Molecular biomarker method is immunoassay."
  * #methylationAnalysis "methylation analysis category" "Molecular biomarker method is methylation analysis."
  * #molgen "molecular sequence adjacent category" "Molecular biomarker method is molecular sequence adjacent observation."



---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/CGConceptMaps.fsh

// Concept Maps
Instance: dna-change-type-map
InstanceOf: ConceptMap
Usage: #definition
Description: "Mapping from http://loinc.org ValueSet for DNA Change Type to http://www.sequenceontology.org codes"
* title = "DNA Change Type Map"
* experimental = false
* name = "DNAChangeType"
* status = #draft
* copyright = "This material contains content from LOINC (http://loinc.org). LOINC is copyright © 1995-2020, Regenstrief Institute, Inc. and the Logical Observation Identifiers Names and Codes (LOINC) Committee and is available at no cost under the license at http://loinc.org/license. LOINC® is a registered United States trademark of Regenstrief Institute, Inc."
* publisher = "HL7 International Clinical Genomics Work Group"
* description = "LOINC and SequenceOntology mappings for dna change type"
* sourceCanonical = "http://loinc.org/vs/LL379-9"
* targetCanonical = "http://hl7.org/fhir/uv/genomics-reporting/ValueSet/dna-change-type-vs"
* group[0].source = $LNC
* group[0].target = $SEQONT
* group[0].element[0].code = $LNC#LA9658-1
* group[0].element[0].display = "Wild type"
* group[0].element[0].target[0].code = $SEQONT#SO:0002073
* group[0].element[0].target[0].display = "no_sequence_alteration"
* group[0].element[0].target[0].equivalence = #equivalent
* group[0].element[1].code = $LNC#LA6692-3
* group[0].element[1].display = "Deletion"
* group[0].element[1].target[0].code = $SEQONT#SO:0000159
* group[0].element[1].target[0].display = "deletion"
* group[0].element[1].target[0].equivalence = #equivalent
* group[0].element[2].code = $LNC#LA6686-5
* group[0].element[2].display = "Duplication"
* group[0].element[2].target[0].code = $SEQONT#SO:1000035
* group[0].element[2].target[0].display = "duplication"
* group[0].element[2].target[0].equivalence = #equivalent
* group[0].element[3].code = $LNC#LA6687-3
* group[0].element[3].display = "Insertion"
* group[0].element[3].target[0].code = $SEQONT#SO:0000667
* group[0].element[3].target[0].display = "insertion"
* group[0].element[3].target[0].equivalence = #equivalent
* group[0].element[4].code = $LNC#LA6688-1
* group[0].element[4].display = "Insertion/Deletion"
* group[0].element[4].target[0].code = $SEQONT#SO:1000032
* group[0].element[4].target[0].display = "delins"
* group[0].element[4].target[0].equivalence = #equivalent
* group[0].element[5].code = $LNC#LA6689-9
* group[0].element[5].display = "Inversion"
* group[0].element[5].target[0].code = $SEQONT#SO:1000036
* group[0].element[5].target[0].display = "inversion"
* group[0].element[5].target[0].equivalence = #equivalent
* group[0].element[6].code = $LNC#LA6690-7
* group[0].element[6].display = "Substitution"
* group[0].element[6].target[0].code = $SEQONT#SO:1000002
* group[0].element[6].target[0].display = "substitution"
* group[0].element[6].target[0].equivalence = #equivalent

---

File: repos/HL7_SLASH_genomics-reporting/input/fsh/CGDatatypes.fsh

Profile:        CodedAnnotation
Parent:         Annotation
Id:             coded-annotation
Title:          "Coded Annotation"
Description:    "Annotation DataType with added CodeableConcept extension element"
* extension contains AnnotationCode named code 0..1


---

File: repos/HL7_SLASH_genomics-reporting/input/pages/changes.md

### Version = STU3
- FHIR version: 4.0.1

#### Summary of updates
  
- Introduction of new operations along with extensive implementation guidance and examples. Refer to [here](operations.html) for details.
- Addition of support for GenomicStudy to capture structed, study level metadata. This is back porting a resource introduced in R5 as a set of profiles, code systems, and value sets into this R4 based IG. This includes structured extensions to capture region studied details, and the existing profile was removed. See [here](StructureDefinition-genomic-study.html) for more information.
- Broadening support for biomarker observations by introducing a new, more generic MolecularBiomarker profile, along with refactoring existing profiles and examples. Explore the new profile [here](StructureDefinition-molecular-biomarker.html).
- Improved guidance on the Somatic Reporting page, including a detailed synthetic report and additional example instances. See the new page [here](somatics.html).
- Update in the representation of molecular consequences through the creation of a new Molecular Consequences observation profile that is treated as an Implication. This is a non-passive change, as several Variant(Observation).component and DiagnosticImplication(Observation).component slices were moved to the new profile. See [here](StructureDefinition-molecular-consequence.html) for the new profile. A list of changes includes: 
  * Variant
    * rename component:coding-hgvs to component:representative-coding-hgvs (same LOINC code)
    * rename component:transcript-ref-seq to component:representative-transcript-ref-seq (same LOINC code)
    * rename component:protein-hgvs to component:representative-protein-hgvs (same LOINC code)
    * delete component:amino-acid-change-type
    * move component:molecular-consequence to Molecular Consequence profile, renamed component:feature-consequence
  * DiagnosticImplication
    * move component:functional-effect to Molecular Consequence profile
- Support repeat expansions on the Variant profile. See [here](sequencing.html#defining-repeat-expansion-variants) for guidance on the new slices.

#### Trackers

|JIRA&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Summary|
|----|-------|
|[FHIR-25170](https://jira.hl7.org/browse/FHIR-25170)|Send related artifacts at the Observation.component level|
|[FHIR-28943](https://jira.hl7.org/browse/FHIR-28943)|Introduced new MolecularBiomarker artifacts|
|[FHIR-31030](https://jira.hl7.org/browse/FHIR-31030)|Add pattern for GenomicReport.code for LOINC 51969-4|
|[FHIR-31506](https://jira.hl7.org/browse/FHIR-31506)|Add genetics category to observation profiles|
|[FHIR-32101](https://jira.hl7.org/browse/FHIR-32101)|Observation.component to capture variant inheritance basis|
|[FHIR-32696](https://jira.hl7.org/browse/FHIR-32696)|Improved somatic reporting guidance|
|[FHIR-34418](https://jira.hl7.org/browse/FHIR-34418)|Support repeat expansions|
|[FHIR-35864](https://jira.hl7.org/browse/FHIR-35864)|Add GenomicStudy to support study-level metadata|
|[FHIR-36041](https://jira.hl7.org/browse/FHIR-36041)|New operations, additional guidance|
|[FHIR-37892](https://jira.hl7.org/browse/FHIR-37892)|Add phase data to find subject variants operation|
|[FHIR-37893](https://jira.hl7.org/browse/FHIR-37893)|Add ranges parameter to subject phenotype operations|
|[FHIR-40320](https://jira.hl7.org/browse/FHIR-40320)|Create a new MolecularConsequence profile|
|[FHIR-40805](https://jira.hl7.org/browse/FHIR-40805)|Support copy number range and decimal in Variant profile|
|[FHIR-41245](https://jira.hl7.org/browse/FHIR-41245)|Merge predicted-therapeutic-implication slice into therapeutic-implication|
|[FHIR-41246](https://jira.hl7.org/browse/FHIR-41246)|Remove prognosis slice from implications, and prognostic-implication TBD code|
|[FHIR-41247](https://jira.hl7.org/browse/FHIR-41247)|For therapy- and medication-assessed extensions, move extensions to corresponding component slices|
|[FHIR-41248](https://jira.hl7.org/browse/FHIR-41248)|Add region studied meta data to Genomic Study and remove existing profile|
|[FHIR-41355](https://jira.hl7.org/browse/FHIR-41355)|Add new molecular consequence phenotype operations|
|[FHIR-41587](https://jira.hl7.org/browse/FHIR-41587)|Additional meta data for Genomic Study for regions: studied, called, uncalled|
|[FHIR-41610](https://jira.hl7.org/browse/FHIR-41610)|Guidance for consistently representing Variant value (especially for pertinent negatives)|
|[FHIR-42850](https://jira.hl7.org/browse/FHIR-42850)|Cleanup guidance for star alleles|
|[FHIR-43574](https://jira.hl7.org/browse/FHIR-43574)|Resolve discrepancies on Somatic page|
|[FHIR-43517](https://jira.hl7.org/browse/FHIR-43517)|Enhance somatic table of examples|
|[FHIR-43744](https://jira.hl7.org/browse/FHIR-43744)|Add additional GenomicStudyAnalysis metadata|
|[FHIR-43745](https://jira.hl7.org/browse/FHIR-43745)|Advance several GenomicStudyAnalysis value sets to Preferred|
|[FHIR-45434](https://jira.hl7.org/browse/FHIR-45434)|Add clarity to representation of Significance and Evidence in implication profiles|
|[FHIR-43691](https://jira.hl7.org/browse/FHIR-43691)|Remove Overall Interpretation profile and provide [guidance](StructureDefinition-genomic-report.html#overall-interpretation-and-conclusion) for GenomicsReport attributes for conclusion and conclusionCode.
|[FHIR-43745](https://jira.hl7.org/browse/FHIR-43745)|Advance several GenomicStudyAnalysis value sets to Preferred|


---

File: repos/HL7_SLASH_genomics-reporting/input/pages/codings.md

Genomics covers a broad landscape involving many organizations outside of HL7 that have established codes and name systems for different use cases. The <a href="https://confluence.hl7.org/display/TA">HL7 Terminology Authority (HTA)</a> governs the use of terminologies that are external to HL7 and they have begun to document these. As a part of the <a href="https://confluence.hl7.org/display/TSMG">Terminology Services Management Group</a>, the HTA is working with these organizations to establish canonical URIs that can be used as HL7 code and name systems. HL7 has established <a href="https://terminology.hl7.org">terminology.hl7.org (THO)</a> as the authoritative source for code system information, including <a href="https://terminology.hl7.org/external_terminologies.html">external terminologies</a> Many of these are also listed in <a href="{{site.data.fhir.path}}terminologies-systems.html">the core specification guidance</a>.

We have listed many, but not all, of genomics-related systems below. In this table, we provide links for more information -- to the THO or HTA entries when available, or to the organization home page if not. We also list a URI that has been approved by the THO/HTA or by <a href="https://confluence.hl7.org/display/VOC/Vocabulary+Work+Group">Vocabulary Work Group</a>, and a coding system label defined in the <a href="http://www.hl7.org/implement/standards/product_brief.cfm?product_id=279">HL7 Version 2.5.1 Implementation Guide: Laboratory Results Interface</a> (LRI).

For those organizations that list the URI as "Not Assigned," or "Not Official," check the <a href="https://terminology.hl7.org/external_terminologies.html">THO</a> or <a href="https://confluence.hl7.org/display/TA/External+Terminologies+-+Information">HTA sites</a> to see if it has been added since the publication of this IG. If not, we recommend following the <a href="https://confluence.hl7.org/display/TA/External+Code+System+Owner+Engagement+Process">External Code System Owner Engagement Processing</a>. As a last resort, we suggest using the organization site URL, change any "https" to "http", and remove trailing slashes.

Please contact the <a href="https://confluence.hl7.org/display/CGW">Clinical Genomics Workgroup</a> co-chairs if you have any questions about using these genomics code systems.

<style type="text/css">
    table {
        border-collapse: collapse;
        border: 1px solid grey;
        overflow-y: scroll;
        table-layout: fixed;
        overflow: hidden;
        word-break: break-word;
    }
    tbody {
        overflow-x: hidden;
    }
    th {
        background-color: lightgrey;
        border: 1px solid gray;
        border-bottom-width: 0px;
        text-align: center;
        vertical-align: middle;
    }
    td {
        border: 1px solid gray;
        border-width: 1px;
        text-align: left;
        vertical-align: top
    }
    tr:first-child td {
        border-top-width: 0;
        border-left-width: 0;
    }
    tr:nth-child(even) {
        background-color: #eee;
    }
    th:nth-child(1), td:nth-child(1) {
      width: 15%;
    }
    th:nth-child(2), td:nth-child(2) {
      width: 55%;
    }
    th:nth-child(3), td:nth-child(3) {
      width: 30%;
    }
</style>

|Coding System|Description|URI<br><br>LRI Code|
|---|---|---|
|<a href="https://clinicaltrials.gov/">ClinicalTrials.gov</a>|Database of privately and publicly funded clinical studies conducted around the world.|**URI:** http://clinicaltrials.gov (Not Official)<br><br>**LRI Code:** None|
|<a href="https://terminology.hl7.org/CodeSystem-ClinVarV.html">ClinVar Variant ID</a>|ClinVar processes submissions reporting variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data. The alleles described in submissions are mapped to reference sequences, and reported according to the HGVS standard. ClinVar includes simple and complex variants composed of multiple small variants. However, it now also includes large structural variants, which have a known clinical implication. So now simple, complex and many structural variants can all be found in ClinVar. The ClinVar records have a field for Allele ID and for Variant ID. We focus on the variant ID in this guide. This coding system uses the variant ID as the code and the variant name from NCBI’s "variant_summary.txt.gz" file as the code’s print string. The "variant_summary.txt.gz" file caries more than 20 useful fields, including the separate components of the variant name, the cytogenetic location, the genomic reference, etc. So based on the Variant ID, you can use ClinVar to find most you would ever want to know about the variant. In the LHC Clinical Table Search Service and LHC-Forms, we have indexed many of these attributes to assist users and applications that need to find the ID for a particular variant.|**URI:** http://www.ncbi.nlm.nih.gov/clinvar<br><br>**LRI Code:** CLINVAR-V|
|<a href="https://cancer.sanger.ac.uk/cosmic">COSMIC</a>|COSMIC, the Catalogue Of Somatic Mutations In Cancer, is the world's largest and most comprehensive resource for exploring the impact of somatic mutations in human cancer. COSMIC includes only simple somatic (cancer) mutations, one per unique mutation ID. The code is the COSMIC mutation ID, and the name is constructed from Ensembl transcript reference sequences and p.HGVS that use the single letter codes for amino acids. It carries fields analogous to most of the key fields in ClinVar, but its reference sequences are Ensembl transcript reference sequences with prefixes of ENST; it specifies amino acid changes with the older HGVS single letter codes and it carries examples of primary cancers and primary tissues - fields that are not in ClinVar. COSMIC's source table includes multiple records per mutation - one per submission. The COSMIC- SimpleVariants table that we have extracted from the original file includes only one record per unique mutation – a total of more than 3 million records. These contents are copyright COSMIC (http://cancer.sanger.ac.uk/cosmic/license). LHC has produced a lookup table for these records, and for users to look up particular mutation IDs, both with permission from COSMIC. However, interested parties must contact COSMIC directly for permission to download these records.|**URI:** http://cancer.sanger.ac.uk/cancergenome/projects/cosmic (Not Official)<br><br>**LRI Code:** COSMIC-Smpl, COSMIC-Stru|
|<a href="https://medlineplus.gov/genetics/understanding/howgeneswork/genelocation/">Cytogenetic (chromosome) location</a>|Chromosome location (AKA chromosome locus or cytogenetic location), is the standardized syntax for recording the position of genes and large variants. It consists of three parts: the Chromosome number (e.g. 1-22, X, Y), an indicator of which arm – either “p” for the short or “q” for the long, and then generally a series of numbers separated by dots that indicate the region and any applicable band, sub-band, and sub-sub-band of the locus (e.g. 2p16.3). There are other conventions for reporting ranges and locations at the ends of the chromosomes. The table of these chromosome locations was loaded with all of the locations found in NCBI’s ClinVar variation tables. It will expand as additional sources become available. This does not include all finely grained chromosome locations that exist. Users can add to it as needed.|**URI:** (Not Assigned)<br><br>**LRI Code:** Chrom-Loc|
|<a href="https://terminology.hl7.org/CodeSystem-v3-dbSNP.html">dbSNP</a>|The Short Genetic Variations database (dbSNP) is a public-domain archive maintained by NCBI for a broad collection of short genetic polymorphisms. The SNP ID is unique for each position and length of DNA change. For example, a change of three nucleotides will have a different SNP ID than a change of four nucleotides at the same locus, but the code will be the same for all changes at the same locus and with the same length. So, to specify a variation in relation to a reference sequence, both the alt allele and the SNP code must be included.|**URI:** http://www.ncbi.nlm.nih.gov/projects/SNP<br><br>**LRI Code:** dbSNP|
|<a href="https://www.ncbi.nlm.nih.gov/dbvar/">dbVar</a>|dbVar is NCBI's database of genomic structural variations (including copy number variants) that are larger than 50 contiguous base pairs. It is the complement of dbSNP, which identifies variants occurring in 50 or fewer contiguous base pairs. dbVar contains insertions, deletions, duplications, inversions, multi-nucleotide substitutions, mobile element insertions, translocations, and complex chromosomal rearrangements. dbVar carries structured Germline and Somatic variants in separate files. Accordingly, we have divided the coding system for dbVar the same way. This coding system represents the Germline dbVar variants. Its record ID may begin with one of four prefixes: nsv, nssv, esv and essv. These are accession prefixes for variant regions (nsv) and variant calls (or instances, nssv), respectively. Typically, one or more variant instances (nssv – variant calls based directly on experimental evidence) are merged into one variant region (nsv – a pair of start-stop coordinates reflecting the submitters’ assertion of the region of the genome that is affected by the variant instances). The “n” preceding sv or indicates that the variants were submitted to NCBI (dbVar). The prefix, “e” for esv and essv represent variant entities (corresponding to NCBI’s nsv and nssv) that were submitted to EBI (DGVa). The relation between variant call, and variant region, instances is many to one. The LHC lookup table for dbVar germline variations includes both variant instances (essv or nssv) and the variant region records (nsv, esv). Users can sub-select by searching on the appropriate prefix.|**URI:** (Not Assigned)<br><br>**LRI Code:** dbVar-GL, dbVar-Som|
|<a href="https://www.ncbi.nlm.nih.gov/gene">NCBI Gene Codes</a>|Gene supplies gene-specific connections in the nexus of map, sequence, expression, structure, function, citation, and homology data.When applicable, this variable identifies the gene on which the variant is located. The gene identifier is also carried in the transcript reference sequence database, and is part of a full HGVS expression. Not all genes have HGNC names and codes so NCBI has created gene IDs that cover the genes that are not registered by HGNC. For example, SHOX (HGNC:10853) is in the pseudoautosomal region of ChrX and ChrY. NCBI gene symbols are 'SHOX' on ChrX and 'SHOX-2' on ChrY.|**URI:** (Not Assigned)<br><br>**LRI Code:** NCBI-gene|
|<a href="http://glstring.org">GL String</a>|The GL String Code (GLSC) code system combines Genotype List String (GL String) grammar with established HLA and KIR nomenclatures, thereby specifying a syntax for encoding of nomenclature-level genotyping results. The established HLA and KIR nomenclatures are primarily concerned with identification and characterization of individual alleles, and do not incorporate a grammar for genotyping results. GL String grammar is a string format for genotyping results from genetic systems with defined nomenclatures, such as HLA and KIR. In addition to genotyping results, the grammar can also describe data analysis artifacts such as multi-locus (multi-gene) haplotypes. From a terminology standpoint, the result of adding a compositional grammar to an established code system such as HLA or KIR nomenclature is a new code system, because the grammar may be used to construct expressions not explicitly defined in the original system as concept codes.|**URI:** http://glstring.org (Not Official)<br><br>**LRI Code:** None|
|<a href="https://terminology.hl7.org/CodeSystem-v3-hgnc.html">HGNC</a>|The HUGO Gene Nomenclature Committee (HGNC) table carries the gene ID, gene symbol and full gene name. Gene IDs must be sent as codes and begin with "HGNC:". Gene symbols should be sent as display. GENE IDs are specific to the species, whereas gene symbols and names are shared by all species with the same gene. The HGNC-Symb table provided by HUGO carries only human genes and is available in a table by LHC. The code for this coding system is the HGNC gene code, the "name" or print string is the HGNC gene symbol. More than 28,000 human gene symbols and names have been assigned so far, including almost all of the protein coding genes. NCBI creates what might be thought of as interim codes. Older systems might send HGNC gene IDs without the "HGNC:" prefix. This is not following the guidance of the creators of the code system, and introduces ambiguity. So caution must be taken to confirm alignment. HGNC also provides an index on gene families/groups. GeneGroup IDs do not begin with "HGNC:", so care must be made to ensure alignment of concepts when viewing an HGNC ID from an older system that may be referring to the GeneID and not a gene group. For example, <a href="https://www.genenames.org/data/genegroup/#!/group/588">588</a> refers to the HLA gene family, but <a href="https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:588">HGNC:588</a> identifies the ATG12 gene.|**URI:** http://www.genenames.org (Gene)<br>http://www.genenames.org/genegroup (Group)<br><br>**LRI Code:** HGNC-Symb|
|<a href="https://terminology.hl7.org/CodeSystem-v3-hgvs.html">HGVS</a>|The HGVS Nomenclature is an internationally-recognized standard for the description of DNA, RNA, and protein sequence variants. It is used to convey variants in clinical reports and to share variants in publications and databases. The format of the HGVS representation is "reference:description" where the reference is the reference sequence, and the description has the format of "prefix.position(s) change," where the prefix indicates indicates the type of reference sequence used, and the position(s) change describes the location of the change, and the nucleotide(s) changed.|**URI:** http://varnomen.hgvs.org<br><br>**LRI Code:** HGVS.g, HGVS.p, HGVS.c|
|<a href="https://www.ebi.ac.uk/ipd/imgt/hla/">HLA Nomenclature</a>|Human leukocyte antigen (HLA) is a gene family found in the Major Histocompatibility Complex (MHC) of Chromosome 6 in humans. This family includes more than 50 genes. A subset of these are commonly used as markers for histocompatibility testing for stem cell and solid organ transplantation, drug sensitivity, and disease association. The WHO Nomenclature Committee for Factors of the HLA System is responsible for a common nomenclature of HLA alleles, allele sequences, and quality control, to communicate histocompatibility typing information to match donors and recipients. An HLA allele is defined as any set of variations found on a sequence of DNA comprising a HLA gene. So, if there are five variations found in this one gene sequence, this set is defined as one allele (vs. the definition of an allele being the variation found between the test specimen and the reference along a contiguous stretch of DNA). In the case of HLA, the contiguous stretch of DNA represents the entire gene, and the variations do not need to be contiguous within the gene sequence. Each HLA allele name has a unique name consisting of the gene name followed by up to four fields, each containing at least two digits, separated by colons. There are also optional suffixes added to indicate expression status. Individual HLA genes may have thousands of different alleles. For the full specification, please go to this website: http://hla.alleles.org/nomenclature/naming.html. HLA nomenclature can also be used to represent sets of alleles that share sequence identity in the Antigen Recognition Site (ARS). G-groups are alleles that have identical DNA sequences in the ARS, while P-groups are alleles that have identical protein sequences in the ARS. These are described, respectively, in <a href="http://hla.alleles.org/alleles/g_groups.html">http://hla.alleles.org/alleles/g_groups.html</a> and <a href="http://hla.alleles.org/alleles/p_groups.html">http://hla.alleles.org/alleles/p_groups.html</a>.|**URI:** http://www.ebi.ac.uk/ipd/imgt/hla (Not Official)<br><br>**LRI Code:** HLA-Allele|
|<a href="https://terminology.hl7.org/CodeSystem-HPO.html">Human Phenotype Ontology</a>|The Human Phenotype Ontology (HPO) aims to provide a standardized vocabulary of phenotypic abnormalities encountered in human disease. Each term in the HPO describes a phenotypic abnormality, such as atrial septal defect.|**URI:** http://human-phenotype-ontology.org<br><br>**LRI Code:** HPO|
|<a href="https://terminology.hl7.org/CodeSystem-ISCN.html">ISCN</a>|Like HGVS, The International System for Human Cytogenetic Nomenclature (ISCN) is a syntax. It came out of cytopathology and deals with reporting karyotypes down to the chromosome fusions and many types of small copy number variants. However, cytogenetics is out of the scope in this guide. ISCN syntax is often used to report large deletion-duplications in structural variants and to include other variants that have been observed.|**URI:** https://iscn.karger.com<br><br>**LRI Code:** ISCN|
|<a href="https://terminology.hl7.org/CodeSystem-MONDO.html">Mondo Disease Ontology</a>|The Mondo Disease Ontology is a semi-automatically constructed ontology that merges in multiple disease resources to yield a coherent merged ontology that contains cross-species disease terminology. <br/><br/> Numerous sources for disease definitions and data models currently exist, which include HPO, OMIM, SNOMED CT, ICD, PhenoDB, MedDRA, MedGen, ORDO, DO, GARD, etc; however, these sources partially overlap and sometimes conflict, making it difficult to know definitively how they relate to each other. This has resulted in a proliferation of mappings between disease entries in different resources; however mappings are problematic: collectively, they are expensive to create and maintain. Most importantly, the mappings lack completeness, accuracy, and precision; as a result, mapping calls are often inconsistent between resources. The UMLS provides intermediate concepts through which other resources can be mapped, but these mappings suffer from the same challenges: they are not guaranteed to be one-to-one, especially in areas with evolving disease concepts such as rare disease.|**URI:** http://purl.obolibrary.org/obo/mondo.owl<br><br>**LRI Code:** MONDO|
|<a href="https://www.pharmvar.org/">PHARMVAR</a>|The Pharmacogene Variation (PharmVar) Consortium is a central repository for pharmacogene (PGx) variation that focuses on haplotype structure and allelic variation. The major focus of PharmVar is to catalogue allelic variation of genes impacting drug metabolism, disposition and response and provide a unifying designation system (nomenclature) for the global pharmacogenetic/genomic community. Efforts are synchronized between PharmVar, the Pharmacogenomic KnowledgeBase, and the Clinical Pharmacogenetic Implementation Consortium.|**URI:** http://www.pharmvar.org<br><br>**LRI Code:** Star-Allele|
|<a href="https://terminology.hl7.org/CodeSystem-v3-refSeq.html">RefSeq</a>|The NCBI Reference Sequence Database (RefSeq) is a set of reference sequences including genomic, transcript, and protient. RefSeqs include a prefix to describe the type of reference, such as NC_ or NG_. Those prefixed with "NC_" represent the whole genomic RefSeq for individual chromosomes. Those prefixed with "NG_" represent genes with all of their introns and flanking regions and other larger or smaller genomic sequences. These are available separately in the NCBI source data file, which includes all human RefSeqs (including those with prefix of NR_ or XM_): see ftp://ftp.ncbi.nlm.nih.gov/genomes/Homo_sapiens|**URI:** http://www.ncbi.nlm.nih.gov/refseq<br><br>**LRI Code:** RefSeq-G, RefSeq-P, RefSeq-T|
|<a href="https://terminology.hl7.org/CodeSystem-SO.html">Sequence Ontology</a>|Collaborative ontology project for the definition of sequence features used in biological sequence annotation.|**URI:** http://www.sequenceontology.org<br><br>**LRI Code:** None|


---

File: repos/HL7_SLASH_genomics-reporting/input/pages/pharmacogenomics.md


### Building a Pharmacogenomic Report: An Example

This section provides a walk-through of mapping from an example Pharmacogenomics (PGx) report to FHIR using the structure definitions and guidance defined in this IG. The PGx report is represented as a Genomic Report. The related observations and other resources are available as links in the table at the bottom of this section. The JSON representation of this report can be found [here](DiagnosticReport-PGxGenomicReportEMERGE.json.html).

A PGx report focuses on the impacts of a patient's genetics to the behavior of medications, and can provide recommended adjustments to medication based on these impacts. The intention of this page is to provide a “User Guide”, based on a specific report example, highlighting how the defined profiles can be used together to encode important relations in PGx reporting. The report and images for this guide are built upon work done by the eMERGE (Electronic Medical Records and Genomics) program, funded by the National Human Genome Research Institute (NHGRI). The Clinical Genomics Working Group has worked closely with eMERGE to integrate community feedback from their initial mapping into this PGx user guide.

### Ordering a Pharmacogenomic Test

In order to receive a PGx report, you must first order a PGx test. Genomic tests, like all laboratory tests, are ordered using the [Service Request](http://hl7.org/fhir/servicerequest.html) resource. The example report in this section is based on a pre-coordinated gene panel defined by the sequencing laboratory. As such, the “code” element on the PGx report aligns with a pre-coordinated “code” element on the corresponding ServiceRequest, though the laboratory may provide additional codes as well. Additional structure may be in scope for the report if additional “orderDetail” data are present in the Service Request or if supporting information such as current medications are available during generation. An overview of the general relationships involved in ordering a report and receiving the results can be found [here](general.html#relationships).

```xml
<ServiceRequest xmlns="http://hl7.org/fhir">
 <id value="eMERGEServiceRequest"/>
 ...
 <intent value="order"/>
 <code>
   <coding>
     <system value="http://example.org/hgsc.bcm.edu/lab-test-codes/"/>
     <code value="emerge-seq-ngs-pnl"/>
   </coding>
 </code>
 <subject>
   <reference value="Patient/CGPatientExample01"/>
 </subject>
 <specimen>
   <reference value="Specimen/GenomicSpecimenExample01"/>
 </specimen>
</ServiceRequest>
```

### How to use the Report Mapping Images

The guide is based on three figures. Figure 1 is an overview of the components of a PGx report. Figure 2 is a schema describing the use of the components from the HL7 Clinical Genomics WG Implementation Guide, and figure 3 is an example report with each element of the report mapped to the schema in figure 2. Use the numbers in each figure to find where the section/element goes. For example, #7 in figure 1 is information about the performing lab. In figure 2, #7 indicates an instance of the Organization Resource for the performer which would be a link through the performer element in the Genetics Diagnostic report. In figure 3, #7 indicates specific information about the performing lab found in the sample report. Thus, the specific performing lab information in figure 3 maps to an instance profile of Organization Resource linked to the Genetic Diagnostic report (#7 in figure 2).

In addition to the numbers, the colors in Figure 1 are intended to help guide the grouping of information in the report.

These images should provide you with a clear understanding of how the various FHIR PGx resources may be used when constructing a PGx report.

{% include img.html img="pgx-guidance-figure-1.jpg" caption="Figure 1: Color-coded/Numbered “Legend” for PGx Report" %}

### Example Schema for PGx Reporting

The Schema below depicts a “decomposition” of the PGx FHIR resources used to construct the example report. The color coding and numbers in the schema are mapped via the legend (image above) to the detailed PGx report shown in Figure #3.

{% include img.html img="pgx-guidance-figure-2.jpg" caption="Figure 2: Example PGx Report Schema" %}

This PGx report, like all genomic reporting, is packaged with the [diagnostic report](StructureDefinition-genomic-report.html) profile. It conveys metadata about the overall report via the [genomic study profile](StructureDefinition-genomic-study.html) and carries multiple “result” [Observations]({{site.data.fhir.path}}observation.html).
All of the observation findings may “hang” directly off of the [Genomic Report](StructureDefinition-genomic-report.html). <!--However, they can also be part of group referenced by a grouping observation. The grouping observation here has observation.code for the 'Group'/'Panel' NCI Metathesaurus term. It is used in this example PGx report to group all PGx-specific content (Therapeutic Implications and their derived Variants and Genotypes) into one section for viewing purposes. Note that the grouping by this profile does not add additional meaning to its child Observations. It is used just for convenience of presentation. Other configurations of one or more grouping Observations would also be valid, as no pre-coordinated groupings are recommended in this IG. If a group represents a specific panel then a specific panel code (such as a LOINC code) could be used as the observation.code for the grouping observation to give the grouping observation meaning.-->

All pharmacogenomic implications are intended to be communicated using components within the [Therapeutic Implication](StructureDefinition-therapeutic-implication.html) profile. The Medication-assessed component of the Therapeutic Implication profile, is the place to communicate the medication whose implication is being described, including a place for the coded value of the medication. Because this is an international profile, no guidance is provided on the medication coding system to be used. Instead, the appropriate code system for the jurisdiction, can and should be used.

To associate a Therapeutic-implication instance with a proposed recommended action (e.g., discontinuing a medication, altering dosage, or "choose alternative medication") one can use the [Medication Recommendation](StructureDefinition-medication-recommendation.html), potentially with a [MedicationStatement](http://hl7.org/fhir/R4/medicationstatement.html) instance. Use [RelatedArtifact](http://hl7.org/fhir/R4/extension-workflow-relatedartifact.html) in Therapeutic-implication to supply additional or supporting information, such as CPIC guidelines. In this report example, the text which explains the methodology of the test, was modeled using the [PlanDefinition](https://www.hl7.org/fhir/plandefinition.html) resource (#5 in the schema). The clinical genomics work group has not yet determined the value of providing a profile on PlanDefinition, and requests community feedback on encoding testing methodology details.

{% include img.html img="pgx-guidance-figure-3.jpg" caption="Figure 3: Detailed PGx Report Example - Mapped to Schema" %}

### Creating the Example FHIR PGx Report

In the example used in this User Guide, the PGx results are based on diplotypes (star alleles), and the underlying variant observations, found in each relevant PGx gene that is covered by the PGx gene panel. These diplotypes are then used as a basis for relating PGx gene-drug knowledge implications. Note that there was also the choice in this use case, to derive Implication statements from one or more genotype Observations and not from variants directly.

Below is one of the therapeutic implications in the report:

```xml
<Observation xmlns="http://hl7.org/fhir">
...
 <extension url="http://hl7.org/fhir/StructureDefinition/workflow-relatedArtifact">
   <valueRelatedArtifact>
     <type value="citation"/>
     <url value="https://cpicpgx.org/guidelines/guideline-for-warfarin-and-cyp2c9-and-vkorc1/"/>
   </valueRelatedArtifact>
 </extension>
 <code>
   <coding>
     <system value="http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs"/>
     <code value="therapeutic-implication"/>
   </coding>
 </code>
 <derivedFrom>
   <reference value="Observation/Pgx-geno-1002"/>
   <display value="CYP2C9*1/*1"/>
 </derivedFrom>
 <derivedFrom>
   <reference value="Observation/Pgx-geno-1003"/>
   <display value="VKORC1 rs9923231 C/T"/>
 </derivedFrom>
 <component>
   <code>
     <coding>
       <system value="http://loinc.org"/>
       <code value="51963-7"/>
     </coding>
   </code>
   <valueCodeableConcept>
     <coding>
       <system value="http://ncimeta.nci.nih.gov"/>
       <code value="C0043031"/>
       <display value="warfarin"/>
     </coding>
   </valueCodeableConcept>
 </component>
 <component>
   <code>
     <coding>
       <system value="http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs"/>
       <code value="therapeutic-implication"/>
     </coding>
   </code>
   <valueCodeableConcept>
     <system value="http://loinc.org"/>
     <code value="LA25391-6"/>
     <display value="Normal Metabolizer"/>
   </valueCodeableConcept>
 </component>
 <component>
   <code>
     <coding>
       <system value="http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/tbd-codes-cs"/>
       <code value="conclusion-string"/>
     </coding>
   </code>
   <valueString value="This individual is homozygous for the normal allele for the CYP2C9 gene. Based on the genotype result, this patient is predicted to have normal CYP2C9 function. This individual is also heterozygous for the variant allele for the VKORC1 gene. Expression level of the VKORC1 gene is associated with warfarin sensitivity. Based on the genotype result, this patient is predicted to have medium sensitivity to warfarin."/>
 </component>
</Observation>
```

The following table contains additional information regarding the profiles used in mapping this example PGx report to FHIR, as well as individual example instances

| **_Profile_**| **_Description of PGx usage_**| **_Example links_**| **_Example Notes_**|
| --- | --- | --- | --- |
| [Genomic Report](StructureDefinition-genomic-report-definitions.html) | Intended to capture a single report, with a Code element aligning with a Service Request and multiple genomic result Observations. Will also capture report conclusions.<br/> Not suitable for displaying summary information covering multiple reports. Extensions on DiagnosticReport can point to other reports. | [#DiagnosticReport-PGxGenomicReportEMERGE.html](DiagnosticReport-PGxGenomicReportEMERGE.html) | <!--Results may be on the report directly or referenced through grouping observations. -->This report has results directly referenced<!-- and 3 Observations used to group the PGX data into sections for viewing purposes. For an example without the grouping observations see [Emerge report without grouping](DiagnosticReport-PGxGenomicReportEMERGE.html)--> |
| [Genomic Study](StructureDefinition-genomic-study.html) | Describes the metadata about the study that was performed, including references to various analyses that were performed, and references to files created during the process. Will also create a reference describing metadata like the Regions Studied. |  [#Procedure-PGXGenomicStudy.html](Procedure-PGXGenomicStudy.html) | A simple example of a Genomic Study profile for a PGX panel. |
| [Genotype](StructureDefinition-genotype.html) | Connects haplotype or variant observations for homologous positions in the genome, or sets of homologous positions. In practice, it takes the capability of performing bioinformatic calculations to normalize across variant data to bring genotypes together. Genotype allows the sender to communicate a relationship between variant observations without the recipient needing to calculate the relationship. | [#Observation-Pgx-geno-1001.html](Observation-Pgx-geno-1001.html) <br/> [#Observation-Pgx-geno-1002.html](Observation-Pgx-geno-1002.html) <br/> [#Observation-Pgx-geno-1003.html](Observation-Pgx-geno-1003.html) | Genotype names such as star-alleles can appear in PGx reports either with or without a full description of the underlying coded haplotypes and observed variants. Genotype names MAY be coded, but will oftentimes be textual descriptions only. PGx guidance is often available at the level of these genotypes. |
| [Haplotype](StructureDefinition-haplotype.html) | Connects variant observations for changes in the same chromosome. In practice, it takes the capability of performing bioinformatic calculations to normalize across variant data to bring haplotypes together. Haplotype allows the sender to communicate a relationship between variant observations without the recipient needing to calculate the relationship. | (examples TBD) | Haplotype names will be star-alleles and can appear in PGx reports either with or without a full description of the underlying observed variants. Haplotype names SHOULD be coded values. PGx guidance is often available at the level of genotypes (diplotypes). |
| [Variant](StructureDefinition-variant.html) | Intended for communication of positional information derived from the patient’s sample. The data should be suitable for calculations involving variant level details, such as clinical decision support or presence of a specific variant in a patient. Variant also carries important observational information about the occurrence of the variant in the patient/sample, and some pre-coordinated annotations such as gene name. More complex annotations such as pathogenicity should use the DiagnosticImplication profile for additional structure.<br/>Not all variants observed in a patient’s sample are required to be included in a report. | [#Observation-Pgx-var-1011.html](Observation-Pgx-var-1011.html)<br/>[#Observation-Pgx-var-1012.html](Observation-Pgx-var-1012.html) <br/>[#Observation-Pgx-var-1013.html](Observation-Pgx-var-1013.html) <br/>[#Observation-Pgx-var-1014.html](Observation-Pgx-var-1014.html) <br/>[#Observation-Pgx-var-1015.html](Observation-Pgx-var-1015.html) <br/>[#Observation-Pgx-var-1016.html](Observation-Pgx-var-1016.html) <br/>[#Observation-Pgx-var-1017.html](Observation-Pgx-var-1017.html) <br/>[#Observation-Pgx-var-1018.html](Observation-Pgx-var-1018.html) | In this example, variants are represented primarily using HGVS, with an Observation value of ‘Present’. This provides a computable observation of change, position in a reference sequence. <br/> Other components could be used to “define” the variant in question, for example ClinVar codes or a VCF column-like representation as seen in other examples. |
| [Therapeutic Implication](StructureDefinition-therapeutic-implication.html) | All pharmacogenomic implications are intended to be communicated using components within the Therapeutic Implication profile, which has properties for observations that convey the potential impact of genomic characteristics on a medication or non-medicinal therapy. The medication-assessed component can be repeated to represent combination therapies. Otherwise, the recommendation is to limit Observations to one drug each to maintain semantic relationships in downstream systems. | [#Observation-TxImp01.html](Observation-TxImp01.html) <br/>[#Observation-TxImp02.html](Observation-TxImp02.html) <br/>[#Observation-TxImp03.html](Observation-TxImp03.html) <br/>[#Observation-TxImp04.html](Observation-TxImp04.html) <br/>[#Observation-TxImp05.html](Observation-TxImp05.html) | In addition to the coded components, the conclusion-string component can provide text that the lab would normally include in a report narrative.<br /><br />Note that one implication can be derived from multiple genotypes and/or variant Observations, and those Observations may derive multiple implications. |
| [Medication Recommendation](StructureDefinition-medication-recommendation.html) | The Medication Recommendation is a recommended action Task describing what sort of change (if any) should be made for a specific medication based on an identified finding. This is a computable specific recommendation for action on a specific medication, such as changing a dose. It is a structured means of reporting recommendations. Here tied to a Therapeutic Implication to provide guidance such as changes to a dose or alternative medications. | [#Task-PGxRecEx01.html](Task-PGxRecEx01.html) <br/>[#Task-PGxRecEx02.html](Task-PGxRecEx02.html)<br/>[#Task-PGxRecEx03.html](Task-PGxRecEx03.html)<br/>[#Task-PGxRecEx04.html](Task-PGxRecEx04.html)<br/>[#Task-PGxRecEx05.html](Task-PGxRecEx05.html) | Not included in this example, ‘Task.focus’ can be used to refer to a MedicationStatement for the patient, if available. <br/>See an alternative example using this pattern at [#Task-MedicationRecommendationExample1.html](Task-MedicationRecommendationExample1.html). |
| [Recommended Action](StructureDefinition-recommended-action.html) (extension) | The recommended action (extension) references a proposed action that is recommended based on the results of the diagnostic report. It is not linked to individual variants and medications. Another possible type of recommendation is for additional follow-up testing of the subject or family members. This is an extension and may not be understood by all systems so care must be made that receivers are able to consume Task resources not normally linked with DiagnosticReports. It is intended as a textual recommendation. The information could also appear in the ‘text’ element of the diagnostic report, which may be improved to support coded data in future releases of FHIR. | | Note that extensions may not be included in generated narrative summaries. |
| [Related Artifact](http://hl7.org/fhir/R4/extension-workflow-relatedartifact.html) (extension) | Captures citations, evidence and other supporting documentation for the observation or report. Included on the DiagnosticReport and Implication Observation structure through an extension.<br /><br />This dataType allows more structure than what would easily be represented in a text narrative.<br /><br />Useful for referring to guidelines, infobutton, lab methodology or any other supporting documentation. | | Note that extensions may not be included in generated narrative summaries. |
| [Genomic Risk Assessment](StructureDefinition-genomic-risk-assessment.html) | While not used in the eMERGE example, the Genomic Risk Assessment is used in a diagnostic report or observation to refer to a RiskAssessment resource. The RiskAssessment resource captures predicted outcomes for a patient or population on the basis of source information. Examples include: A prognosis statement for a particular condition; Risk of health outcome (heart attack, particular type of cancer) on the basis of lifestyle factors and/or family history; List of potential health risks based on a patient's genetic analysis.<br /><br />This resource can be used to represent the results of formal scoring/decision support tools that evaluate risk. It can also be used to capture a practitioner's subjective assessment of the patient's risk based on existing knowledge and previous experience. | | Not used in this example |
| [Plan Definition](https://www.hl7.org/fhir/plandefinition.html) | A plan definition is a pre-defined group of actions to be taken in particular circumstances, often including conditional elements, options, and other decision points. The resource is flexible enough to be used to represent a variety of workflows, as well as clinical decision support and quality improvement assets, including order sets, protocols, and decision support rules. | | In the eMERGE report, this is used to provide additional methodology information, as an alternative to using related artifact or Observation.method. This use is still experimental and has not been included in our example. It is labeled as 'X1' in the figures. |

{: .grid}

This [Artifact List](artifacts.html) provides a comprehensive listing of the FHIR artifacts defined as part of the Genomics Implementation Guide and may be helpful when developing the PGx report.

### Mapping to eMERGE III FHIR Template

Notable changes from the [eMERGE III FHIR Template](https://emerge-fhir-spec.readthedocs.io/en/latest/) docs include:

1. Scope - this example focuses on the PGx panel within the eMERGE report and does not include a separate diagnostic panel.
1. Extensions - the eMERGE ‘summary interpretation text extension’ is now supported through an Observation component. The eMERGE usage of the ‘report disclaimer text extension’ is not included in our example but is seen by the 'X2' label in the figures.
1. Medication Recommendation - the original eMERGE spec did not structure medication recommendation proposals, which were left in plain text comments.

For further information about the eMERGE Program, see: [Pre-Print Journal Article: "Genomic Considerations for FHIR; eMERGE Implementation Lessons"](https://www.biorxiv.org/content/10.1101/2021.01.31.429037v1.full) and [eMERGE Network Website](https://emerge-network.org/).


---

File: repos/HL7_SLASH_genomics-reporting/input/pages/spec.md

<!-- spec.md {% comment %}
*****************************************************************************************
*                            WARNING: DO NOT EDIT THIS FILE                             *
*                                                                                       *
* This file is generated by SUSHI. Any edits you make to this file will be overwritten. *
*                                                                                       *
* To change the contents of this file, edit the original source file at:                *
* genomics-reporting/input/pages/spec.md                                                *
*****************************************************************************************
{% endcomment %} -->
### A Heading
You can also use markdown if that's your thing

And an icon: ![resource](icon-resource.png)


---

